{
 "cells": [
  {
   "cell_type": "code",
   "execution_count": 1,
   "id": "9732b29d-55c0-4a0d-a4f1-8cd54548d398",
   "metadata": {},
   "outputs": [],
   "source": [
    "import pandas as pd\n",
    "\n",
    "#### MAKE SCORE FILE\n",
    "\n",
    "def make_score(df): \n",
    "    small = df[[\"Genotype\", \"REF\", \"ALT\", \"beta\"]].copy()\n",
    "    small_bad = small[small[\"beta\"] < 0 ] \n",
    "    small_good = small[small[\"beta\"] > 0 ] \n",
    "    small_bad[\"weight\"] = small_bad[\"beta\"] * -1 \n",
    "    small_bad[\"A1\"] = small_bad[\"REF\"] \n",
    "    small_bad = small_bad[[\"Genotype\", \"A1\", \"weight\"]].copy()\n",
    "    small_good[\"weight\"] = small_good[\"beta\"]\n",
    "    small_good[\"A1\"] = small_good[\"ALT\"]\n",
    "    small_good = small_good[[\"Genotype\", \"A1\", \"weight\"]].copy()\n",
    "    SCORE = pd.concat([small_good, small_bad], 0 ) \n",
    "    SCORE[\"Genotype\"] = SCORE[\"Genotype\"].str.replace(\"-\",\":\")\n",
    "    return(SCORE)\n",
    "\n",
    "###### BASE FUNCTION TO DO CLUMPING BASED ON VARIANT CLASS\n",
    "import warnings\n",
    "warnings.filterwarnings('ignore')\n",
    "\n",
    "\n",
    "def normal_clump(df): \n",
    "    df = df.sort_values([\"CHR\",\"BP\"])\n",
    "    df = df.drop_duplicates(subset=['Genotype'], keep='first') ##### This is added as same protein/genotype across different array types\n",
    "    print (\"Number of variants before clump: \" + str(df.shape[0]) ) \n",
    "    remove_list = []\n",
    "    list1 = df.Genotype.to_list()\n",
    "    list2 = list1[::-1]\n",
    "    for i in list1: \n",
    "        for b in list2: \n",
    "            if i == b: \n",
    "                next\n",
    "            if i != b: \n",
    "                i_chr =  df[df[\"Genotype\"] == i ].CHR.values\n",
    "                b_chr =  df[df[\"Genotype\"] == b ].CHR.values\n",
    "                i_bp =  df[df[\"Genotype\"] == i ].BP.values\n",
    "                b_bp =  df[df[\"Genotype\"] == b ].BP.values\n",
    "                i_group =  df[df[\"Genotype\"] == i ][\"p-value\"].values\n",
    "                b_group =  df[df[\"Genotype\"] == b ][\"p-value\"].values\n",
    "\n",
    "                if i_chr == b_chr : \n",
    "                    closeby = abs(int(b_bp) - int(i_bp))\n",
    "                    if closeby < 10000000 : \n",
    "                        i2 = i.replace(\"-\", \":\")\n",
    "                        b2 = b.replace(\"-\", \":\")\n",
    "                        try: \n",
    "                            R = ld_matrix[(ld_matrix[\"SNP_A\"] == i2 ) & (ld_matrix[\"SNP_B\"] == b2 )].R2.values \n",
    "                        except: \n",
    "                            R = 0\n",
    "                        if R > 0.05 : \n",
    "                            if i_group > b_group: \n",
    "                                remove_list.append(i) \n",
    "                            elif i_group < b_group:\n",
    "                                remove_list.append(b) \n",
    "                            else: \n",
    "                                remove_list.append(i)\n",
    "    temp_df = df[~df[\"Genotype\"].isin(remove_list) ] \n",
    "    print (\"Number of variants after clump: \" + str(temp_df.shape[0]))\n",
    "    return temp_df "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 44,
   "id": "e21836fa-4e7f-4daa-8641-ce4360f15d07",
   "metadata": {},
   "outputs": [],
   "source": [
    "### add something\n",
    "#raw = pd.read_csv(\"ewas_results_genomodel.txt\", sep=\"\\t\")\n",
    "raw = pd.read_excel(\"exome_gwas.xlsx\" ) \n",
    "proteins_of_interest = pd.read_csv(\"proteins.txt\",sep=\"\\t\", header=None)\n",
    "proteins_of_interest.columns = [\"Protein\"] \n",
    "our_proteins = raw[raw[\"Protein\"].isin(proteins_of_interest[\"Protein\"] ) ]\n",
    "### LD matrix was created on the RAP on variants defined in paper\n",
    "ld_matrix = pd.read_csv(\"ld_matrix.ld\", delimiter=r\"\\s+\" )\n",
    "ld_matrix = ld_matrix[[\"SNP_A\", \"SNP_B\",\"R2\"]].copy() \n",
    "our_proteins[[\"CHR\", \"BP\", \"REF\" , \"ALT\"]] = our_proteins.Genotype.str.split(\"-\", expand=True) \n",
    "grouping_lists = [ [\"stop_gained\", \"disruptive_inframe_deletion\", \"disruptive_inframe_insertion\", \"frameshift_variant\", \"conservative_inframe_deletion\", \"splice_acceptor_variant\", \"start_lost\", \"splice_donor_variant\"], [\"missense_variant\"] , \n",
    "                  [\"synonymous_variant\"] ,  [\"non_coding_transcript_exon_variant\" ], ['3_prime_UTR_variant','5_prime_UTR_variant', 'synonymous_variant&splice_region_variant', \"5_prime_UTR_premature_start_codon_gain_variant\",\"missense_variant&splice_region_variant\", \"stop_retained_variant&splice_region_variant\", \"5_prime_UTR_truncation\", \"stop_retained_variant\"]] \n",
    "names =  ['High_IMPACT', 'MISS', 'SYNONYMOUS', 'NON_CODING', 'OTHER']\n",
    "\n",
    "var_scores = [\"5\", \"4\" , \"2\", \"3\", \"1\"]   \n",
    "\n",
    "\n",
    "var_class = (pd.DataFrame({'Groups': names, \"Most Damaging Effect\": grouping_lists , \"variant_class_score\": var_scores})\n",
    "        .explode('Most Damaging Effect')\n",
    "        .reset_index(drop=True))\n",
    "\n",
    "our_protein1 = our_proteins.merge(var_class,  on = \"Most Damaging Effect\", how='left').fillna(\"Other\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 34,
   "id": "597e64dc-d30c-450c-a105-78d5ed875e03",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Protein</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>135</th>\n",
       "      <td>C19ORF12</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "      Protein\n",
       "135  C19ORF12"
      ]
     },
     "execution_count": 34,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "proteins_of_interest[proteins_of_interest[\"Protein\"] == \"C19ORF12\"] "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 41,
   "id": "b8190fc7-460c-416a-84db-aeaa44050eb0",
   "metadata": {
    "tags": []
   },
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Gene</th>\n",
       "      <th>Protein</th>\n",
       "      <th>Match(?Cis)</th>\n",
       "      <th>cis_trans_position_1mb_coding_region</th>\n",
       "      <th>Num_Proteins_p&lt;1x10-8</th>\n",
       "      <th>Genotype</th>\n",
       "      <th>Most Damaging Effect</th>\n",
       "      <th>nSamples</th>\n",
       "      <th>nCarriers</th>\n",
       "      <th>nNonCarriers</th>\n",
       "      <th>p-value</th>\n",
       "      <th>beta</th>\n",
       "      <th>lower</th>\n",
       "      <th>upper</th>\n",
       "      <th>se</th>\n",
       "      <th>model</th>\n",
       "      <th>panel</th>\n",
       "      <th>Olink Assay</th>\n",
       "      <th>MAF</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>53685</th>\n",
       "      <td>PNPLA3</td>\n",
       "      <td>C19orf12</td>\n",
       "      <td>No</td>\n",
       "      <td>trans-position, trans-gene</td>\n",
       "      <td>96</td>\n",
       "      <td>22-43928847-C-G</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>44579</td>\n",
       "      <td>16517</td>\n",
       "      <td>28062</td>\n",
       "      <td>4.489905e-30</td>\n",
       "      <td>0.090495</td>\n",
       "      <td>0.074939</td>\n",
       "      <td>0.106051</td>\n",
       "      <td>0.007937</td>\n",
       "      <td>genotypic</td>\n",
       "      <td>Neurology</td>\n",
       "      <td>C19orf12:Q9NSK7:OID20804:v1:Neurology</td>\n",
       "      <td>0.209341</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>62572</th>\n",
       "      <td>PNPLA3</td>\n",
       "      <td>C19orf12</td>\n",
       "      <td>No</td>\n",
       "      <td>trans-position, trans-gene</td>\n",
       "      <td>96</td>\n",
       "      <td>22-43928847-C-G</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>44579</td>\n",
       "      <td>2150</td>\n",
       "      <td>42429</td>\n",
       "      <td>1.265995e-25</td>\n",
       "      <td>0.226142</td>\n",
       "      <td>0.183809</td>\n",
       "      <td>0.268475</td>\n",
       "      <td>0.021598</td>\n",
       "      <td>recessive</td>\n",
       "      <td>Neurology</td>\n",
       "      <td>C19orf12:Q9NSK7:OID20804:v1:Neurology</td>\n",
       "      <td>0.209341</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>71018</th>\n",
       "      <td>C19orf12</td>\n",
       "      <td>C19orf12</td>\n",
       "      <td>Yes</td>\n",
       "      <td>cis-position, cis-gene</td>\n",
       "      <td>1</td>\n",
       "      <td>19-29702747-T-C</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>45354</td>\n",
       "      <td>83</td>\n",
       "      <td>45271</td>\n",
       "      <td>2.102760e-22</td>\n",
       "      <td>-1.045686</td>\n",
       "      <td>-1.256074</td>\n",
       "      <td>-0.835299</td>\n",
       "      <td>0.107340</td>\n",
       "      <td>dominant</td>\n",
       "      <td>Neurology</td>\n",
       "      <td>C19orf12:Q9NSK7:OID20804:v1:Neurology</td>\n",
       "      <td>0.000898</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>71019</th>\n",
       "      <td>C19orf12</td>\n",
       "      <td>C19orf12</td>\n",
       "      <td>Yes</td>\n",
       "      <td>cis-position, cis-gene</td>\n",
       "      <td>1</td>\n",
       "      <td>19-29702747-T-C</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>45354</td>\n",
       "      <td>83</td>\n",
       "      <td>45271</td>\n",
       "      <td>2.102760e-22</td>\n",
       "      <td>-1.045686</td>\n",
       "      <td>-1.256074</td>\n",
       "      <td>-0.835299</td>\n",
       "      <td>0.107340</td>\n",
       "      <td>genotypic</td>\n",
       "      <td>Neurology</td>\n",
       "      <td>C19orf12:Q9NSK7:OID20804:v1:Neurology</td>\n",
       "      <td>0.000898</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>72643</th>\n",
       "      <td>PON1</td>\n",
       "      <td>C19orf12</td>\n",
       "      <td>No</td>\n",
       "      <td>trans-position, trans-gene</td>\n",
       "      <td>9</td>\n",
       "      <td>7-95316772-A-T</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>45041</td>\n",
       "      <td>26626</td>\n",
       "      <td>18415</td>\n",
       "      <td>6.914395e-22</td>\n",
       "      <td>0.064944</td>\n",
       "      <td>0.051712</td>\n",
       "      <td>0.078176</td>\n",
       "      <td>0.006751</td>\n",
       "      <td>genotypic</td>\n",
       "      <td>Neurology</td>\n",
       "      <td>C19orf12:Q9NSK7:OID20804:v1:Neurology</td>\n",
       "      <td>0.362249</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>80111</th>\n",
       "      <td>PNPLA3</td>\n",
       "      <td>C19orf12</td>\n",
       "      <td>No</td>\n",
       "      <td>trans-position, trans-gene</td>\n",
       "      <td>96</td>\n",
       "      <td>22-43928847-C-G</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>44579</td>\n",
       "      <td>16517</td>\n",
       "      <td>28062</td>\n",
       "      <td>8.372943e-20</td>\n",
       "      <td>0.087349</td>\n",
       "      <td>0.068561</td>\n",
       "      <td>0.106137</td>\n",
       "      <td>0.009586</td>\n",
       "      <td>dominant</td>\n",
       "      <td>Neurology</td>\n",
       "      <td>C19orf12:Q9NSK7:OID20804:v1:Neurology</td>\n",
       "      <td>0.209341</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>80157</th>\n",
       "      <td>PON1</td>\n",
       "      <td>C19orf12</td>\n",
       "      <td>No</td>\n",
       "      <td>trans-position, trans-gene</td>\n",
       "      <td>5</td>\n",
       "      <td>7-95308134-T-C</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>45215</td>\n",
       "      <td>22253</td>\n",
       "      <td>22962</td>\n",
       "      <td>8.570174e-20</td>\n",
       "      <td>-0.083754</td>\n",
       "      <td>-0.101774</td>\n",
       "      <td>-0.065734</td>\n",
       "      <td>0.009194</td>\n",
       "      <td>dominant</td>\n",
       "      <td>Neurology</td>\n",
       "      <td>C19orf12:Q9NSK7:OID20804:v1:Neurology</td>\n",
       "      <td>0.288201</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>80271</th>\n",
       "      <td>PON3</td>\n",
       "      <td>C19orf12</td>\n",
       "      <td>No</td>\n",
       "      <td>trans-position, trans-gene</td>\n",
       "      <td>8</td>\n",
       "      <td>7-95396288-G-A</td>\n",
       "      <td>synonymous_variant</td>\n",
       "      <td>45167</td>\n",
       "      <td>20492</td>\n",
       "      <td>24675</td>\n",
       "      <td>9.106496e-20</td>\n",
       "      <td>-0.066891</td>\n",
       "      <td>-0.081294</td>\n",
       "      <td>-0.052489</td>\n",
       "      <td>0.007348</td>\n",
       "      <td>genotypic</td>\n",
       "      <td>Neurology</td>\n",
       "      <td>C19orf12:Q9NSK7:OID20804:v1:Neurology</td>\n",
       "      <td>0.261920</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>80510</th>\n",
       "      <td>SAMM50</td>\n",
       "      <td>C19orf12</td>\n",
       "      <td>No</td>\n",
       "      <td>trans-position, trans-gene</td>\n",
       "      <td>40</td>\n",
       "      <td>22-43972242-A-G</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>44307</td>\n",
       "      <td>11977</td>\n",
       "      <td>32330</td>\n",
       "      <td>1.033495e-19</td>\n",
       "      <td>0.083177</td>\n",
       "      <td>0.065241</td>\n",
       "      <td>0.101112</td>\n",
       "      <td>0.009151</td>\n",
       "      <td>genotypic</td>\n",
       "      <td>Neurology</td>\n",
       "      <td>C19orf12:Q9NSK7:OID20804:v1:Neurology</td>\n",
       "      <td>0.147319</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>80566</th>\n",
       "      <td>PON1</td>\n",
       "      <td>C19orf12</td>\n",
       "      <td>No</td>\n",
       "      <td>trans-position, trans-gene</td>\n",
       "      <td>5</td>\n",
       "      <td>7-95308134-T-C</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>45215</td>\n",
       "      <td>22253</td>\n",
       "      <td>22962</td>\n",
       "      <td>1.062281e-19</td>\n",
       "      <td>-0.065073</td>\n",
       "      <td>-0.079110</td>\n",
       "      <td>-0.051036</td>\n",
       "      <td>0.007161</td>\n",
       "      <td>genotypic</td>\n",
       "      <td>Neurology</td>\n",
       "      <td>C19orf12:Q9NSK7:OID20804:v1:Neurology</td>\n",
       "      <td>0.288201</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>80584</th>\n",
       "      <td>PON2</td>\n",
       "      <td>C19orf12</td>\n",
       "      <td>No</td>\n",
       "      <td>trans-position, trans-gene</td>\n",
       "      <td>11</td>\n",
       "      <td>7-95411704-G-C</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>45186</td>\n",
       "      <td>18826</td>\n",
       "      <td>26360</td>\n",
       "      <td>1.073595e-19</td>\n",
       "      <td>-0.069097</td>\n",
       "      <td>-0.084004</td>\n",
       "      <td>-0.054191</td>\n",
       "      <td>0.007605</td>\n",
       "      <td>genotypic</td>\n",
       "      <td>Neurology</td>\n",
       "      <td>C19orf12:Q9NSK7:OID20804:v1:Neurology</td>\n",
       "      <td>0.236988</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>83022</th>\n",
       "      <td>PON3</td>\n",
       "      <td>C19orf12</td>\n",
       "      <td>No</td>\n",
       "      <td>trans-position, trans-gene</td>\n",
       "      <td>8</td>\n",
       "      <td>7-95396288-G-A</td>\n",
       "      <td>synonymous_variant</td>\n",
       "      <td>45167</td>\n",
       "      <td>20492</td>\n",
       "      <td>24675</td>\n",
       "      <td>3.966466e-19</td>\n",
       "      <td>-0.082595</td>\n",
       "      <td>-0.100700</td>\n",
       "      <td>-0.064491</td>\n",
       "      <td>0.009237</td>\n",
       "      <td>dominant</td>\n",
       "      <td>Neurology</td>\n",
       "      <td>C19orf12:Q9NSK7:OID20804:v1:Neurology</td>\n",
       "      <td>0.261920</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>86869</th>\n",
       "      <td>PON2</td>\n",
       "      <td>C19orf12</td>\n",
       "      <td>No</td>\n",
       "      <td>trans-position, trans-gene</td>\n",
       "      <td>11</td>\n",
       "      <td>7-95411704-G-C</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>45186</td>\n",
       "      <td>18826</td>\n",
       "      <td>26360</td>\n",
       "      <td>2.591988e-18</td>\n",
       "      <td>-0.081437</td>\n",
       "      <td>-0.099717</td>\n",
       "      <td>-0.063157</td>\n",
       "      <td>0.009327</td>\n",
       "      <td>dominant</td>\n",
       "      <td>Neurology</td>\n",
       "      <td>C19orf12:Q9NSK7:OID20804:v1:Neurology</td>\n",
       "      <td>0.236988</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>94714</th>\n",
       "      <td>PON3</td>\n",
       "      <td>C19orf12</td>\n",
       "      <td>No</td>\n",
       "      <td>trans-position, trans-gene</td>\n",
       "      <td>4</td>\n",
       "      <td>7-95372243-C-T</td>\n",
       "      <td>synonymous_variant</td>\n",
       "      <td>45127</td>\n",
       "      <td>32522</td>\n",
       "      <td>12605</td>\n",
       "      <td>7.389682e-17</td>\n",
       "      <td>-0.054160</td>\n",
       "      <td>-0.066882</td>\n",
       "      <td>-0.041437</td>\n",
       "      <td>0.006491</td>\n",
       "      <td>genotypic</td>\n",
       "      <td>Neurology</td>\n",
       "      <td>C19orf12:Q9NSK7:OID20804:v1:Neurology</td>\n",
       "      <td>0.473081</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>98307</th>\n",
       "      <td>SERPINA1</td>\n",
       "      <td>C19orf12</td>\n",
       "      <td>No</td>\n",
       "      <td>trans-position, trans-gene</td>\n",
       "      <td>135</td>\n",
       "      <td>14-94378610-C-T</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>45346</td>\n",
       "      <td>1835</td>\n",
       "      <td>43511</td>\n",
       "      <td>2.850509e-16</td>\n",
       "      <td>0.188398</td>\n",
       "      <td>0.143270</td>\n",
       "      <td>0.233526</td>\n",
       "      <td>0.023024</td>\n",
       "      <td>genotypic</td>\n",
       "      <td>Neurology</td>\n",
       "      <td>C19orf12:Q9NSK7:OID20804:v1:Neurology</td>\n",
       "      <td>0.020598</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>100007</th>\n",
       "      <td>SERPINA1</td>\n",
       "      <td>C19orf12</td>\n",
       "      <td>No</td>\n",
       "      <td>trans-position, trans-gene</td>\n",
       "      <td>135</td>\n",
       "      <td>14-94378610-C-T</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>45346</td>\n",
       "      <td>1835</td>\n",
       "      <td>43511</td>\n",
       "      <td>5.262737e-16</td>\n",
       "      <td>0.188841</td>\n",
       "      <td>0.143193</td>\n",
       "      <td>0.234490</td>\n",
       "      <td>0.023290</td>\n",
       "      <td>dominant</td>\n",
       "      <td>Neurology</td>\n",
       "      <td>C19orf12:Q9NSK7:OID20804:v1:Neurology</td>\n",
       "      <td>0.020598</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>100978</th>\n",
       "      <td>PON1</td>\n",
       "      <td>C19orf12</td>\n",
       "      <td>No</td>\n",
       "      <td>trans-position, trans-gene</td>\n",
       "      <td>9</td>\n",
       "      <td>7-95316772-A-T</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>45041</td>\n",
       "      <td>5983</td>\n",
       "      <td>39058</td>\n",
       "      <td>7.516038e-16</td>\n",
       "      <td>0.109467</td>\n",
       "      <td>0.082862</td>\n",
       "      <td>0.136071</td>\n",
       "      <td>0.013573</td>\n",
       "      <td>recessive</td>\n",
       "      <td>Neurology</td>\n",
       "      <td>C19orf12:Q9NSK7:OID20804:v1:Neurology</td>\n",
       "      <td>0.362249</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>102543</th>\n",
       "      <td>PON3</td>\n",
       "      <td>C19orf12</td>\n",
       "      <td>No</td>\n",
       "      <td>trans-position, trans-gene</td>\n",
       "      <td>3</td>\n",
       "      <td>7-95394734-A-G</td>\n",
       "      <td>missense_variant&amp;splice_region_variant</td>\n",
       "      <td>44639</td>\n",
       "      <td>31475</td>\n",
       "      <td>13164</td>\n",
       "      <td>1.286384e-15</td>\n",
       "      <td>-0.052099</td>\n",
       "      <td>-0.064865</td>\n",
       "      <td>-0.039333</td>\n",
       "      <td>0.006513</td>\n",
       "      <td>genotypic</td>\n",
       "      <td>Neurology</td>\n",
       "      <td>C19orf12:Q9NSK7:OID20804:v1:Neurology</td>\n",
       "      <td>0.460052</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>107821</th>\n",
       "      <td>SAMM50</td>\n",
       "      <td>C19orf12</td>\n",
       "      <td>No</td>\n",
       "      <td>trans-position, trans-gene</td>\n",
       "      <td>40</td>\n",
       "      <td>22-43972242-A-G</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>44307</td>\n",
       "      <td>11977</td>\n",
       "      <td>32330</td>\n",
       "      <td>6.380197e-15</td>\n",
       "      <td>0.081493</td>\n",
       "      <td>0.061012</td>\n",
       "      <td>0.101974</td>\n",
       "      <td>0.010449</td>\n",
       "      <td>dominant</td>\n",
       "      <td>Neurology</td>\n",
       "      <td>C19orf12:Q9NSK7:OID20804:v1:Neurology</td>\n",
       "      <td>0.147319</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>108449</th>\n",
       "      <td>PON1</td>\n",
       "      <td>C19orf12</td>\n",
       "      <td>No</td>\n",
       "      <td>trans-position, trans-gene</td>\n",
       "      <td>9</td>\n",
       "      <td>7-95316772-A-T</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>45041</td>\n",
       "      <td>26626</td>\n",
       "      <td>18415</td>\n",
       "      <td>7.539049e-15</td>\n",
       "      <td>0.072897</td>\n",
       "      <td>0.054526</td>\n",
       "      <td>0.091267</td>\n",
       "      <td>0.009372</td>\n",
       "      <td>dominant</td>\n",
       "      <td>Neurology</td>\n",
       "      <td>C19orf12:Q9NSK7:OID20804:v1:Neurology</td>\n",
       "      <td>0.362249</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>109034</th>\n",
       "      <td>PON3</td>\n",
       "      <td>C19orf12</td>\n",
       "      <td>No</td>\n",
       "      <td>trans-position, trans-gene</td>\n",
       "      <td>4</td>\n",
       "      <td>7-95372243-C-T</td>\n",
       "      <td>synonymous_variant</td>\n",
       "      <td>45127</td>\n",
       "      <td>32522</td>\n",
       "      <td>12605</td>\n",
       "      <td>8.923837e-15</td>\n",
       "      <td>-0.079542</td>\n",
       "      <td>-0.099642</td>\n",
       "      <td>-0.059441</td>\n",
       "      <td>0.010255</td>\n",
       "      <td>dominant</td>\n",
       "      <td>Neurology</td>\n",
       "      <td>C19orf12:Q9NSK7:OID20804:v1:Neurology</td>\n",
       "      <td>0.473081</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>118162</th>\n",
       "      <td>PARVB</td>\n",
       "      <td>C19orf12</td>\n",
       "      <td>No</td>\n",
       "      <td>trans-position, trans-gene</td>\n",
       "      <td>23</td>\n",
       "      <td>22-43999571-T-C</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>44892</td>\n",
       "      <td>27862</td>\n",
       "      <td>17030</td>\n",
       "      <td>9.704103e-14</td>\n",
       "      <td>0.049686</td>\n",
       "      <td>0.036609</td>\n",
       "      <td>0.062762</td>\n",
       "      <td>0.006672</td>\n",
       "      <td>genotypic</td>\n",
       "      <td>Neurology</td>\n",
       "      <td>C19orf12:Q9NSK7:OID20804:v1:Neurology</td>\n",
       "      <td>0.385625</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>118676</th>\n",
       "      <td>PON3</td>\n",
       "      <td>C19orf12</td>\n",
       "      <td>No</td>\n",
       "      <td>trans-position, trans-gene</td>\n",
       "      <td>3</td>\n",
       "      <td>7-95394734-A-G</td>\n",
       "      <td>missense_variant&amp;splice_region_variant</td>\n",
       "      <td>44639</td>\n",
       "      <td>31475</td>\n",
       "      <td>13164</td>\n",
       "      <td>1.102429e-13</td>\n",
       "      <td>-0.075378</td>\n",
       "      <td>-0.095261</td>\n",
       "      <td>-0.055494</td>\n",
       "      <td>0.010145</td>\n",
       "      <td>dominant</td>\n",
       "      <td>Neurology</td>\n",
       "      <td>C19orf12:Q9NSK7:OID20804:v1:Neurology</td>\n",
       "      <td>0.460052</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>120112</th>\n",
       "      <td>SAMM50</td>\n",
       "      <td>C19orf12</td>\n",
       "      <td>No</td>\n",
       "      <td>trans-position, trans-gene</td>\n",
       "      <td>40</td>\n",
       "      <td>22-43972242-A-G</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>44307</td>\n",
       "      <td>1089</td>\n",
       "      <td>43218</td>\n",
       "      <td>1.526849e-13</td>\n",
       "      <td>0.221422</td>\n",
       "      <td>0.162672</td>\n",
       "      <td>0.280172</td>\n",
       "      <td>0.029974</td>\n",
       "      <td>recessive</td>\n",
       "      <td>Neurology</td>\n",
       "      <td>C19orf12:Q9NSK7:OID20804:v1:Neurology</td>\n",
       "      <td>0.147319</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>134913</th>\n",
       "      <td>PARVB</td>\n",
       "      <td>C19orf12</td>\n",
       "      <td>No</td>\n",
       "      <td>trans-position, trans-gene</td>\n",
       "      <td>23</td>\n",
       "      <td>22-43999571-T-C</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>44892</td>\n",
       "      <td>6786</td>\n",
       "      <td>38106</td>\n",
       "      <td>2.972377e-12</td>\n",
       "      <td>0.089952</td>\n",
       "      <td>0.064697</td>\n",
       "      <td>0.115208</td>\n",
       "      <td>0.012885</td>\n",
       "      <td>recessive</td>\n",
       "      <td>Neurology</td>\n",
       "      <td>C19orf12:Q9NSK7:OID20804:v1:Neurology</td>\n",
       "      <td>0.385625</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>135046</th>\n",
       "      <td>C19orf12</td>\n",
       "      <td>C19orf12</td>\n",
       "      <td>Yes</td>\n",
       "      <td>cis-position, cis-gene</td>\n",
       "      <td>1</td>\n",
       "      <td>19-29702725-T-C</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>45355</td>\n",
       "      <td>370</td>\n",
       "      <td>44985</td>\n",
       "      <td>3.038379e-12</td>\n",
       "      <td>0.356081</td>\n",
       "      <td>0.256061</td>\n",
       "      <td>0.456101</td>\n",
       "      <td>0.051030</td>\n",
       "      <td>dominant</td>\n",
       "      <td>Neurology</td>\n",
       "      <td>C19orf12:Q9NSK7:OID20804:v1:Neurology</td>\n",
       "      <td>0.004172</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>135047</th>\n",
       "      <td>C19orf12</td>\n",
       "      <td>C19orf12</td>\n",
       "      <td>Yes</td>\n",
       "      <td>cis-position, cis-gene</td>\n",
       "      <td>1</td>\n",
       "      <td>19-29702725-T-C</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>45355</td>\n",
       "      <td>370</td>\n",
       "      <td>44985</td>\n",
       "      <td>3.038379e-12</td>\n",
       "      <td>0.356081</td>\n",
       "      <td>0.256061</td>\n",
       "      <td>0.456101</td>\n",
       "      <td>0.051030</td>\n",
       "      <td>genotypic</td>\n",
       "      <td>Neurology</td>\n",
       "      <td>C19orf12:Q9NSK7:OID20804:v1:Neurology</td>\n",
       "      <td>0.004172</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>183993</th>\n",
       "      <td>APOL3</td>\n",
       "      <td>C19orf12</td>\n",
       "      <td>No</td>\n",
       "      <td>trans-position, trans-gene</td>\n",
       "      <td>5</td>\n",
       "      <td>22-36149089-A-T</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>45337</td>\n",
       "      <td>44014</td>\n",
       "      <td>1323</td>\n",
       "      <td>1.557476e-09</td>\n",
       "      <td>0.052173</td>\n",
       "      <td>0.035242</td>\n",
       "      <td>0.069105</td>\n",
       "      <td>0.008639</td>\n",
       "      <td>genotypic</td>\n",
       "      <td>Neurology</td>\n",
       "      <td>C19orf12:Q9NSK7:OID20804:v1:Neurology</td>\n",
       "      <td>0.169414</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>200473</th>\n",
       "      <td>PON3</td>\n",
       "      <td>C19orf12</td>\n",
       "      <td>No</td>\n",
       "      <td>trans-position, trans-gene</td>\n",
       "      <td>4</td>\n",
       "      <td>7-95372243-C-T</td>\n",
       "      <td>synonymous_variant</td>\n",
       "      <td>45127</td>\n",
       "      <td>10195</td>\n",
       "      <td>34932</td>\n",
       "      <td>6.037003e-09</td>\n",
       "      <td>-0.064017</td>\n",
       "      <td>-0.085588</td>\n",
       "      <td>-0.042446</td>\n",
       "      <td>0.011005</td>\n",
       "      <td>recessive</td>\n",
       "      <td>Neurology</td>\n",
       "      <td>C19orf12:Q9NSK7:OID20804:v1:Neurology</td>\n",
       "      <td>0.473081</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "            Gene   Protein Match(?Cis) cis_trans_position_1mb_coding_region  \\\n",
       "53685     PNPLA3  C19orf12          No           trans-position, trans-gene   \n",
       "62572     PNPLA3  C19orf12          No           trans-position, trans-gene   \n",
       "71018   C19orf12  C19orf12         Yes               cis-position, cis-gene   \n",
       "71019   C19orf12  C19orf12         Yes               cis-position, cis-gene   \n",
       "72643       PON1  C19orf12          No           trans-position, trans-gene   \n",
       "80111     PNPLA3  C19orf12          No           trans-position, trans-gene   \n",
       "80157       PON1  C19orf12          No           trans-position, trans-gene   \n",
       "80271       PON3  C19orf12          No           trans-position, trans-gene   \n",
       "80510     SAMM50  C19orf12          No           trans-position, trans-gene   \n",
       "80566       PON1  C19orf12          No           trans-position, trans-gene   \n",
       "80584       PON2  C19orf12          No           trans-position, trans-gene   \n",
       "83022       PON3  C19orf12          No           trans-position, trans-gene   \n",
       "86869       PON2  C19orf12          No           trans-position, trans-gene   \n",
       "94714       PON3  C19orf12          No           trans-position, trans-gene   \n",
       "98307   SERPINA1  C19orf12          No           trans-position, trans-gene   \n",
       "100007  SERPINA1  C19orf12          No           trans-position, trans-gene   \n",
       "100978      PON1  C19orf12          No           trans-position, trans-gene   \n",
       "102543      PON3  C19orf12          No           trans-position, trans-gene   \n",
       "107821    SAMM50  C19orf12          No           trans-position, trans-gene   \n",
       "108449      PON1  C19orf12          No           trans-position, trans-gene   \n",
       "109034      PON3  C19orf12          No           trans-position, trans-gene   \n",
       "118162     PARVB  C19orf12          No           trans-position, trans-gene   \n",
       "118676      PON3  C19orf12          No           trans-position, trans-gene   \n",
       "120112    SAMM50  C19orf12          No           trans-position, trans-gene   \n",
       "134913     PARVB  C19orf12          No           trans-position, trans-gene   \n",
       "135046  C19orf12  C19orf12         Yes               cis-position, cis-gene   \n",
       "135047  C19orf12  C19orf12         Yes               cis-position, cis-gene   \n",
       "183993     APOL3  C19orf12          No           trans-position, trans-gene   \n",
       "200473      PON3  C19orf12          No           trans-position, trans-gene   \n",
       "\n",
       "        Num_Proteins_p<1x10-8         Genotype  \\\n",
       "53685                      96  22-43928847-C-G   \n",
       "62572                      96  22-43928847-C-G   \n",
       "71018                       1  19-29702747-T-C   \n",
       "71019                       1  19-29702747-T-C   \n",
       "72643                       9   7-95316772-A-T   \n",
       "80111                      96  22-43928847-C-G   \n",
       "80157                       5   7-95308134-T-C   \n",
       "80271                       8   7-95396288-G-A   \n",
       "80510                      40  22-43972242-A-G   \n",
       "80566                       5   7-95308134-T-C   \n",
       "80584                      11   7-95411704-G-C   \n",
       "83022                       8   7-95396288-G-A   \n",
       "86869                      11   7-95411704-G-C   \n",
       "94714                       4   7-95372243-C-T   \n",
       "98307                     135  14-94378610-C-T   \n",
       "100007                    135  14-94378610-C-T   \n",
       "100978                      9   7-95316772-A-T   \n",
       "102543                      3   7-95394734-A-G   \n",
       "107821                     40  22-43972242-A-G   \n",
       "108449                      9   7-95316772-A-T   \n",
       "109034                      4   7-95372243-C-T   \n",
       "118162                     23  22-43999571-T-C   \n",
       "118676                      3   7-95394734-A-G   \n",
       "120112                     40  22-43972242-A-G   \n",
       "134913                     23  22-43999571-T-C   \n",
       "135046                      1  19-29702725-T-C   \n",
       "135047                      1  19-29702725-T-C   \n",
       "183993                      5  22-36149089-A-T   \n",
       "200473                      4   7-95372243-C-T   \n",
       "\n",
       "                          Most Damaging Effect  nSamples  nCarriers  \\\n",
       "53685                         missense_variant     44579      16517   \n",
       "62572                         missense_variant     44579       2150   \n",
       "71018                         missense_variant     45354         83   \n",
       "71019                         missense_variant     45354         83   \n",
       "72643                         missense_variant     45041      26626   \n",
       "80111                         missense_variant     44579      16517   \n",
       "80157                         missense_variant     45215      22253   \n",
       "80271                       synonymous_variant     45167      20492   \n",
       "80510                         missense_variant     44307      11977   \n",
       "80566                         missense_variant     45215      22253   \n",
       "80584                         missense_variant     45186      18826   \n",
       "83022                       synonymous_variant     45167      20492   \n",
       "86869                         missense_variant     45186      18826   \n",
       "94714                       synonymous_variant     45127      32522   \n",
       "98307                         missense_variant     45346       1835   \n",
       "100007                        missense_variant     45346       1835   \n",
       "100978                        missense_variant     45041       5983   \n",
       "102543  missense_variant&splice_region_variant     44639      31475   \n",
       "107821                        missense_variant     44307      11977   \n",
       "108449                        missense_variant     45041      26626   \n",
       "109034                      synonymous_variant     45127      32522   \n",
       "118162                        missense_variant     44892      27862   \n",
       "118676  missense_variant&splice_region_variant     44639      31475   \n",
       "120112                        missense_variant     44307       1089   \n",
       "134913                        missense_variant     44892       6786   \n",
       "135046                        missense_variant     45355        370   \n",
       "135047                        missense_variant     45355        370   \n",
       "183993                        missense_variant     45337      44014   \n",
       "200473                      synonymous_variant     45127      10195   \n",
       "\n",
       "        nNonCarriers       p-value      beta     lower     upper        se  \\\n",
       "53685          28062  4.489905e-30  0.090495  0.074939  0.106051  0.007937   \n",
       "62572          42429  1.265995e-25  0.226142  0.183809  0.268475  0.021598   \n",
       "71018          45271  2.102760e-22 -1.045686 -1.256074 -0.835299  0.107340   \n",
       "71019          45271  2.102760e-22 -1.045686 -1.256074 -0.835299  0.107340   \n",
       "72643          18415  6.914395e-22  0.064944  0.051712  0.078176  0.006751   \n",
       "80111          28062  8.372943e-20  0.087349  0.068561  0.106137  0.009586   \n",
       "80157          22962  8.570174e-20 -0.083754 -0.101774 -0.065734  0.009194   \n",
       "80271          24675  9.106496e-20 -0.066891 -0.081294 -0.052489  0.007348   \n",
       "80510          32330  1.033495e-19  0.083177  0.065241  0.101112  0.009151   \n",
       "80566          22962  1.062281e-19 -0.065073 -0.079110 -0.051036  0.007161   \n",
       "80584          26360  1.073595e-19 -0.069097 -0.084004 -0.054191  0.007605   \n",
       "83022          24675  3.966466e-19 -0.082595 -0.100700 -0.064491  0.009237   \n",
       "86869          26360  2.591988e-18 -0.081437 -0.099717 -0.063157  0.009327   \n",
       "94714          12605  7.389682e-17 -0.054160 -0.066882 -0.041437  0.006491   \n",
       "98307          43511  2.850509e-16  0.188398  0.143270  0.233526  0.023024   \n",
       "100007         43511  5.262737e-16  0.188841  0.143193  0.234490  0.023290   \n",
       "100978         39058  7.516038e-16  0.109467  0.082862  0.136071  0.013573   \n",
       "102543         13164  1.286384e-15 -0.052099 -0.064865 -0.039333  0.006513   \n",
       "107821         32330  6.380197e-15  0.081493  0.061012  0.101974  0.010449   \n",
       "108449         18415  7.539049e-15  0.072897  0.054526  0.091267  0.009372   \n",
       "109034         12605  8.923837e-15 -0.079542 -0.099642 -0.059441  0.010255   \n",
       "118162         17030  9.704103e-14  0.049686  0.036609  0.062762  0.006672   \n",
       "118676         13164  1.102429e-13 -0.075378 -0.095261 -0.055494  0.010145   \n",
       "120112         43218  1.526849e-13  0.221422  0.162672  0.280172  0.029974   \n",
       "134913         38106  2.972377e-12  0.089952  0.064697  0.115208  0.012885   \n",
       "135046         44985  3.038379e-12  0.356081  0.256061  0.456101  0.051030   \n",
       "135047         44985  3.038379e-12  0.356081  0.256061  0.456101  0.051030   \n",
       "183993          1323  1.557476e-09  0.052173  0.035242  0.069105  0.008639   \n",
       "200473         34932  6.037003e-09 -0.064017 -0.085588 -0.042446  0.011005   \n",
       "\n",
       "            model      panel                            Olink Assay       MAF  \n",
       "53685   genotypic  Neurology  C19orf12:Q9NSK7:OID20804:v1:Neurology  0.209341  \n",
       "62572   recessive  Neurology  C19orf12:Q9NSK7:OID20804:v1:Neurology  0.209341  \n",
       "71018    dominant  Neurology  C19orf12:Q9NSK7:OID20804:v1:Neurology  0.000898  \n",
       "71019   genotypic  Neurology  C19orf12:Q9NSK7:OID20804:v1:Neurology  0.000898  \n",
       "72643   genotypic  Neurology  C19orf12:Q9NSK7:OID20804:v1:Neurology  0.362249  \n",
       "80111    dominant  Neurology  C19orf12:Q9NSK7:OID20804:v1:Neurology  0.209341  \n",
       "80157    dominant  Neurology  C19orf12:Q9NSK7:OID20804:v1:Neurology  0.288201  \n",
       "80271   genotypic  Neurology  C19orf12:Q9NSK7:OID20804:v1:Neurology  0.261920  \n",
       "80510   genotypic  Neurology  C19orf12:Q9NSK7:OID20804:v1:Neurology  0.147319  \n",
       "80566   genotypic  Neurology  C19orf12:Q9NSK7:OID20804:v1:Neurology  0.288201  \n",
       "80584   genotypic  Neurology  C19orf12:Q9NSK7:OID20804:v1:Neurology  0.236988  \n",
       "83022    dominant  Neurology  C19orf12:Q9NSK7:OID20804:v1:Neurology  0.261920  \n",
       "86869    dominant  Neurology  C19orf12:Q9NSK7:OID20804:v1:Neurology  0.236988  \n",
       "94714   genotypic  Neurology  C19orf12:Q9NSK7:OID20804:v1:Neurology  0.473081  \n",
       "98307   genotypic  Neurology  C19orf12:Q9NSK7:OID20804:v1:Neurology  0.020598  \n",
       "100007   dominant  Neurology  C19orf12:Q9NSK7:OID20804:v1:Neurology  0.020598  \n",
       "100978  recessive  Neurology  C19orf12:Q9NSK7:OID20804:v1:Neurology  0.362249  \n",
       "102543  genotypic  Neurology  C19orf12:Q9NSK7:OID20804:v1:Neurology  0.460052  \n",
       "107821   dominant  Neurology  C19orf12:Q9NSK7:OID20804:v1:Neurology  0.147319  \n",
       "108449   dominant  Neurology  C19orf12:Q9NSK7:OID20804:v1:Neurology  0.362249  \n",
       "109034   dominant  Neurology  C19orf12:Q9NSK7:OID20804:v1:Neurology  0.473081  \n",
       "118162  genotypic  Neurology  C19orf12:Q9NSK7:OID20804:v1:Neurology  0.385625  \n",
       "118676   dominant  Neurology  C19orf12:Q9NSK7:OID20804:v1:Neurology  0.460052  \n",
       "120112  recessive  Neurology  C19orf12:Q9NSK7:OID20804:v1:Neurology  0.147319  \n",
       "134913  recessive  Neurology  C19orf12:Q9NSK7:OID20804:v1:Neurology  0.385625  \n",
       "135046   dominant  Neurology  C19orf12:Q9NSK7:OID20804:v1:Neurology  0.004172  \n",
       "135047  genotypic  Neurology  C19orf12:Q9NSK7:OID20804:v1:Neurology  0.004172  \n",
       "183993  genotypic  Neurology  C19orf12:Q9NSK7:OID20804:v1:Neurology  0.169414  \n",
       "200473  recessive  Neurology  C19orf12:Q9NSK7:OID20804:v1:Neurology  0.473081  "
      ]
     },
     "execution_count": 41,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "raw[raw[\"Protein\"] == \"C19orf12\"] "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 14,
   "id": "2ea82aca-dfbc-432f-bffe-c8b480a75989",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "ACE2\n",
      "Number of variants before clump: 53\n",
      "Number of variants after clump: 27\n",
      "ACP5\n",
      "Number of variants before clump: 71\n",
      "Number of variants after clump: 35\n",
      "ACY1\n",
      "Number of variants before clump: 55\n",
      "Number of variants after clump: 34\n",
      "ADA2\n",
      "Number of variants before clump: 258\n",
      "Number of variants after clump: 87\n",
      "ADAM8\n",
      "Number of variants before clump: 140\n",
      "Number of variants after clump: 63\n",
      "ADGRE2\n",
      "Number of variants before clump: 71\n",
      "Number of variants after clump: 28\n",
      "ADGRE5\n",
      "Number of variants before clump: 85\n",
      "Number of variants after clump: 28\n",
      "ADGRG1\n",
      "Number of variants before clump: 38\n",
      "Number of variants after clump: 22\n",
      "ADM\n",
      "Number of variants before clump: 74\n",
      "Number of variants after clump: 37\n",
      "AGR2\n",
      "Number of variants before clump: 41\n",
      "Number of variants after clump: 21\n",
      "ALCAM\n",
      "Number of variants before clump: 52\n",
      "Number of variants after clump: 28\n",
      "ALPP\n",
      "Number of variants before clump: 122\n",
      "Number of variants after clump: 49\n",
      "AMIGO2\n",
      "Number of variants before clump: 64\n",
      "Number of variants after clump: 32\n",
      "ANG\n",
      "Number of variants before clump: 20\n",
      "Number of variants after clump: 12\n",
      "ANGPT2\n",
      "Number of variants before clump: 38\n",
      "Number of variants after clump: 16\n",
      "ANGPTL4\n",
      "Number of variants before clump: 27\n",
      "Number of variants after clump: 17\n",
      "ANPEP\n",
      "Number of variants before clump: 54\n",
      "Number of variants after clump: 27\n",
      "ANXA10\n",
      "Number of variants before clump: 4\n",
      "Number of variants after clump: 2\n",
      "APEX1\n",
      "Number of variants before clump: 27\n",
      "Number of variants after clump: 16\n",
      "AREG\n",
      "Number of variants before clump: 42\n",
      "Number of variants after clump: 19\n",
      "ARSA\n",
      "Number of variants before clump: 90\n",
      "Number of variants after clump: 53\n",
      "ATOX1\n",
      "Number of variants before clump: 15\n",
      "Number of variants after clump: 7\n",
      "B4GALT1\n",
      "Number of variants before clump: 46\n",
      "Number of variants after clump: 13\n",
      "BCAM\n",
      "Number of variants before clump: 139\n",
      "Number of variants after clump: 52\n",
      "BMP6\n",
      "Number of variants before clump: 60\n",
      "Number of variants after clump: 26\n",
      "BST2\n",
      "Number of variants before clump: 80\n",
      "Number of variants after clump: 42\n",
      "BTN2A1\n",
      "Number of variants before clump: 520\n",
      "Number of variants after clump: 116\n",
      "BTN3A2\n",
      "Number of variants before clump: 768\n",
      "Number of variants after clump: 174\n",
      "C1QA\n",
      "Number of variants before clump: 116\n",
      "Number of variants after clump: 52\n",
      "CANT1\n",
      "Number of variants before clump: 43\n",
      "Number of variants after clump: 19\n",
      "CCL14\n",
      "Number of variants before clump: 32\n",
      "Number of variants after clump: 14\n",
      "CCL15\n",
      "Number of variants before clump: 113\n",
      "Number of variants after clump: 37\n",
      "CCL21\n",
      "Number of variants before clump: 84\n",
      "Number of variants after clump: 34\n",
      "CCL22\n",
      "Number of variants before clump: 31\n",
      "Number of variants after clump: 15\n",
      "CCL3\n",
      "Number of variants before clump: 55\n",
      "Number of variants after clump: 22\n",
      "CD163\n",
      "Number of variants before clump: 105\n",
      "Number of variants after clump: 39\n",
      "CD1C\n",
      "Number of variants before clump: 457\n",
      "Number of variants after clump: 142\n",
      "CD200\n",
      "Number of variants before clump: 68\n",
      "Number of variants after clump: 22\n",
      "CD200R1\n",
      "Number of variants before clump: 101\n",
      "Number of variants after clump: 50\n",
      "CD22\n",
      "Number of variants before clump: 226\n",
      "Number of variants after clump: 80\n",
      "CD27\n",
      "Number of variants before clump: 48\n",
      "Number of variants after clump: 26\n",
      "CD274\n",
      "Number of variants before clump: 11\n",
      "Number of variants after clump: 10\n",
      "CD28\n",
      "Number of variants before clump: 62\n",
      "Number of variants after clump: 24\n",
      "CD300E\n",
      "Number of variants before clump: 63\n",
      "Number of variants after clump: 32\n",
      "CD48\n",
      "Number of variants before clump: 64\n",
      "Number of variants after clump: 28\n",
      "CD5\n",
      "Number of variants before clump: 112\n",
      "Number of variants after clump: 51\n",
      "CD6\n",
      "Number of variants before clump: 86\n",
      "Number of variants after clump: 43\n",
      "CD70\n",
      "Number of variants before clump: 67\n",
      "Number of variants after clump: 24\n",
      "CD74\n",
      "Number of variants before clump: 386\n",
      "Number of variants after clump: 121\n",
      "CD79B\n",
      "Number of variants before clump: 117\n",
      "Number of variants after clump: 30\n",
      "CD83\n",
      "Number of variants before clump: 103\n",
      "Number of variants after clump: 39\n",
      "CDCP1\n",
      "Number of variants before clump: 71\n",
      "Number of variants after clump: 38\n",
      "CDH2\n",
      "Number of variants before clump: 40\n",
      "Number of variants after clump: 26\n",
      "CDH3\n",
      "Number of variants before clump: 37\n",
      "Number of variants after clump: 17\n",
      "CDH6\n",
      "Number of variants before clump: 11\n",
      "Number of variants after clump: 8\n",
      "CDHR2\n",
      "Number of variants before clump: 43\n",
      "Number of variants after clump: 17\n",
      "CEACAM1\n",
      "Number of variants before clump: 138\n",
      "Number of variants after clump: 49\n",
      "CEACAM21\n",
      "Number of variants before clump: 146\n",
      "Number of variants after clump: 80\n",
      "CEACAM5\n",
      "Number of variants before clump: 112\n",
      "Number of variants after clump: 31\n",
      "CHI3L1\n",
      "Number of variants before clump: 37\n",
      "Number of variants after clump: 13\n",
      "CHRDL2\n",
      "Number of variants before clump: 31\n",
      "Number of variants after clump: 11\n",
      "CKAP4\n",
      "Number of variants before clump: 20\n",
      "Number of variants after clump: 13\n",
      "CLEC4C\n",
      "Number of variants before clump: 69\n",
      "Number of variants after clump: 37\n",
      "CLEC6A\n",
      "Number of variants before clump: 34\n",
      "Number of variants after clump: 18\n",
      "CLSTN2\n",
      "Number of variants before clump: 20\n",
      "Number of variants after clump: 18\n",
      "CNDP1\n",
      "Number of variants before clump: 53\n",
      "Number of variants after clump: 32\n",
      "CNTN5\n",
      "Number of variants before clump: 28\n",
      "Number of variants after clump: 14\n",
      "COL18A1\n",
      "Number of variants before clump: 42\n",
      "Number of variants after clump: 24\n",
      "COL4A1\n",
      "Number of variants before clump: 73\n",
      "Number of variants after clump: 31\n",
      "CR2\n",
      "Number of variants before clump: 167\n",
      "Number of variants after clump: 59\n",
      "CRELD2\n",
      "Number of variants before clump: 113\n",
      "Number of variants after clump: 46\n",
      "CRH\n",
      "Number of variants before clump: 89\n",
      "Number of variants after clump: 37\n",
      "CRIM1\n",
      "Number of variants before clump: 27\n",
      "Number of variants after clump: 15\n",
      "CRLF1\n",
      "Number of variants before clump: 49\n",
      "Number of variants after clump: 14\n",
      "CRTAM\n",
      "Number of variants before clump: 168\n",
      "Number of variants after clump: 62\n",
      "CSF1\n",
      "Number of variants before clump: 87\n",
      "Number of variants after clump: 46\n",
      "CST3\n",
      "Number of variants before clump: 18\n",
      "Number of variants after clump: 10\n",
      "CTSD\n",
      "Number of variants before clump: 84\n",
      "Number of variants after clump: 28\n",
      "CTSL\n",
      "Number of variants before clump: 99\n",
      "Number of variants after clump: 39\n",
      "CXADR\n",
      "Number of variants before clump: 74\n",
      "Number of variants after clump: 17\n",
      "CXCL11\n",
      "Number of variants before clump: 66\n",
      "Number of variants after clump: 33\n",
      "CXCL13\n",
      "Number of variants before clump: 18\n",
      "Number of variants after clump: 13\n",
      "CXCL17\n",
      "Number of variants before clump: 34\n",
      "Number of variants after clump: 14\n",
      "CXCL9\n",
      "Number of variants before clump: 141\n",
      "Number of variants after clump: 62\n",
      "DCTPP1\n",
      "Number of variants before clump: 26\n",
      "Number of variants after clump: 14\n",
      "DDR1\n",
      "Number of variants before clump: 400\n",
      "Number of variants after clump: 190\n",
      "DKK4\n",
      "Number of variants before clump: 61\n",
      "Number of variants after clump: 23\n",
      "DLL1\n",
      "Number of variants before clump: 79\n",
      "Number of variants after clump: 33\n",
      "DPP10\n",
      "Number of variants before clump: 12\n",
      "Number of variants after clump: 10\n",
      "DPP4\n",
      "Number of variants before clump: 42\n",
      "Number of variants after clump: 29\n",
      "DSC2\n",
      "Number of variants before clump: 55\n",
      "Number of variants after clump: 21\n",
      "EBI3_IL27\n",
      "Number of variants before clump: 158\n",
      "Number of variants after clump: 45\n",
      "EDA2R\n",
      "Number of variants before clump: 27\n",
      "Number of variants after clump: 21\n",
      "EFEMP1\n",
      "Number of variants before clump: 18\n",
      "Number of variants after clump: 11\n",
      "EFNA4\n",
      "Number of variants before clump: 22\n",
      "Number of variants after clump: 12\n",
      "EGFL7\n",
      "Number of variants before clump: 98\n",
      "Number of variants after clump: 43\n",
      "ELOA\n",
      "Number of variants before clump: 20\n",
      "Number of variants after clump: 6\n",
      "ENG\n",
      "Number of variants before clump: 80\n",
      "Number of variants after clump: 34\n",
      "ENPP2\n",
      "Number of variants before clump: 53\n",
      "Number of variants after clump: 23\n",
      "ENPP7\n",
      "Number of variants before clump: 29\n",
      "Number of variants after clump: 19\n",
      "ENTPD2\n",
      "Number of variants before clump: 111\n",
      "Number of variants after clump: 58\n",
      "EPHA2\n",
      "Number of variants before clump: 162\n",
      "Number of variants after clump: 68\n",
      "EPS8L2\n",
      "Number of variants before clump: 97\n",
      "Number of variants after clump: 29\n",
      "ERBB2\n",
      "Number of variants before clump: 77\n",
      "Number of variants after clump: 42\n",
      "ESM1\n",
      "Number of variants before clump: 40\n",
      "Number of variants after clump: 19\n",
      "EZR\n",
      "Number of variants before clump: 19\n",
      "Number of variants after clump: 8\n",
      "F9\n",
      "Number of variants before clump: 25\n",
      "Number of variants after clump: 8\n",
      "FABP1\n",
      "Number of variants before clump: 10\n",
      "Number of variants after clump: 6\n",
      "FAP\n",
      "Number of variants before clump: 276\n",
      "Number of variants after clump: 103\n",
      "FCER2\n",
      "Number of variants before clump: 169\n",
      "Number of variants after clump: 68\n",
      "FCN2\n",
      "Number of variants before clump: 80\n",
      "Number of variants after clump: 30\n",
      "FCRL1\n",
      "Number of variants before clump: 109\n",
      "Number of variants after clump: 51\n",
      "FCRL2\n",
      "Number of variants before clump: 265\n",
      "Number of variants after clump: 129\n",
      "FCRL3\n",
      "Number of variants before clump: 51\n",
      "Number of variants after clump: 35\n",
      "FCRL5\n",
      "Number of variants before clump: 105\n",
      "Number of variants after clump: 52\n",
      "FCRLB\n",
      "Number of variants before clump: 60\n",
      "Number of variants after clump: 36\n",
      "FGFR2\n",
      "Number of variants before clump: 35\n",
      "Number of variants after clump: 13\n",
      "FLT3LG\n",
      "Number of variants before clump: 58\n",
      "Number of variants after clump: 27\n",
      "FRZB\n",
      "Number of variants before clump: 238\n",
      "Number of variants after clump: 97\n",
      "FSTL3\n",
      "Number of variants before clump: 12\n",
      "Number of variants after clump: 7\n",
      "FUS\n",
      "Number of variants before clump: 3\n",
      "Number of variants after clump: 2\n",
      "FUT3_FUT5\n",
      "Number of variants before clump: 107\n",
      "Number of variants after clump: 35\n",
      "GAL\n",
      "Number of variants before clump: 70\n",
      "Number of variants after clump: 33\n",
      "GALNT10\n",
      "Number of variants before clump: 58\n",
      "Number of variants after clump: 28\n",
      "GALNT3\n",
      "Number of variants before clump: 96\n",
      "Number of variants after clump: 37\n",
      "GAS6\n",
      "Number of variants before clump: 72\n",
      "Number of variants after clump: 34\n",
      "GCNT1\n",
      "Number of variants before clump: 75\n",
      "Number of variants after clump: 26\n",
      "GDF15\n",
      "Number of variants before clump: 26\n",
      "Number of variants after clump: 13\n",
      "GDNF\n",
      "Number of variants before clump: 7\n",
      "Number of variants after clump: 5\n",
      "GFAP\n",
      "Number of variants before clump: 27\n",
      "Number of variants after clump: 13\n",
      "GGH\n",
      "Number of variants before clump: 76\n",
      "Number of variants after clump: 34\n",
      "GGT1\n",
      "Number of variants before clump: 59\n",
      "Number of variants after clump: 18\n",
      "GOLM2\n",
      "Number of variants before clump: 181\n",
      "Number of variants after clump: 73\n",
      "GP2\n",
      "Number of variants before clump: 112\n",
      "Number of variants after clump: 31\n",
      "GRN\n",
      "Number of variants before clump: 144\n",
      "Number of variants after clump: 57\n",
      "GZMA\n",
      "Number of variants before clump: 511\n",
      "Number of variants after clump: 144\n",
      "HAVCR1\n",
      "Number of variants before clump: 37\n",
      "Number of variants after clump: 21\n",
      "HAVCR2\n",
      "Number of variants before clump: 36\n",
      "Number of variants after clump: 19\n",
      "HGF\n",
      "Number of variants before clump: 27\n",
      "Number of variants after clump: 12\n",
      "HS6ST1\n",
      "Number of variants before clump: 77\n",
      "Number of variants after clump: 45\n",
      "HSPB6\n",
      "Number of variants before clump: 24\n",
      "Number of variants after clump: 10\n",
      "HYOU1\n",
      "Number of variants before clump: 76\n",
      "Number of variants after clump: 28\n",
      "ICAM1\n",
      "Number of variants before clump: 132\n",
      "Number of variants after clump: 55\n",
      "ICAM3\n",
      "Number of variants before clump: 438\n",
      "Number of variants after clump: 136\n",
      "ICAM5\n",
      "Number of variants before clump: 165\n",
      "Number of variants after clump: 80\n",
      "ICOSLG\n",
      "Number of variants before clump: 52\n",
      "Number of variants after clump: 23\n",
      "IDS\n",
      "Number of variants before clump: 21\n",
      "Number of variants after clump: 12\n",
      "IFNL1\n",
      "Number of variants before clump: 25\n",
      "Number of variants after clump: 22\n",
      "IFNLR1\n",
      "Number of variants before clump: 15\n",
      "Number of variants after clump: 8\n",
      "IGFBP3\n",
      "Number of variants before clump: 81\n",
      "Number of variants after clump: 43\n",
      "IGFBP7\n",
      "Number of variants before clump: 55\n",
      "Number of variants after clump: 34\n",
      "IGFBPL1\n",
      "Number of variants before clump: 26\n",
      "Number of variants after clump: 13\n",
      "IGSF3\n",
      "Number of variants before clump: 29\n",
      "Number of variants after clump: 18\n",
      "IL10\n",
      "Number of variants before clump: 86\n",
      "Number of variants after clump: 43\n",
      "IL10RB\n",
      "Number of variants before clump: 43\n",
      "Number of variants after clump: 20\n",
      "IL12A_IL12B\n",
      "Number of variants before clump: 264\n",
      "Number of variants after clump: 80\n",
      "IL12B\n",
      "Number of variants before clump: 258\n",
      "Number of variants after clump: 80\n",
      "IL12RB1\n",
      "Number of variants before clump: 104\n",
      "Number of variants after clump: 36\n",
      "IL15RA\n",
      "Number of variants before clump: 148\n",
      "Number of variants after clump: 59\n",
      "IL18BP\n",
      "Number of variants before clump: 122\n",
      "Number of variants after clump: 54\n",
      "IL18R1\n",
      "Number of variants before clump: 61\n",
      "Number of variants after clump: 29\n",
      "IL1R1\n",
      "Number of variants before clump: 72\n",
      "Number of variants after clump: 31\n",
      "IL2RA\n",
      "Number of variants before clump: 251\n",
      "Number of variants after clump: 79\n",
      "IL4R\n",
      "Number of variants before clump: 43\n",
      "Number of variants after clump: 21\n",
      "IL5RA\n",
      "Number of variants before clump: 315\n",
      "Number of variants after clump: 108\n",
      "IL6R\n",
      "Number of variants before clump: 92\n",
      "Number of variants after clump: 57\n",
      "IL6ST\n",
      "Number of variants before clump: 64\n",
      "Number of variants after clump: 26\n",
      "ITGA5\n",
      "Number of variants before clump: 81\n",
      "Number of variants after clump: 29\n",
      "ITGB2\n",
      "Number of variants before clump: 79\n",
      "Number of variants after clump: 29\n",
      "ITGB7\n",
      "Number of variants before clump: 215\n",
      "Number of variants after clump: 85\n",
      "ITM2A\n",
      "Number of variants before clump: 49\n",
      "Number of variants after clump: 20\n",
      "JCHAIN\n",
      "Number of variants before clump: 91\n",
      "Number of variants after clump: 38\n",
      "KAZALD1\n",
      "Number of variants before clump: 105\n",
      "Number of variants after clump: 57\n",
      "KITLG\n",
      "Number of variants before clump: 123\n",
      "Number of variants after clump: 42\n",
      "KLRB1\n",
      "Number of variants before clump: 106\n",
      "Number of variants after clump: 51\n",
      "KRT18\n",
      "Number of variants before clump: 25\n",
      "Number of variants after clump: 8\n",
      "KRT19\n",
      "Number of variants before clump: 17\n",
      "Number of variants after clump: 7\n",
      "L1CAM\n",
      "Number of variants before clump: 122\n",
      "Number of variants after clump: 64\n",
      "LAG3\n",
      "Number of variants before clump: 99\n",
      "Number of variants after clump: 42\n",
      "LAIR1\n",
      "Number of variants before clump: 39\n",
      "Number of variants after clump: 21\n",
      "LAIR2\n",
      "Number of variants before clump: 97\n",
      "Number of variants after clump: 42\n",
      "LAMP3\n",
      "Number of variants before clump: 67\n",
      "Number of variants after clump: 34\n",
      "LEFTY2\n",
      "Number of variants before clump: 67\n",
      "Number of variants after clump: 32\n",
      "LGALS9\n",
      "Number of variants before clump: 61\n",
      "Number of variants after clump: 21\n",
      "LGMN\n",
      "Number of variants before clump: 79\n",
      "Number of variants after clump: 31\n",
      "LIFR\n",
      "Number of variants before clump: 72\n",
      "Number of variants after clump: 30\n",
      "LILRB1\n",
      "Number of variants before clump: 483\n",
      "Number of variants after clump: 195\n",
      "LILRB4\n",
      "Number of variants before clump: 697\n",
      "Number of variants after clump: 186\n",
      "LRIG1\n",
      "Number of variants before clump: 42\n",
      "Number of variants after clump: 27\n",
      "LSP1\n",
      "Number of variants before clump: 44\n",
      "Number of variants after clump: 7\n",
      "LTA\n",
      "Number of variants before clump: 632\n",
      "Number of variants after clump: 252\n",
      "LTBP2\n",
      "Number of variants before clump: 69\n",
      "Number of variants after clump: 24\n",
      "LTBR\n",
      "Number of variants before clump: 138\n",
      "Number of variants after clump: 64\n",
      "LY75\n",
      "Number of variants before clump: 122\n",
      "Number of variants after clump: 60\n",
      "LY9\n",
      "Number of variants before clump: 107\n",
      "Number of variants after clump: 40\n",
      "LYAR\n",
      "Number of variants before clump: 2\n",
      "Number of variants after clump: 2\n",
      "LYPD8\n",
      "Number of variants before clump: 73\n",
      "Number of variants after clump: 31\n",
      "MARCO\n",
      "Number of variants before clump: 52\n",
      "Number of variants after clump: 25\n",
      "MDK\n",
      "Number of variants before clump: 46\n",
      "Number of variants after clump: 20\n",
      "MERTK\n",
      "Number of variants before clump: 129\n",
      "Number of variants after clump: 57\n",
      "MILR1\n",
      "Number of variants before clump: 105\n",
      "Number of variants after clump: 52\n",
      "MME\n",
      "Number of variants before clump: 183\n",
      "Number of variants after clump: 75\n",
      "MMP12\n",
      "Number of variants before clump: 74\n",
      "Number of variants after clump: 39\n",
      "MSLN\n",
      "Number of variants before clump: 161\n",
      "Number of variants after clump: 72\n",
      "MSR1\n",
      "Number of variants before clump: 179\n",
      "Number of variants after clump: 67\n",
      "MUC13\n",
      "Number of variants before clump: 63\n",
      "Number of variants after clump: 18\n",
      "MZB1\n",
      "Number of variants before clump: 57\n",
      "Number of variants after clump: 18\n",
      "NBN\n",
      "Number of variants before clump: 4\n",
      "Number of variants after clump: 4\n",
      "NFASC\n",
      "Number of variants before clump: 44\n",
      "Number of variants after clump: 24\n",
      "NFKBIE\n",
      "Number of variants before clump: 10\n",
      "Number of variants after clump: 5\n",
      "NME3\n",
      "Number of variants before clump: 132\n",
      "Number of variants after clump: 34\n",
      "NMNAT1\n",
      "Number of variants before clump: 41\n",
      "Number of variants after clump: 19\n",
      "NOMO1\n",
      "Number of variants before clump: 28\n",
      "Number of variants after clump: 15\n",
      "NOS1\n",
      "Number of variants before clump: 10\n",
      "Number of variants after clump: 5\n",
      "NOS3\n",
      "Number of variants before clump: 11\n",
      "Number of variants after clump: 10\n",
      "NOTCH1\n",
      "Number of variants before clump: 60\n",
      "Number of variants after clump: 21\n",
      "NOTCH3\n",
      "Number of variants before clump: 50\n",
      "Number of variants after clump: 23\n",
      "NRCAM\n",
      "Number of variants before clump: 44\n",
      "Number of variants after clump: 20\n",
      "NRP1\n",
      "Number of variants before clump: 58\n",
      "Number of variants after clump: 22\n",
      "NRP2\n",
      "Number of variants before clump: 79\n",
      "Number of variants after clump: 39\n",
      "NT5E\n",
      "Number of variants before clump: 66\n",
      "Number of variants after clump: 27\n",
      "PARP1\n",
      "Number of variants before clump: 14\n",
      "Number of variants after clump: 5\n",
      "PCDH17\n",
      "Number of variants before clump: 78\n",
      "Number of variants after clump: 26\n",
      "PCOLCE\n",
      "Number of variants before clump: 22\n",
      "Number of variants after clump: 3\n",
      "PDCD1\n",
      "Number of variants before clump: 78\n",
      "Number of variants after clump: 30\n",
      "PDGFC\n",
      "Number of variants before clump: 32\n",
      "Number of variants after clump: 12\n",
      "PDGFRA\n",
      "Number of variants before clump: 56\n",
      "Number of variants after clump: 36\n",
      "PGF\n",
      "Number of variants before clump: 22\n",
      "Number of variants after clump: 8\n",
      "PIGR\n",
      "Number of variants before clump: 45\n",
      "Number of variants after clump: 14\n",
      "PIK3AP1\n",
      "Number of variants before clump: 15\n",
      "Number of variants after clump: 8\n",
      "PLA2G15\n",
      "Number of variants before clump: 337\n",
      "Number of variants after clump: 112\n",
      "PLAU\n",
      "Number of variants before clump: 112\n",
      "Number of variants after clump: 46\n",
      "PLAUR\n",
      "Number of variants before clump: 105\n",
      "Number of variants after clump: 39\n",
      "PLXNB2\n",
      "Number of variants before clump: 192\n",
      "Number of variants after clump: 100\n",
      "POLR2F\n",
      "Number of variants before clump: 6\n",
      "Number of variants after clump: 4\n",
      "PON2\n",
      "Number of variants before clump: 41\n",
      "Number of variants after clump: 27\n",
      "PRELP\n",
      "Number of variants before clump: 14\n",
      "Number of variants after clump: 9\n",
      "PROC\n",
      "Number of variants before clump: 161\n",
      "Number of variants after clump: 57\n",
      "PRSS8\n",
      "Number of variants before clump: 89\n",
      "Number of variants after clump: 35\n",
      "PRTG\n",
      "Number of variants before clump: 102\n",
      "Number of variants after clump: 52\n",
      "PSIP1\n",
      "Number of variants before clump: 19\n",
      "Number of variants after clump: 10\n",
      "PTGDS\n",
      "Number of variants before clump: 47\n",
      "Number of variants after clump: 18\n",
      "PTPRS\n",
      "Number of variants before clump: 38\n",
      "Number of variants after clump: 18\n",
      "PTS\n",
      "Number of variants before clump: 54\n",
      "Number of variants after clump: 13\n",
      "PVR\n",
      "Number of variants before clump: 78\n",
      "Number of variants after clump: 48\n",
      "QPCT\n",
      "Number of variants before clump: 111\n",
      "Number of variants after clump: 58\n",
      "RARRES2\n",
      "Number of variants before clump: 53\n",
      "Number of variants after clump: 25\n",
      "RASSF2\n",
      "Number of variants before clump: 3\n",
      "Number of variants after clump: 2\n",
      "RBP2\n",
      "Number of variants before clump: 60\n",
      "Number of variants after clump: 30\n",
      "RBP5\n",
      "Number of variants before clump: 54\n",
      "Number of variants after clump: 26\n",
      "REG3A\n",
      "Number of variants before clump: 60\n",
      "Number of variants after clump: 37\n",
      "REG4\n",
      "Number of variants before clump: 21\n",
      "Number of variants after clump: 14\n",
      "RELT\n",
      "Number of variants before clump: 27\n",
      "Number of variants after clump: 10\n",
      "RGMA\n",
      "Number of variants before clump: 95\n",
      "Number of variants after clump: 36\n",
      "RNASET2\n",
      "Number of variants before clump: 346\n",
      "Number of variants after clump: 96\n",
      "ROBO1\n",
      "Number of variants before clump: 104\n",
      "Number of variants after clump: 61\n",
      "ROBO2\n",
      "Number of variants before clump: 22\n",
      "Number of variants after clump: 17\n",
      "ROR1\n",
      "Number of variants before clump: 28\n",
      "Number of variants after clump: 17\n",
      "RRM2\n",
      "Number of variants before clump: 20\n",
      "Number of variants after clump: 13\n",
      "RTBDN\n",
      "Number of variants before clump: 38\n",
      "Number of variants after clump: 10\n",
      "SCARF2\n",
      "Number of variants before clump: 52\n",
      "Number of variants after clump: 18\n",
      "SDC1\n",
      "Number of variants before clump: 61\n",
      "Number of variants after clump: 26\n",
      "SDC4\n",
      "Number of variants before clump: 80\n",
      "Number of variants after clump: 33\n",
      "SEMA4D\n",
      "Number of variants before clump: 18\n",
      "Number of variants after clump: 11\n",
      "SEMA7A\n",
      "Number of variants before clump: 65\n",
      "Number of variants after clump: 29\n",
      "SERPINA9\n",
      "Number of variants before clump: 49\n",
      "Number of variants after clump: 27\n",
      "SEZ6L2\n",
      "Number of variants before clump: 180\n",
      "Number of variants after clump: 78\n",
      "SFRP1\n",
      "Number of variants before clump: 20\n",
      "Number of variants after clump: 15\n",
      "SFTPA2\n",
      "Number of variants before clump: 43\n",
      "Number of variants after clump: 18\n",
      "SFTPD\n",
      "Number of variants before clump: 333\n",
      "Number of variants after clump: 103\n",
      "SH2D1A\n",
      "Number of variants before clump: 7\n",
      "Number of variants after clump: 4\n",
      "SIGLEC1\n",
      "Number of variants before clump: 155\n",
      "Number of variants after clump: 81\n",
      "SIGLEC10\n",
      "Number of variants before clump: 121\n",
      "Number of variants after clump: 63\n",
      "SIGLEC6\n",
      "Number of variants before clump: 311\n",
      "Number of variants after clump: 112\n",
      "SIGLEC7\n",
      "Number of variants before clump: 27\n",
      "Number of variants after clump: 14\n",
      "SIRPB1\n",
      "Number of variants before clump: 65\n",
      "Number of variants after clump: 30\n",
      "SLAMF1\n",
      "Number of variants before clump: 33\n",
      "Number of variants after clump: 16\n",
      "SLAMF6\n",
      "Number of variants before clump: 5\n",
      "Number of variants after clump: 4\n",
      "SLAMF7\n",
      "Number of variants before clump: 236\n",
      "Number of variants after clump: 87\n",
      "SLAMF8\n",
      "Number of variants before clump: 43\n",
      "Number of variants after clump: 24\n",
      "SLITRK2\n",
      "Number of variants before clump: 18\n",
      "Number of variants after clump: 14\n",
      "SMOC1\n",
      "Number of variants before clump: 32\n",
      "Number of variants after clump: 21\n",
      "SPARCL1\n",
      "Number of variants before clump: 105\n",
      "Number of variants after clump: 32\n",
      "SPINK1\n",
      "Number of variants before clump: 22\n",
      "Number of variants after clump: 13\n",
      "SPINK4\n",
      "Number of variants before clump: 84\n",
      "Number of variants after clump: 47\n",
      "SPINT1\n",
      "Number of variants before clump: 169\n",
      "Number of variants after clump: 69\n",
      "SPON1\n",
      "Number of variants before clump: 111\n",
      "Number of variants after clump: 54\n",
      "SPON2\n",
      "Number of variants before clump: 48\n",
      "Number of variants after clump: 13\n",
      "SPP1\n",
      "Number of variants before clump: 33\n",
      "Number of variants after clump: 22\n",
      "SRP14\n",
      "Number of variants before clump: 12\n",
      "Number of variants after clump: 6\n",
      "ST6GAL1\n",
      "Number of variants before clump: 16\n",
      "Number of variants after clump: 13\n",
      "STC1\n",
      "Number of variants before clump: 16\n",
      "Number of variants after clump: 4\n",
      "STC2\n",
      "Number of variants before clump: 52\n",
      "Number of variants after clump: 17\n",
      "SULT2A1\n",
      "Number of variants before clump: 13\n",
      "Number of variants after clump: 5\n",
      "TAFA5\n",
      "Number of variants before clump: 8\n",
      "Number of variants after clump: 3\n",
      "TCL1A\n",
      "Number of variants before clump: 82\n",
      "Number of variants after clump: 39\n",
      "TFF1\n",
      "Number of variants before clump: 38\n",
      "Number of variants after clump: 12\n",
      "TFF2\n",
      "Number of variants before clump: 56\n",
      "Number of variants after clump: 17\n",
      "TFF3\n",
      "Number of variants before clump: 336\n",
      "Number of variants after clump: 143\n",
      "TFPI2\n",
      "Number of variants before clump: 64\n",
      "Number of variants after clump: 20\n",
      "TGFBI\n",
      "Number of variants before clump: 140\n",
      "Number of variants after clump: 61\n",
      "THBS2\n",
      "Number of variants before clump: 34\n",
      "Number of variants after clump: 22\n",
      "TIMP1\n",
      "Number of variants before clump: 46\n",
      "Number of variants after clump: 20\n",
      "TNF\n",
      "Number of variants before clump: 83\n",
      "Number of variants after clump: 47\n",
      "TNFRSF10A\n",
      "Number of variants before clump: 51\n",
      "Number of variants after clump: 31\n",
      "TNFRSF10B\n",
      "Number of variants before clump: 19\n",
      "Number of variants after clump: 13\n",
      "TNFRSF11B\n",
      "Number of variants before clump: 34\n",
      "Number of variants after clump: 20\n",
      "TNFRSF13B\n",
      "Number of variants before clump: 57\n",
      "Number of variants after clump: 32\n",
      "TNFRSF13C\n",
      "Number of variants before clump: 107\n",
      "Number of variants after clump: 45\n",
      "TNFRSF14\n",
      "Number of variants before clump: 37\n",
      "Number of variants after clump: 14\n",
      "TNFRSF1B\n",
      "Number of variants before clump: 108\n",
      "Number of variants after clump: 57\n",
      "TNFRSF21\n",
      "Number of variants before clump: 110\n",
      "Number of variants after clump: 49\n",
      "TNFRSF4\n",
      "Number of variants before clump: 313\n",
      "Number of variants after clump: 82\n",
      "TNFRSF6B\n",
      "Number of variants before clump: 108\n",
      "Number of variants after clump: 33\n",
      "TNFRSF8\n",
      "Number of variants before clump: 244\n",
      "Number of variants after clump: 78\n",
      "TNFRSF9\n",
      "Number of variants before clump: 209\n",
      "Number of variants after clump: 74\n",
      "TNFSF13\n",
      "Number of variants before clump: 102\n",
      "Number of variants after clump: 37\n",
      "TNFSF13B\n",
      "Number of variants before clump: 181\n",
      "Number of variants after clump: 84\n",
      "TNR\n",
      "Number of variants before clump: 115\n",
      "Number of variants after clump: 45\n",
      "TPP1\n",
      "Number of variants before clump: 97\n",
      "Number of variants after clump: 43\n",
      "TREM2\n",
      "Number of variants before clump: 109\n",
      "Number of variants after clump: 51\n",
      "TREML2\n",
      "Number of variants before clump: 42\n",
      "Number of variants after clump: 22\n",
      "TSPAN1\n",
      "Number of variants before clump: 22\n",
      "Number of variants after clump: 8\n",
      "TXNDC15\n",
      "Number of variants before clump: 269\n",
      "Number of variants after clump: 123\n",
      "VAMP5\n",
      "Number of variants before clump: 14\n",
      "Number of variants after clump: 5\n",
      "VCAM1\n",
      "Number of variants before clump: 101\n",
      "Number of variants after clump: 43\n",
      "VTCN1\n",
      "Number of variants before clump: 17\n",
      "Number of variants after clump: 5\n",
      "WFDC2\n",
      "Number of variants before clump: 27\n",
      "Number of variants after clump: 6\n",
      "XCL1\n",
      "Number of variants before clump: 60\n",
      "Number of variants after clump: 20\n",
      "ZBTB17\n",
      "Number of variants before clump: 7\n",
      "Number of variants after clump: 3\n",
      "CPU times: user 5h 45min 56s, sys: 20.3 s, total: 5h 46min 16s\n",
      "Wall time: 5h 46min 43s\n"
     ]
    }
   ],
   "source": [
    "%%time\n",
    "test_proteins = our_protein1[\"Protein\"].unique()\n",
    "for i in test_proteins: \n",
    "    print (i)\n",
    "    try:\n",
    "        temp = our_protein1[our_protein1[\"Protein\"] == i ] \n",
    "        temp_clump = normal_clump(temp) \n",
    "        score = make_score(temp_clump)\n",
    "        file_name = \"weights_normal_clump/\" + i + \".clumpbyp.weights\"\n",
    "        score.to_csv(file_name, sep=\"\\t\", index=None) \n",
    "    \n",
    "    except: \n",
    "        print (\"naughty protein- find out why\")\n",
    "        next"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 20,
   "id": "6e8cf16c-aaaa-4256-927a-747376500174",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Gene</th>\n",
       "      <th>Protein</th>\n",
       "      <th>Match(?Cis)</th>\n",
       "      <th>cis_trans_position_1mb_coding_region</th>\n",
       "      <th>Num_Proteins_p&lt;1x10-8</th>\n",
       "      <th>Genotype</th>\n",
       "      <th>Most Damaging Effect</th>\n",
       "      <th>nSamples</th>\n",
       "      <th>nCarriers</th>\n",
       "      <th>nNonCarriers</th>\n",
       "      <th>...</th>\n",
       "      <th>model</th>\n",
       "      <th>panel</th>\n",
       "      <th>Olink Assay</th>\n",
       "      <th>MAF</th>\n",
       "      <th>CHR</th>\n",
       "      <th>BP</th>\n",
       "      <th>REF</th>\n",
       "      <th>ALT</th>\n",
       "      <th>Groups</th>\n",
       "      <th>variant_class_score</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>0 rows × 25 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "Empty DataFrame\n",
       "Columns: [Gene, Protein, Match(?Cis), cis_trans_position_1mb_coding_region, Num_Proteins_p<1x10-8, Genotype, Most Damaging Effect, nSamples, nCarriers, nNonCarriers, p-value, beta, lower, upper, se, model, panel, Olink Assay, MAF, CHR, BP, REF, ALT, Groups, variant_class_score]\n",
       "Index: []\n",
       "\n",
       "[0 rows x 25 columns]"
      ]
     },
     "execution_count": 20,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "our_protein1[our_protein1[\"Protein\"] ==\"HLA-E\"] "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 21,
   "id": "014fa446-206c-46b5-8b88-b62745a0899d",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Gene</th>\n",
       "      <th>Protein</th>\n",
       "      <th>Match(?Cis)</th>\n",
       "      <th>cis_trans_position_1mb_coding_region</th>\n",
       "      <th>Num_Proteins_p&lt;1x10-8</th>\n",
       "      <th>Genotype</th>\n",
       "      <th>Most Damaging Effect</th>\n",
       "      <th>nSamples</th>\n",
       "      <th>nCarriers</th>\n",
       "      <th>nNonCarriers</th>\n",
       "      <th>p-value</th>\n",
       "      <th>beta</th>\n",
       "      <th>lower</th>\n",
       "      <th>upper</th>\n",
       "      <th>se</th>\n",
       "      <th>model</th>\n",
       "      <th>panel</th>\n",
       "      <th>Olink Assay</th>\n",
       "      <th>MAF</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>42047</th>\n",
       "      <td>TRIM27</td>\n",
       "      <td>HLA-E</td>\n",
       "      <td>No</td>\n",
       "      <td>trans-position, trans-gene</td>\n",
       "      <td>79</td>\n",
       "      <td>6-28923399-T-C</td>\n",
       "      <td>synonymous_variant</td>\n",
       "      <td>46241</td>\n",
       "      <td>10294</td>\n",
       "      <td>35947</td>\n",
       "      <td>0.000000e+00</td>\n",
       "      <td>0.372897</td>\n",
       "      <td>0.354149</td>\n",
       "      <td>0.391645</td>\n",
       "      <td>0.009565</td>\n",
       "      <td>genotypic</td>\n",
       "      <td>Inflammation</td>\n",
       "      <td>HLA-E:P13747:OID20532:v1:Inflammation</td>\n",
       "      <td>0.118784</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>42048</th>\n",
       "      <td>OR12D3</td>\n",
       "      <td>HLA-E</td>\n",
       "      <td>No</td>\n",
       "      <td>trans-position, trans-gene</td>\n",
       "      <td>89</td>\n",
       "      <td>6-29374998-G-A</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>46223</td>\n",
       "      <td>10625</td>\n",
       "      <td>35598</td>\n",
       "      <td>0.000000e+00</td>\n",
       "      <td>0.373874</td>\n",
       "      <td>0.355381</td>\n",
       "      <td>0.392368</td>\n",
       "      <td>0.009435</td>\n",
       "      <td>genotypic</td>\n",
       "      <td>Inflammation</td>\n",
       "      <td>HLA-E:P13747:OID20532:v1:Inflammation</td>\n",
       "      <td>0.122719</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>42049</th>\n",
       "      <td>UBD</td>\n",
       "      <td>HLA-E</td>\n",
       "      <td>No</td>\n",
       "      <td>cis-position, trans-gene</td>\n",
       "      <td>101</td>\n",
       "      <td>6-29556180-A-G</td>\n",
       "      <td>synonymous_variant</td>\n",
       "      <td>46245</td>\n",
       "      <td>10755</td>\n",
       "      <td>35490</td>\n",
       "      <td>0.000000e+00</td>\n",
       "      <td>0.388539</td>\n",
       "      <td>0.370260</td>\n",
       "      <td>0.406818</td>\n",
       "      <td>0.009326</td>\n",
       "      <td>genotypic</td>\n",
       "      <td>Inflammation</td>\n",
       "      <td>HLA-E:P13747:OID20532:v1:Inflammation</td>\n",
       "      <td>0.124715</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>42050</th>\n",
       "      <td>HLA-A</td>\n",
       "      <td>HLA-E</td>\n",
       "      <td>No</td>\n",
       "      <td>cis-position, trans-gene</td>\n",
       "      <td>74</td>\n",
       "      <td>6-29942954-G-A</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>44601</td>\n",
       "      <td>14646</td>\n",
       "      <td>29955</td>\n",
       "      <td>0.000000e+00</td>\n",
       "      <td>0.315361</td>\n",
       "      <td>0.299753</td>\n",
       "      <td>0.330970</td>\n",
       "      <td>0.007963</td>\n",
       "      <td>genotypic</td>\n",
       "      <td>Inflammation</td>\n",
       "      <td>HLA-E:P13747:OID20532:v1:Inflammation</td>\n",
       "      <td>0.184563</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>42051</th>\n",
       "      <td>ZNRD1</td>\n",
       "      <td>HLA-E</td>\n",
       "      <td>No</td>\n",
       "      <td>cis-position, trans-gene</td>\n",
       "      <td>106</td>\n",
       "      <td>6-30064745-A-C</td>\n",
       "      <td>3_prime_UTR_variant</td>\n",
       "      <td>45431</td>\n",
       "      <td>11067</td>\n",
       "      <td>34364</td>\n",
       "      <td>0.000000e+00</td>\n",
       "      <td>0.402805</td>\n",
       "      <td>0.384788</td>\n",
       "      <td>0.420822</td>\n",
       "      <td>0.009192</td>\n",
       "      <td>genotypic</td>\n",
       "      <td>Inflammation</td>\n",
       "      <td>HLA-E:P13747:OID20532:v1:Inflammation</td>\n",
       "      <td>0.131194</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>...</th>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "      <td>...</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>42743</th>\n",
       "      <td>HIST1H2BG</td>\n",
       "      <td>HLA-E</td>\n",
       "      <td>No</td>\n",
       "      <td>trans-position, trans-gene</td>\n",
       "      <td>2</td>\n",
       "      <td>6-26216356-C-T</td>\n",
       "      <td>synonymous_variant</td>\n",
       "      <td>46249</td>\n",
       "      <td>3963</td>\n",
       "      <td>42286</td>\n",
       "      <td>6.760000e-09</td>\n",
       "      <td>-0.089205</td>\n",
       "      <td>-0.119361</td>\n",
       "      <td>-0.059049</td>\n",
       "      <td>0.015386</td>\n",
       "      <td>genotypic</td>\n",
       "      <td>Inflammation</td>\n",
       "      <td>HLA-E:P13747:OID20532:v1:Inflammation</td>\n",
       "      <td>0.044106</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>42744</th>\n",
       "      <td>MUC22</td>\n",
       "      <td>HLA-E</td>\n",
       "      <td>No</td>\n",
       "      <td>cis-position, trans-gene</td>\n",
       "      <td>2</td>\n",
       "      <td>6-31026455-A-G</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>45626</td>\n",
       "      <td>485</td>\n",
       "      <td>45141</td>\n",
       "      <td>7.290000e-09</td>\n",
       "      <td>-0.212990</td>\n",
       "      <td>-0.285150</td>\n",
       "      <td>-0.140829</td>\n",
       "      <td>0.036816</td>\n",
       "      <td>genotypic</td>\n",
       "      <td>Inflammation</td>\n",
       "      <td>HLA-E:P13747:OID20532:v1:Inflammation</td>\n",
       "      <td>0.006029</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>42745</th>\n",
       "      <td>MUC22</td>\n",
       "      <td>HLA-E</td>\n",
       "      <td>No</td>\n",
       "      <td>cis-position, trans-gene</td>\n",
       "      <td>2</td>\n",
       "      <td>6-31029498-C-G</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>46260</td>\n",
       "      <td>482</td>\n",
       "      <td>45778</td>\n",
       "      <td>7.660000e-09</td>\n",
       "      <td>-0.252239</td>\n",
       "      <td>-0.337820</td>\n",
       "      <td>-0.166657</td>\n",
       "      <td>0.043664</td>\n",
       "      <td>genotypic</td>\n",
       "      <td>Inflammation</td>\n",
       "      <td>HLA-E:P13747:OID20532:v1:Inflammation</td>\n",
       "      <td>0.005218</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>42746</th>\n",
       "      <td>CFB</td>\n",
       "      <td>HLA-E</td>\n",
       "      <td>No</td>\n",
       "      <td>trans-position, trans-gene</td>\n",
       "      <td>18</td>\n",
       "      <td>6-31950687-A-G</td>\n",
       "      <td>missense_variant</td>\n",
       "      <td>46261</td>\n",
       "      <td>1267</td>\n",
       "      <td>44994</td>\n",
       "      <td>8.660000e-09</td>\n",
       "      <td>-0.155780</td>\n",
       "      <td>-0.208824</td>\n",
       "      <td>-0.102735</td>\n",
       "      <td>0.027063</td>\n",
       "      <td>genotypic</td>\n",
       "      <td>Inflammation</td>\n",
       "      <td>HLA-E:P13747:OID20532:v1:Inflammation</td>\n",
       "      <td>0.013825</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>42747</th>\n",
       "      <td>GABBR1</td>\n",
       "      <td>HLA-E</td>\n",
       "      <td>No</td>\n",
       "      <td>cis-position, trans-gene</td>\n",
       "      <td>11</td>\n",
       "      <td>6-29607226-G-A</td>\n",
       "      <td>splice_region_variant</td>\n",
       "      <td>46117</td>\n",
       "      <td>10476</td>\n",
       "      <td>35641</td>\n",
       "      <td>9.660000e-09</td>\n",
       "      <td>0.055599</td>\n",
       "      <td>0.036606</td>\n",
       "      <td>0.074592</td>\n",
       "      <td>0.009690</td>\n",
       "      <td>genotypic</td>\n",
       "      <td>Inflammation</td>\n",
       "      <td>HLA-E:P13747:OID20532:v1:Inflammation</td>\n",
       "      <td>0.120868</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>701 rows × 19 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "            Gene Protein Match(?Cis) cis_trans_position_1mb_coding_region  \\\n",
       "42047     TRIM27   HLA-E          No           trans-position, trans-gene   \n",
       "42048     OR12D3   HLA-E          No           trans-position, trans-gene   \n",
       "42049        UBD   HLA-E          No             cis-position, trans-gene   \n",
       "42050      HLA-A   HLA-E          No             cis-position, trans-gene   \n",
       "42051      ZNRD1   HLA-E          No             cis-position, trans-gene   \n",
       "...          ...     ...         ...                                  ...   \n",
       "42743  HIST1H2BG   HLA-E          No           trans-position, trans-gene   \n",
       "42744      MUC22   HLA-E          No             cis-position, trans-gene   \n",
       "42745      MUC22   HLA-E          No             cis-position, trans-gene   \n",
       "42746        CFB   HLA-E          No           trans-position, trans-gene   \n",
       "42747     GABBR1   HLA-E          No             cis-position, trans-gene   \n",
       "\n",
       "       Num_Proteins_p<1x10-8        Genotype   Most Damaging Effect  nSamples  \\\n",
       "42047                     79  6-28923399-T-C     synonymous_variant     46241   \n",
       "42048                     89  6-29374998-G-A       missense_variant     46223   \n",
       "42049                    101  6-29556180-A-G     synonymous_variant     46245   \n",
       "42050                     74  6-29942954-G-A       missense_variant     44601   \n",
       "42051                    106  6-30064745-A-C    3_prime_UTR_variant     45431   \n",
       "...                      ...             ...                    ...       ...   \n",
       "42743                      2  6-26216356-C-T     synonymous_variant     46249   \n",
       "42744                      2  6-31026455-A-G       missense_variant     45626   \n",
       "42745                      2  6-31029498-C-G       missense_variant     46260   \n",
       "42746                     18  6-31950687-A-G       missense_variant     46261   \n",
       "42747                     11  6-29607226-G-A  splice_region_variant     46117   \n",
       "\n",
       "       nCarriers  nNonCarriers       p-value      beta     lower     upper  \\\n",
       "42047      10294         35947  0.000000e+00  0.372897  0.354149  0.391645   \n",
       "42048      10625         35598  0.000000e+00  0.373874  0.355381  0.392368   \n",
       "42049      10755         35490  0.000000e+00  0.388539  0.370260  0.406818   \n",
       "42050      14646         29955  0.000000e+00  0.315361  0.299753  0.330970   \n",
       "42051      11067         34364  0.000000e+00  0.402805  0.384788  0.420822   \n",
       "...          ...           ...           ...       ...       ...       ...   \n",
       "42743       3963         42286  6.760000e-09 -0.089205 -0.119361 -0.059049   \n",
       "42744        485         45141  7.290000e-09 -0.212990 -0.285150 -0.140829   \n",
       "42745        482         45778  7.660000e-09 -0.252239 -0.337820 -0.166657   \n",
       "42746       1267         44994  8.660000e-09 -0.155780 -0.208824 -0.102735   \n",
       "42747      10476         35641  9.660000e-09  0.055599  0.036606  0.074592   \n",
       "\n",
       "             se      model         panel  \\\n",
       "42047  0.009565  genotypic  Inflammation   \n",
       "42048  0.009435  genotypic  Inflammation   \n",
       "42049  0.009326  genotypic  Inflammation   \n",
       "42050  0.007963  genotypic  Inflammation   \n",
       "42051  0.009192  genotypic  Inflammation   \n",
       "...         ...        ...           ...   \n",
       "42743  0.015386  genotypic  Inflammation   \n",
       "42744  0.036816  genotypic  Inflammation   \n",
       "42745  0.043664  genotypic  Inflammation   \n",
       "42746  0.027063  genotypic  Inflammation   \n",
       "42747  0.009690  genotypic  Inflammation   \n",
       "\n",
       "                                 Olink Assay       MAF  \n",
       "42047  HLA-E:P13747:OID20532:v1:Inflammation  0.118784  \n",
       "42048  HLA-E:P13747:OID20532:v1:Inflammation  0.122719  \n",
       "42049  HLA-E:P13747:OID20532:v1:Inflammation  0.124715  \n",
       "42050  HLA-E:P13747:OID20532:v1:Inflammation  0.184563  \n",
       "42051  HLA-E:P13747:OID20532:v1:Inflammation  0.131194  \n",
       "...                                      ...       ...  \n",
       "42743  HLA-E:P13747:OID20532:v1:Inflammation  0.044106  \n",
       "42744  HLA-E:P13747:OID20532:v1:Inflammation  0.006029  \n",
       "42745  HLA-E:P13747:OID20532:v1:Inflammation  0.005218  \n",
       "42746  HLA-E:P13747:OID20532:v1:Inflammation  0.013825  \n",
       "42747  HLA-E:P13747:OID20532:v1:Inflammation  0.120868  \n",
       "\n",
       "[701 rows x 19 columns]"
      ]
     },
     "execution_count": 21,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "raw[raw[\"Protein\"] ==\"HLA-E\"] "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 26,
   "id": "3972e9e8-7b87-488e-a9b9-78b5d61f3335",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Protein</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "Empty DataFrame\n",
       "Columns: [Protein]\n",
       "Index: []"
      ]
     },
     "execution_count": 26,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "proteins_of_interest[proteins_of_interest[\"Protein\"] == \"HLA-E\"] "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 45,
   "id": "62bd29d3-0944-4e3f-a778-f6e526e2e948",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Number of variants before clump: 13\n",
      "Number of variants after clump: 6\n"
     ]
    }
   ],
   "source": [
    "i = \"C19orf12\"\n",
    "temp = our_protein1[our_protein1[\"Protein\"] == i ] \n",
    "temp_clump = normal_clump(temp) \n",
    "score = make_score(temp_clump)\n",
    "file_name = \"weights_normal_clump/\" + i + \".clumpbyp.weights\"\n",
    "score.to_csv(file_name, sep=\"\\t\", index=None) "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 49,
   "id": "383b3012-c331-42c9-a91f-81f7c9365de8",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "Number of variants before clump: 707\n",
      "Number of variants after clump: 181\n"
     ]
    }
   ],
   "source": [
    "i = \"HLA-E\"\n",
    "temp = our_protein1[our_protein1[\"Protein\"] == i ] \n",
    "temp_clump = normal_clump(temp) \n",
    "score = make_score(temp_clump)\n",
    "file_name = \"weights_normal_clump/\" + i + \".clumpbyp.weights\"\n",
    "score.to_csv(file_name, sep=\"\\t\", index=None) "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "id": "39a57cd8-4a8e-488b-9b94-f4b9b2c0ae02",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>0</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>WAS</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>TPP1</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "      0\n",
       "0   WAS\n",
       "1  TPP1"
      ]
     },
     "execution_count": 3,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "new_list = pd.read_csv(\"get_scores.txt\", header=None) \n",
    "new_list.head(2)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 52,
   "id": "51ce295f-58d6-49b3-931d-6ded89b5bd0b",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "WAS\n",
      "TPP1\n",
      "AGRN\n",
      "CA12\n",
      "CD300C\n",
      "CD300E\n",
      "CD302\n",
      "CD38\n",
      "CLEC14A\n",
      "COLEC12\n",
      "EFNA1\n",
      "EPHB4\n",
      "FOLR1\n",
      "FOLR2\n",
      "GFRA1\n",
      "HAVCR2\n",
      "IFNGR1\n",
      "IGFBP6\n",
      "JAM2\n",
      "KLRB1\n",
      "LAYN\n",
      "LGALS9\n",
      "LRP11\n",
      "LTBR\n",
      "MMP7\n",
      "NBL1\n",
      "NECTIN2\n",
      "NECTIN4\n",
      "RTN4R\n",
      "SEMA3F\n",
      "TGFBR2\n",
      "THBD\n",
      "TNFRSF19\n",
      "TNFRSF1A\n",
      "BAMBI\n",
      "CLEC5A\n",
      "ITIH3\n",
      "MMP9\n",
      "ITIH3\n",
      "BAMBI\n",
      "KLK4\n",
      "WFIKKN1\n"
     ]
    }
   ],
   "source": [
    "for i in new_list[0]: \n",
    "    print (i)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "fc4100c6-44d6-4cff-af6e-935f3dc872c0",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "b6806d19-76dd-4ad5-94fa-8ad641503ca4",
   "metadata": {},
   "outputs": [],
   "source": [
    "### add something\n",
    "#raw = pd.read_csv(\"ewas_results_genomodel.txt\", sep=\"\\t\")\n",
    "raw = pd.read_excel(\"exome_gwas.xlsx\" ) \n",
    "proteins_of_interest = pd.read_csv(\"get_scores.txt\",sep=\"\\t\", header=None)\n",
    "proteins_of_interest.columns = [\"Protein\"] \n",
    "our_proteins = raw[raw[\"Protein\"].isin(proteins_of_interest[\"Protein\"] ) ] \n",
    "ld_matrix = pd.read_csv(\"ld_matrix.ld\", delimiter=r\"\\s+\" )\n",
    "ld_matrix = ld_matrix[[\"SNP_A\", \"SNP_B\",\"R2\"]].copy() \n",
    "our_proteins[[\"CHR\", \"BP\", \"REF\" , \"ALT\"]] = our_proteins.Genotype.str.split(\"-\", expand=True) \n",
    "grouping_lists = [ [\"stop_gained\", \"disruptive_inframe_deletion\", \"disruptive_inframe_insertion\", \"frameshift_variant\", \"conservative_inframe_deletion\", \"splice_acceptor_variant\", \"start_lost\", \"splice_donor_variant\"], [\"missense_variant\"] , \n",
    "                  [\"synonymous_variant\"] ,  [\"non_coding_transcript_exon_variant\" ], ['3_prime_UTR_variant','5_prime_UTR_variant', 'synonymous_variant&splice_region_variant', \"5_prime_UTR_premature_start_codon_gain_variant\",\"missense_variant&splice_region_variant\", \"stop_retained_variant&splice_region_variant\", \"5_prime_UTR_truncation\", \"stop_retained_variant\"]] \n",
    "names =  ['High_IMPACT', 'MISS', 'SYNONYMOUS', 'NON_CODING', 'OTHER']\n",
    "\n",
    "var_scores = [\"5\", \"4\" , \"2\", \"3\", \"1\"]   \n",
    "\n",
    "\n",
    "var_class = (pd.DataFrame({'Groups': names, \"Most Damaging Effect\": grouping_lists , \"variant_class_score\": var_scores})\n",
    "        .explode('Most Damaging Effect')\n",
    "        .reset_index(drop=True))\n",
    "\n",
    "our_protein1 = our_proteins.merge(var_class,  on = \"Most Damaging Effect\", how='left').fillna(\"Other\")"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 5,
   "id": "dae5afcf-6f91-4740-a454-3a8e3e9aeb71",
   "metadata": {},
   "outputs": [],
   "source": [
    "#our_protein1[our_protein1[\"Protein\"] == \"CD38\" ] "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 4,
   "id": "6eaf7e30-2b7a-4545-acbe-d2fe6d5056ac",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "WAS\n",
      "Number of variants before clump: 15\n",
      "Number of variants after clump: 6\n",
      "TPP1\n",
      "Number of variants before clump: 99\n",
      "Number of variants after clump: 44\n",
      "AGRN\n",
      "Number of variants before clump: 95\n",
      "Number of variants after clump: 90\n",
      "CA12\n",
      "Number of variants before clump: 79\n",
      "Number of variants after clump: 55\n",
      "CD300C\n",
      "Number of variants before clump: 88\n",
      "Number of variants after clump: 49\n",
      "CD300E\n",
      "Number of variants before clump: 66\n",
      "Number of variants after clump: 33\n",
      "CD302\n",
      "Number of variants before clump: 294\n",
      "Number of variants after clump: 114\n",
      "CD38\n",
      "Number of variants before clump: 27\n",
      "Number of variants after clump: 22\n",
      "CLEC14A\n",
      "Number of variants before clump: 51\n",
      "Number of variants after clump: 20\n",
      "COLEC12\n",
      "Number of variants before clump: 14\n",
      "Number of variants after clump: 10\n",
      "EFNA1\n",
      "Number of variants before clump: 69\n",
      "Number of variants after clump: 41\n",
      "EPHB4\n",
      "Number of variants before clump: 93\n",
      "Number of variants after clump: 48\n",
      "FOLR1\n",
      "Number of variants before clump: 45\n",
      "Number of variants after clump: 16\n",
      "FOLR2\n",
      "Number of variants before clump: 235\n",
      "Number of variants after clump: 91\n",
      "GFRA1\n",
      "Number of variants before clump: 18\n",
      "Number of variants after clump: 16\n",
      "HAVCR2\n",
      "Number of variants before clump: 36\n",
      "Number of variants after clump: 19\n",
      "IFNGR1\n",
      "Number of variants before clump: 79\n",
      "Number of variants after clump: 43\n",
      "IGFBP6\n",
      "Number of variants before clump: 27\n",
      "Number of variants after clump: 14\n",
      "JAM2\n",
      "Number of variants before clump: 39\n",
      "Number of variants after clump: 21\n",
      "KLRB1\n",
      "Number of variants before clump: 110\n",
      "Number of variants after clump: 52\n",
      "LAYN\n",
      "Number of variants before clump: 93\n",
      "Number of variants after clump: 59\n",
      "LGALS9\n",
      "Number of variants before clump: 62\n",
      "Number of variants after clump: 21\n",
      "LRP11\n",
      "Number of variants before clump: 54\n",
      "Number of variants after clump: 51\n",
      "LTBR\n",
      "Number of variants before clump: 144\n",
      "Number of variants after clump: 64\n",
      "MMP7\n",
      "Number of variants before clump: 21\n",
      "Number of variants after clump: 9\n",
      "NBL1\n",
      "Number of variants before clump: 28\n",
      "Number of variants after clump: 15\n",
      "NECTIN2\n",
      "Number of variants before clump: 106\n",
      "Number of variants after clump: 46\n",
      "NECTIN4\n",
      "Number of variants before clump: 47\n",
      "Number of variants after clump: 17\n",
      "RTN4R\n",
      "Number of variants before clump: 40\n",
      "Number of variants after clump: 32\n",
      "SEMA3F\n",
      "Number of variants before clump: 146\n",
      "Number of variants after clump: 113\n",
      "TGFBR2\n",
      "Number of variants before clump: 160\n",
      "Number of variants after clump: 65\n",
      "THBD\n",
      "Number of variants before clump: 92\n",
      "Number of variants after clump: 44\n",
      "TNFRSF19\n",
      "Number of variants before clump: 36\n",
      "Number of variants after clump: 24\n",
      "TNFRSF1A\n",
      "Number of variants before clump: 105\n",
      "Number of variants after clump: 53\n",
      "BAMBI\n",
      "Number of variants before clump: 5\n",
      "Number of variants after clump: 4\n",
      "CLEC5A\n",
      "Number of variants before clump: 81\n",
      "Number of variants after clump: 56\n",
      "ITIH3\n",
      "Number of variants before clump: 220\n",
      "Number of variants after clump: 102\n",
      "MMP9\n",
      "Number of variants before clump: 126\n",
      "Number of variants after clump: 68\n",
      "ITIH3\n",
      "Number of variants before clump: 220\n",
      "Number of variants after clump: 102\n",
      "BAMBI\n",
      "Number of variants before clump: 5\n",
      "Number of variants after clump: 4\n",
      "KLK4\n",
      "Number of variants before clump: 37\n",
      "Number of variants after clump: 36\n",
      "WFIKKN1\n",
      "Number of variants before clump: 276\n",
      "Number of variants after clump: 149\n",
      "CPU times: user 25min 27s, sys: 2.61 s, total: 25min 30s\n",
      "Wall time: 25min 31s\n"
     ]
    }
   ],
   "source": [
    "%%time\n",
    "test_proteins = our_protein1[\"Protein\"].unique()\n",
    "for i in new_list[0]:  \n",
    "    print (i)\n",
    "    try:\n",
    "        temp = our_protein1[our_protein1[\"Protein\"] == i ] \n",
    "        temp_clump = normal_clump(temp) \n",
    "        score = make_score(temp_clump)\n",
    "        file_name = \"all_protein_weights/\" + i + \".clumpbyp.weights\"\n",
    "        score.to_csv(file_name, sep=\"\\t\", index=None) \n",
    "    \n",
    "    except: \n",
    "        print (\"naughty protein- find out why\")\n",
    "        next"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 15,
   "id": "7beb053c-6d2f-495b-b0aa-3719f338bc41",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>Gene</th>\n",
       "      <th>Protein</th>\n",
       "      <th>Match(?Cis)</th>\n",
       "      <th>cis_trans_position_1mb_coding_region</th>\n",
       "      <th>Num_Proteins_p&lt;1x10-8</th>\n",
       "      <th>Genotype</th>\n",
       "      <th>Most Damaging Effect</th>\n",
       "      <th>nSamples</th>\n",
       "      <th>nCarriers</th>\n",
       "      <th>nNonCarriers</th>\n",
       "      <th>...</th>\n",
       "      <th>model</th>\n",
       "      <th>panel</th>\n",
       "      <th>Olink Assay</th>\n",
       "      <th>MAF</th>\n",
       "      <th>CHR</th>\n",
       "      <th>BP</th>\n",
       "      <th>REF</th>\n",
       "      <th>ALT</th>\n",
       "      <th>Groups</th>\n",
       "      <th>variant_class_score</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>PLA2G2A</td>\n",
       "      <td>PLA2G2A</td>\n",
       "      <td>Yes</td>\n",
       "      <td>cis-position, cis-gene</td>\n",
       "      <td>3</td>\n",
       "      <td>1-19978364-G-A</td>\n",
       "      <td>non_coding_transcript_exon_variant</td>\n",
       "      <td>45775</td>\n",
       "      <td>18188</td>\n",
       "      <td>27587</td>\n",
       "      <td>...</td>\n",
       "      <td>dominant</td>\n",
       "      <td>Cardiometabolic</td>\n",
       "      <td>PLA2G2A:P14555:OID20274:v1:Cardiometabolic</td>\n",
       "      <td>0.224092</td>\n",
       "      <td>1</td>\n",
       "      <td>19978364</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>NON_CODING</td>\n",
       "      <td>3</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>PLA2G2A</td>\n",
       "      <td>PLA2G2A</td>\n",
       "      <td>Yes</td>\n",
       "      <td>cis-position, cis-gene</td>\n",
       "      <td>3</td>\n",
       "      <td>1-19978433-G-A</td>\n",
       "      <td>synonymous_variant</td>\n",
       "      <td>45808</td>\n",
       "      <td>18195</td>\n",
       "      <td>27613</td>\n",
       "      <td>...</td>\n",
       "      <td>dominant</td>\n",
       "      <td>Cardiometabolic</td>\n",
       "      <td>PLA2G2A:P14555:OID20274:v1:Cardiometabolic</td>\n",
       "      <td>0.223906</td>\n",
       "      <td>1</td>\n",
       "      <td>19978433</td>\n",
       "      <td>G</td>\n",
       "      <td>A</td>\n",
       "      <td>SYNONYMOUS</td>\n",
       "      <td>2</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "<p>2 rows × 25 columns</p>\n",
       "</div>"
      ],
      "text/plain": [
       "      Gene  Protein Match(?Cis) cis_trans_position_1mb_coding_region  \\\n",
       "0  PLA2G2A  PLA2G2A         Yes               cis-position, cis-gene   \n",
       "1  PLA2G2A  PLA2G2A         Yes               cis-position, cis-gene   \n",
       "\n",
       "   Num_Proteins_p<1x10-8        Genotype                Most Damaging Effect  \\\n",
       "0                      3  1-19978364-G-A  non_coding_transcript_exon_variant   \n",
       "1                      3  1-19978433-G-A                  synonymous_variant   \n",
       "\n",
       "   nSamples  nCarriers  nNonCarriers  ...     model            panel  \\\n",
       "0     45775      18188         27587  ...  dominant  Cardiometabolic   \n",
       "1     45808      18195         27613  ...  dominant  Cardiometabolic   \n",
       "\n",
       "                                  Olink Assay       MAF  CHR        BP REF  \\\n",
       "0  PLA2G2A:P14555:OID20274:v1:Cardiometabolic  0.224092    1  19978364   G   \n",
       "1  PLA2G2A:P14555:OID20274:v1:Cardiometabolic  0.223906    1  19978433   G   \n",
       "\n",
       "  ALT      Groups variant_class_score  \n",
       "0   A  NON_CODING                   3  \n",
       "1   A  SYNONYMOUS                   2  \n",
       "\n",
       "[2 rows x 25 columns]"
      ]
     },
     "execution_count": 15,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "raw = pd.read_excel(\"exome_gwas.xlsx\" ) \n",
    "#proteins_of_interest = pd.read_csv(\"get_scores.txt\",sep=\"\\t\", header=None)\n",
    "#proteins_of_interest.columns = [\"Protein\"] \n",
    "our_proteins = raw\n",
    "ld_matrix = pd.read_csv(\"ld_matrix.ld\", delimiter=r\"\\s+\" )\n",
    "ld_matrix = ld_matrix[[\"SNP_A\", \"SNP_B\",\"R2\"]].copy() \n",
    "our_proteins[[\"CHR\", \"BP\", \"REF\" , \"ALT\"]] = our_proteins.Genotype.str.split(\"-\", expand=True) \n",
    "grouping_lists = [ [\"stop_gained\", \"disruptive_inframe_deletion\", \"disruptive_inframe_insertion\", \"frameshift_variant\", \"conservative_inframe_deletion\", \"splice_acceptor_variant\", \"start_lost\", \"splice_donor_variant\"], [\"missense_variant\"] , \n",
    "                  [\"synonymous_variant\"] ,  [\"non_coding_transcript_exon_variant\" ], ['3_prime_UTR_variant','5_prime_UTR_variant', 'synonymous_variant&splice_region_variant', \"5_prime_UTR_premature_start_codon_gain_variant\",\"missense_variant&splice_region_variant\", \"stop_retained_variant&splice_region_variant\", \"5_prime_UTR_truncation\", \"stop_retained_variant\"]] \n",
    "names =  ['High_IMPACT', 'MISS', 'SYNONYMOUS', 'NON_CODING', 'OTHER']\n",
    "\n",
    "var_scores = [\"5\", \"4\" , \"2\", \"3\", \"1\"]   \n",
    "\n",
    "\n",
    "var_class = (pd.DataFrame({'Groups': names, \"Most Damaging Effect\": grouping_lists , \"variant_class_score\": var_scores})\n",
    "        .explode('Most Damaging Effect')\n",
    "        .reset_index(drop=True))\n",
    "\n",
    "our_protein1 = our_proteins.merge(var_class,  on = \"Most Damaging Effect\", how='left').fillna(\"Other\")\n",
    "our_protein1.head(2)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 16,
   "id": "dbac538e-82ce-404c-8945-a6f5a3b11f6a",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "array(['PLA2G2A', 'FUCA1', 'PCSK9', ..., 'RUVBL1', 'STX8', 'GH2'],\n",
       "      dtype=object)"
      ]
     },
     "execution_count": 16,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "our_protein1.Protein.unique()"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 17,
   "id": "4af11777-ab8e-44a3-aeb7-5c4ca33d82fe",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/plain": [
       "(1387,)"
      ]
     },
     "execution_count": 17,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "test_proteins = our_protein1[\"Protein\"].unique()\n",
    "test_proteins.shape"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "9daee7c6-dbcc-45bc-be5e-e3c151effa39",
   "metadata": {},
   "outputs": [],
   "source": []
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "id": "b9e2623b-d6a5-4ede-8b79-f1455d389a01",
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "PLA2G2A\n",
      "Number of variants before clump: 28\n",
      "Number of variants after clump: 25\n",
      "FUCA1\n",
      "Number of variants before clump: 61\n",
      "Number of variants after clump: 45\n",
      "PCSK9\n",
      "Number of variants before clump: 60\n",
      "Number of variants after clump: 39\n",
      "ANGPTL3\n",
      "Number of variants before clump: 37\n",
      "Number of variants after clump: 25\n",
      "AMY2A\n",
      "Number of variants before clump: 61\n",
      "Number of variants after clump: 45\n",
      "AMY2B\n",
      "Number of variants before clump: 57\n",
      "Number of variants after clump: 43\n",
      "IL6R\n",
      "Number of variants before clump: 92\n",
      "Number of variants after clump: 57\n",
      "ADAM15\n",
      "Number of variants before clump: 189\n",
      "Number of variants after clump: 142\n",
      "FCRL1\n",
      "Number of variants before clump: 111\n",
      "Number of variants after clump: 52\n",
      "FCGR2A\n",
      "Number of variants before clump: 92\n",
      "Number of variants after clump: 58\n",
      "FCGR3B\n",
      "Number of variants before clump: 123\n",
      "Number of variants after clump: 59\n",
      "SELP\n",
      "Number of variants before clump: 56\n",
      "Number of variants after clump: 36\n",
      "CHIT1\n",
      "Number of variants before clump: 38\n",
      "Number of variants after clump: 32\n",
      "PM20D1\n",
      "Number of variants before clump: 67\n",
      "Number of variants after clump: 48\n",
      "SFTPD\n",
      "Number of variants before clump: 336\n",
      "Number of variants after clump: 106\n",
      "CRTAC1\n",
      "Number of variants before clump: 70\n",
      "Number of variants after clump: 61\n",
      "CDHR5\n",
      "Number of variants before clump: 223\n",
      "Number of variants after clump: 160\n",
      "CTSD\n",
      "Number of variants before clump: 84\n",
      "Number of variants after clump: 28\n",
      "CHRDL2\n",
      "Number of variants before clump: 31\n",
      "Number of variants after clump: 11\n",
      "NCAM1\n",
      "Number of variants before clump: 73\n",
      "Number of variants after clump: 34\n",
      "FCN2\n",
      "Number of variants before clump: 80\n",
      "Number of variants after clump: 30\n",
      "CLEC1A\n",
      "Number of variants before clump: 66\n",
      "Number of variants after clump: 35\n",
      "SLITRK6\n",
      "Number of variants before clump: 25\n",
      "Number of variants after clump: 15\n",
      "F7\n",
      "Number of variants before clump: 74\n",
      "Number of variants after clump: 40\n",
      "ANG\n",
      "Number of variants before clump: 22\n",
      "Number of variants after clump: 12\n",
      "LGALS3\n",
      "Number of variants before clump: 86\n",
      "Number of variants after clump: 58\n",
      "ENTPD5\n",
      "Number of variants before clump: 70\n",
      "Number of variants after clump: 66\n",
      "SERPINA11\n",
      "Number of variants before clump: 96\n",
      "Number of variants after clump: 52\n",
      "SERPINA12\n",
      "Number of variants before clump: 75\n",
      "Number of variants after clump: 53\n",
      "TYRO3\n",
      "Number of variants before clump: 221\n",
      "Number of variants after clump: 164\n",
      "CTSH\n",
      "Number of variants before clump: 83\n",
      "Number of variants after clump: 77\n",
      "VASN\n",
      "Number of variants before clump: 56\n",
      "Number of variants after clump: 48\n",
      "UMOD\n",
      "Number of variants before clump: 73\n",
      "Number of variants after clump: 58\n",
      "CES1\n",
      "Number of variants before clump: 105\n",
      "Number of variants after clump: 66\n",
      "CA5A\n",
      "Number of variants before clump: 15\n",
      "Number of variants after clump: 14\n",
      "SCARF1\n",
      "Number of variants before clump: 43\n",
      "Number of variants after clump: 34\n",
      "BLMH\n",
      "Number of variants before clump: 227\n",
      "Number of variants after clump: 137\n",
      "CCL16\n",
      "Number of variants before clump: 96\n",
      "Number of variants after clump: 56\n",
      "CCL15\n",
      "Number of variants before clump: 114\n",
      "Number of variants after clump: 40\n",
      "AOC3\n",
      "Number of variants before clump: 156\n",
      "Number of variants after clump: 146\n",
      "CA4\n",
      "Number of variants before clump: 57\n",
      "Number of variants after clump: 32\n",
      "CLUL1\n",
      "Number of variants before clump: 18\n",
      "Number of variants after clump: 11\n",
      "MEP1B\n",
      "Number of variants before clump: 57\n",
      "Number of variants after clump: 37\n",
      "TCN2\n",
      "Number of variants before clump: 154\n",
      "Number of variants after clump: 135\n",
      "ICAM5\n",
      "Number of variants before clump: 166\n",
      "Number of variants after clump: 81\n",
      "ICAM1\n",
      "Number of variants before clump: 134\n",
      "Number of variants after clump: 55\n",
      "ICAM3\n",
      "Number of variants before clump: 439\n",
      "Number of variants after clump: 137\n",
      "ACP5\n",
      "Number of variants before clump: 71\n",
      "Number of variants after clump: 35\n",
      "TNFRSF10C\n",
      "Number of variants before clump: 118\n",
      "Number of variants after clump: 71\n",
      "PRSS27\n",
      "Number of variants before clump: 168\n",
      "Number of variants after clump: 89\n",
      "CDH1\n",
      "Number of variants before clump: 131\n",
      "Number of variants after clump: 47\n",
      "CDH17\n",
      "Number of variants before clump: 200\n",
      "Number of variants after clump: 84\n",
      "CGREF1\n",
      "Number of variants before clump: 208\n",
      "Number of variants after clump: 124\n",
      "IL19\n",
      "Number of variants before clump: 68\n",
      "Number of variants after clump: 24\n",
      "LILRB5\n",
      "Number of variants before clump: 85\n",
      "Number of variants after clump: 69\n",
      "LILRB2\n",
      "Number of variants before clump: 170\n",
      "Number of variants after clump: 97\n",
      "QPCT\n",
      "Number of variants before clump: 113\n",
      "Number of variants after clump: 60\n",
      "IL1RL1\n",
      "Number of variants before clump: 92\n",
      "Number of variants after clump: 52\n",
      "TFPI\n",
      "Number of variants before clump: 59\n",
      "Number of variants after clump: 52\n",
      "CST3\n",
      "Number of variants before clump: 18\n",
      "Number of variants after clump: 10\n",
      "BPIFB1\n",
      "Number of variants before clump: 95\n",
      "Number of variants after clump: 76\n",
      "AHCY\n",
      "Number of variants before clump: 11\n",
      "Number of variants after clump: 11\n",
      "PROC\n",
      "Number of variants before clump: 161\n",
      "Number of variants after clump: 57\n",
      "LBP\n",
      "Number of variants before clump: 38\n",
      "Number of variants after clump: 38\n",
      "PI3\n",
      "Number of variants before clump: 35\n",
      "Number of variants after clump: 34\n",
      "CTSZ\n",
      "Number of variants before clump: 81\n",
      "Number of variants after clump: 48\n",
      "CSTB\n",
      "Number of variants before clump: 32\n",
      "Number of variants after clump: 27\n",
      "COMT\n",
      "Number of variants before clump: 44\n",
      "Number of variants after clump: 43\n",
      "PLXNB2\n",
      "Number of variants before clump: 192\n",
      "Number of variants after clump: 100\n",
      "HYAL1\n",
      "Number of variants before clump: 100\n",
      "Number of variants after clump: 92\n",
      "ITIH3\n",
      "Number of variants before clump: 220\n",
      "Number of variants after clump: 102\n",
      "ICAM2\n",
      "Number of variants before clump: 137\n",
      "Number of variants after clump: 67\n",
      "FETUB\n",
      "Number of variants before clump: 54\n",
      "Number of variants after clump: 45\n",
      "TFRC\n",
      "Number of variants before clump: 138\n",
      "Number of variants after clump: 94\n",
      "PDGFRA\n",
      "Number of variants before clump: 56\n",
      "Number of variants after clump: 36\n",
      "SPARCL1\n",
      "Number of variants before clump: 105\n",
      "Number of variants after clump: 32\n",
      "PDCD6\n",
      "Number of variants before clump: 94\n",
      "Number of variants after clump: 91\n",
      "OSMR\n",
      "Number of variants before clump: 31\n",
      "Number of variants after clump: 23\n",
      "PAM\n",
      "Number of variants before clump: 43\n",
      "Number of variants after clump: 40\n",
      "TGFBI\n",
      "Number of variants before clump: 140\n",
      "Number of variants after clump: 60\n",
      "PDGFRB\n",
      "Number of variants before clump: 89\n",
      "Number of variants after clump: 88\n",
      "C2\n",
      "Number of variants before clump: 446\n",
      "Number of variants after clump: 193\n",
      "APOM\n",
      "Number of variants before clump: 281\n",
      "Number of variants after clump: 170\n",
      "GLO1\n",
      "Number of variants before clump: 10\n",
      "Number of variants after clump: 10\n",
      "GSTA1\n",
      "Number of variants before clump: 29\n",
      "Number of variants after clump: 15\n",
      "LRP11\n",
      "Number of variants before clump: 54\n",
      "Number of variants after clump: 51\n",
      "RNASET2\n",
      "Number of variants before clump: 349\n",
      "Number of variants after clump: 97\n",
      "GPNMB\n",
      "Number of variants before clump: 43\n",
      "Number of variants after clump: 37\n",
      "DDC\n",
      "Number of variants before clump: 42\n",
      "Number of variants after clump: 25\n",
      "PON2\n",
      "Number of variants before clump: 41\n",
      "Number of variants after clump: 27\n",
      "PILRB\n",
      "Number of variants before clump: 395\n",
      "Number of variants after clump: 361\n",
      "CTSB\n",
      "Number of variants before clump: 160\n",
      "Number of variants after clump: 129\n",
      "LPL\n",
      "Number of variants before clump: 21\n",
      "Number of variants after clump: 18\n",
      "GGH\n",
      "Number of variants before clump: 78\n",
      "Number of variants after clump: 36\n",
      "SELE\n",
      "Number of variants before clump: 143\n",
      "Number of variants after clump: 60\n",
      "CD209\n",
      "Number of variants before clump: 138\n",
      "Number of variants after clump: 59\n",
      "ENG\n",
      "Number of variants before clump: 80\n",
      "Number of variants after clump: 34\n",
      "TIE1\n",
      "Number of variants before clump: 114\n",
      "Number of variants after clump: 48\n",
      "GP2\n",
      "Number of variants before clump: 112\n",
      "Number of variants after clump: 31\n",
      "PCDH17\n",
      "Number of variants before clump: 78\n",
      "Number of variants after clump: 26\n",
      "CDH5\n",
      "Number of variants before clump: 83\n",
      "Number of variants after clump: 43\n",
      "CELA3A\n",
      "Number of variants before clump: 145\n",
      "Number of variants after clump: 86\n",
      "NOTCH1\n",
      "Number of variants before clump: 60\n",
      "Number of variants after clump: 21\n",
      "ADAMTS13\n",
      "Number of variants before clump: 111\n",
      "Number of variants after clump: 51\n",
      "VWF\n",
      "Number of variants before clump: 54\n",
      "Number of variants after clump: 24\n",
      "DPP7\n",
      "Number of variants before clump: 185\n",
      "Number of variants after clump: 129\n",
      "F9\n",
      "Number of variants before clump: 25\n",
      "Number of variants after clump: 8\n",
      "NADK\n",
      "Number of variants before clump: 85\n",
      "Number of variants after clump: 50\n",
      "DPT\n",
      "Number of variants before clump: 47\n",
      "Number of variants after clump: 29\n",
      "GAS6\n",
      "Number of variants before clump: 75\n",
      "Number of variants after clump: 37\n",
      "ANPEP\n",
      "Number of variants before clump: 54\n",
      "Number of variants after clump: 27\n",
      "CNTN1\n",
      "Number of variants before clump: 109\n",
      "Number of variants after clump: 89\n",
      "LDLR\n",
      "Number of variants before clump: 95\n",
      "Number of variants after clump: 49\n",
      "PGLYRP1\n",
      "Number of variants before clump: 164\n",
      "Number of variants after clump: 97\n",
      "LILRB1\n",
      "Number of variants before clump: 493\n",
      "Number of variants after clump: 198\n",
      "SSC5D\n",
      "Number of variants before clump: 133\n",
      "Number of variants after clump: 113\n",
      "REG1B\n",
      "Number of variants before clump: 63\n",
      "Number of variants after clump: 20\n",
      "AGXT\n",
      "Number of variants before clump: 45\n",
      "Number of variants after clump: 31\n",
      "CXCL5\n",
      "Number of variants before clump: 55\n",
      "Number of variants after clump: 29\n",
      "CD14\n",
      "Number of variants before clump: 155\n",
      "Number of variants after clump: 154\n",
      "CLEC5A\n",
      "Number of variants before clump: 81\n",
      "Number of variants after clump: 56\n",
      "FBP1\n",
      "Number of variants before clump: 20\n",
      "Number of variants after clump: 13\n",
      "CTRC\n",
      "Number of variants before clump: 202\n",
      "Number of variants after clump: 107\n",
      "C1QA\n",
      "Number of variants before clump: 124\n",
      "Number of variants after clump: 54\n",
      "ICAM4\n",
      "Number of variants before clump: 81\n",
      "Number of variants after clump: 51\n",
      "PROK1\n",
      "Number of variants before clump: 68\n",
      "Number of variants after clump: 48\n",
      "FCRL3\n",
      "Number of variants before clump: 51\n",
      "Number of variants after clump: 35\n",
      "FCRL2\n",
      "Number of variants before clump: 265\n",
      "Number of variants after clump: 129\n",
      "CCL7\n",
      "Number of variants before clump: 44\n",
      "Number of variants after clump: 31\n",
      "FCRL6\n",
      "Number of variants before clump: 176\n",
      "Number of variants after clump: 96\n",
      "CD48\n",
      "Number of variants before clump: 66\n",
      "Number of variants after clump: 30\n",
      "SLAMF7\n",
      "Number of variants before clump: 236\n",
      "Number of variants after clump: 87\n",
      "CD244\n",
      "Number of variants before clump: 94\n",
      "Number of variants after clump: 41\n",
      "PRELP\n",
      "Number of variants before clump: 14\n",
      "Number of variants after clump: 9\n",
      "NFASC\n",
      "Number of variants before clump: 46\n",
      "Number of variants after clump: 24\n",
      "PTX3\n",
      "Number of variants before clump: 66\n",
      "Number of variants after clump: 27\n",
      "PNLIPRP2\n",
      "Number of variants before clump: 63\n",
      "Number of variants after clump: 29\n",
      "MGMT\n",
      "Number of variants before clump: 16\n",
      "Number of variants after clump: 9\n",
      "TRIM5\n",
      "Number of variants before clump: 49\n",
      "Number of variants after clump: 42\n",
      "TPP1\n",
      "Number of variants before clump: 99\n",
      "Number of variants after clump: 44\n",
      "TREM2\n",
      "Number of variants before clump: 110\n",
      "Number of variants after clump: 51\n",
      "CD6\n",
      "Number of variants before clump: 87\n",
      "Number of variants after clump: 44\n",
      "SCGB1A1\n",
      "Number of variants before clump: 90\n",
      "Number of variants after clump: 75\n",
      "CTSC\n",
      "Number of variants before clump: 119\n",
      "Number of variants after clump: 63\n",
      "MMP10\n",
      "Number of variants before clump: 62\n",
      "Number of variants after clump: 37\n",
      "MMP1\n",
      "Number of variants before clump: 79\n",
      "Number of variants after clump: 56\n",
      "CLEC4C\n",
      "Number of variants before clump: 71\n",
      "Number of variants after clump: 38\n",
      "CLEC4A\n",
      "Number of variants before clump: 100\n",
      "Number of variants after clump: 57\n",
      "CLEC4D\n",
      "Number of variants before clump: 39\n",
      "Number of variants after clump: 16\n",
      "CLEC7A\n",
      "Number of variants before clump: 56\n",
      "Number of variants after clump: 51\n",
      "SELPLG\n",
      "Number of variants before clump: 102\n",
      "Number of variants after clump: 58\n",
      "IL17D\n",
      "Number of variants before clump: 16\n",
      "Number of variants after clump: 15\n",
      "GZMB\n",
      "Number of variants before clump: 209\n",
      "Number of variants after clump: 101\n",
      "LTBR\n",
      "Number of variants before clump: 144\n",
      "Number of variants after clump: 64\n",
      "CKAP4\n",
      "Number of variants before clump: 21\n",
      "Number of variants after clump: 14\n",
      "AMN\n",
      "Number of variants before clump: 78\n",
      "Number of variants after clump: 66\n",
      "SCG3\n",
      "Number of variants before clump: 79\n",
      "Number of variants after clump: 72\n",
      "CD276\n",
      "Number of variants before clump: 88\n",
      "Number of variants after clump: 85\n",
      "IL16\n",
      "Number of variants before clump: 36\n",
      "Number of variants after clump: 30\n",
      "TPSAB1\n",
      "Number of variants before clump: 181\n",
      "Number of variants after clump: 148\n",
      "TNFSF12\n",
      "Number of variants before clump: 156\n",
      "Number of variants after clump: 77\n",
      "SHMT1\n",
      "Number of variants before clump: 122\n",
      "Number of variants after clump: 122\n",
      "ALDH3A1\n",
      "Number of variants before clump: 33\n",
      "Number of variants after clump: 33\n",
      "LGALS9\n",
      "Number of variants before clump: 62\n",
      "Number of variants after clump: 21\n",
      "CCL23\n",
      "Number of variants before clump: 85\n",
      "Number of variants after clump: 43\n",
      "CCL3\n",
      "Number of variants before clump: 59\n",
      "Number of variants after clump: 25\n",
      "FABP9\n",
      "Number of variants before clump: 146\n",
      "Number of variants after clump: 114\n",
      "WFIKKN2\n",
      "Number of variants before clump: 66\n",
      "Number of variants after clump: 40\n",
      "MILR1\n",
      "Number of variants before clump: 105\n",
      "Number of variants after clump: 52\n",
      "ENPP7\n",
      "Number of variants before clump: 30\n",
      "Number of variants after clump: 20\n",
      "TNFRSF11A\n",
      "Number of variants before clump: 58\n",
      "Number of variants after clump: 34\n",
      "SERPINB8\n",
      "Number of variants before clump: 54\n",
      "Number of variants after clump: 53\n",
      "CD70\n",
      "Number of variants before clump: 67\n",
      "Number of variants after clump: 24\n",
      "CCL25\n",
      "Number of variants before clump: 190\n",
      "Number of variants after clump: 114\n",
      "IL12RB1\n",
      "Number of variants before clump: 104\n",
      "Number of variants after clump: 36\n",
      "CEACAM21\n",
      "Number of variants before clump: 147\n",
      "Number of variants after clump: 81\n",
      "TNFSF10\n",
      "Number of variants before clump: 93\n",
      "Number of variants after clump: 51\n",
      "SIGLEC10\n",
      "Number of variants before clump: 123\n",
      "Number of variants after clump: 65\n",
      "OSCAR\n",
      "Number of variants before clump: 121\n",
      "Number of variants after clump: 92\n",
      "FCAR\n",
      "Number of variants before clump: 101\n",
      "Number of variants after clump: 65\n",
      "IL1RL2\n",
      "Number of variants before clump: 96\n",
      "Number of variants after clump: 53\n",
      "IL18R1\n",
      "Number of variants before clump: 62\n",
      "Number of variants after clump: 29\n",
      "LY75\n",
      "Number of variants before clump: 122\n",
      "Number of variants after clump: 60\n",
      "GALNT3\n",
      "Number of variants before clump: 97\n",
      "Number of variants after clump: 37\n",
      "ADAM23\n",
      "Number of variants before clump: 75\n",
      "Number of variants after clump: 39\n",
      "SIRPB1\n",
      "Number of variants before clump: 67\n",
      "Number of variants after clump: 32\n",
      "SIGLEC1\n",
      "Number of variants before clump: 169\n",
      "Number of variants after clump: 90\n",
      "CST7\n",
      "Number of variants before clump: 109\n",
      "Number of variants after clump: 75\n",
      "ADA\n",
      "Number of variants before clump: 30\n",
      "Number of variants after clump: 30\n",
      "CD40\n",
      "Number of variants before clump: 47\n",
      "Number of variants after clump: 19\n",
      "TNFRSF13C\n",
      "Number of variants before clump: 110\n",
      "Number of variants after clump: 48\n",
      "CRELD2\n",
      "Number of variants before clump: 114\n",
      "Number of variants after clump: 47\n",
      "CCL4\n",
      "Number of variants before clump: 120\n",
      "Number of variants after clump: 80\n",
      "IL17RB\n",
      "Number of variants before clump: 227\n",
      "Number of variants after clump: 132\n",
      "CD200R1\n",
      "Number of variants before clump: 101\n",
      "Number of variants after clump: 50\n",
      "CLSTN2\n",
      "Number of variants before clump: 21\n",
      "Number of variants after clump: 19\n",
      "CXCL1\n",
      "Number of variants before clump: 57\n",
      "Number of variants after clump: 32\n",
      "CXCL10\n",
      "Number of variants before clump: 64\n",
      "Number of variants after clump: 32\n",
      "FGF5\n",
      "Number of variants before clump: 8\n",
      "Number of variants after clump: 8\n",
      "FGF2\n",
      "Number of variants before clump: 36\n",
      "Number of variants after clump: 34\n",
      "TLR3\n",
      "Number of variants before clump: 44\n",
      "Number of variants after clump: 40\n",
      "RAB6A\n",
      "Number of variants before clump: 94\n",
      "Number of variants after clump: 62\n",
      "CRHBP\n",
      "Number of variants before clump: 12\n",
      "Number of variants after clump: 12\n",
      "MEGF10\n",
      "Number of variants before clump: 79\n",
      "Number of variants after clump: 59\n",
      "IL12B\n",
      "Number of variants before clump: 263\n",
      "Number of variants after clump: 82\n",
      "BTN3A2\n",
      "Number of variants before clump: 774\n",
      "Number of variants after clump: 176\n",
      "BTN2A1\n",
      "Number of variants before clump: 527\n",
      "Number of variants after clump: 117\n",
      "MICB_MICA\n",
      "Number of variants before clump: 1269\n",
      "Number of variants after clump: 599\n",
      "HLA-E\n",
      "Number of variants before clump: 707\n",
      "Number of variants after clump: 181\n",
      "CDSN\n",
      "Number of variants before clump: 1037\n",
      "Number of variants after clump: 433\n",
      "LTA\n",
      "Number of variants before clump: 640\n",
      "Number of variants after clump: 254\n",
      "HLA-DRA\n",
      "Number of variants before clump: 678\n",
      "Number of variants after clump: 308\n",
      "DNPH1\n",
      "Number of variants before clump: 47\n",
      "Number of variants after clump: 45\n",
      "ENPP5\n",
      "Number of variants before clump: 53\n",
      "Number of variants after clump: 52\n",
      "SMPDL3A\n",
      "Number of variants before clump: 62\n",
      "Number of variants after clump: 40\n",
      "CCL24\n",
      "Number of variants before clump: 78\n",
      "Number of variants after clump: 66\n",
      "AOC1\n",
      "Number of variants before clump: 26\n",
      "Number of variants after clump: 25\n",
      "MATN2\n",
      "Number of variants before clump: 50\n",
      "Number of variants after clump: 40\n",
      "LY6D\n",
      "Number of variants before clump: 48\n",
      "Number of variants after clump: 36\n",
      "SPINK4\n",
      "Number of variants before clump: 84\n",
      "Number of variants after clump: 47\n",
      "PTPRM\n",
      "Number of variants before clump: 72\n",
      "Number of variants after clump: 35\n",
      "IL3RA\n",
      "Number of variants before clump: 77\n",
      "Number of variants after clump: 39\n",
      "LGALS4\n",
      "Number of variants before clump: 121\n",
      "Number of variants after clump: 73\n",
      "CLEC4G\n",
      "Number of variants before clump: 150\n",
      "Number of variants after clump: 76\n",
      "CHRDL1\n",
      "Number of variants before clump: 19\n",
      "Number of variants after clump: 15\n",
      "AGRN\n",
      "Number of variants before clump: 95\n",
      "Number of variants after clump: 90\n",
      "CD160\n",
      "Number of variants before clump: 138\n",
      "Number of variants after clump: 73\n",
      "IL18\n",
      "Number of variants before clump: 88\n",
      "Number of variants after clump: 66\n",
      "TNFSF13\n",
      "Number of variants before clump: 104\n",
      "Number of variants after clump: 38\n",
      "FXYD5\n",
      "Number of variants before clump: 19\n",
      "Number of variants after clump: 9\n",
      "MERTK\n",
      "Number of variants before clump: 132\n",
      "Number of variants after clump: 59\n",
      "BCL2L11\n",
      "Number of variants before clump: 103\n",
      "Number of variants after clump: 77\n",
      "GMPR\n",
      "Number of variants before clump: 9\n",
      "Number of variants after clump: 9\n",
      "GZMA\n",
      "Number of variants before clump: 517\n",
      "Number of variants after clump: 145\n",
      "LILRB4\n",
      "Number of variants before clump: 699\n",
      "Number of variants after clump: 187\n",
      "MLN\n",
      "Number of variants before clump: 190\n",
      "Number of variants after clump: 118\n",
      "EPCAM\n",
      "Number of variants before clump: 105\n",
      "Number of variants after clump: 51\n",
      "CA6\n",
      "Number of variants before clump: 61\n",
      "Number of variants after clump: 48\n",
      "MASP1\n",
      "Number of variants before clump: 60\n",
      "Number of variants after clump: 56\n",
      "GBP4\n",
      "Number of variants before clump: 49\n",
      "Number of variants after clump: 49\n",
      "GRN\n",
      "Number of variants before clump: 144\n",
      "Number of variants after clump: 57\n",
      "CTSS\n",
      "Number of variants before clump: 96\n",
      "Number of variants after clump: 82\n",
      "EFNA1\n",
      "Number of variants before clump: 69\n",
      "Number of variants after clump: 41\n",
      "FCRL5\n",
      "Number of variants before clump: 106\n",
      "Number of variants after clump: 52\n",
      "MYOC\n",
      "Number of variants before clump: 41\n",
      "Number of variants after clump: 40\n",
      "IDI2\n",
      "Number of variants before clump: 24\n",
      "Number of variants after clump: 18\n",
      "ASAH2\n",
      "Number of variants before clump: 71\n",
      "Number of variants after clump: 52\n",
      "PBLD\n",
      "Number of variants before clump: 48\n",
      "Number of variants after clump: 48\n",
      "SNCG\n",
      "Number of variants before clump: 46\n",
      "Number of variants after clump: 46\n",
      "ADAM8\n",
      "Number of variants before clump: 141\n",
      "Number of variants after clump: 64\n",
      "SMPD1\n",
      "Number of variants before clump: 96\n",
      "Number of variants after clump: 64\n",
      "PAMR1\n",
      "Number of variants before clump: 452\n",
      "Number of variants after clump: 195\n",
      "CCS\n",
      "Number of variants before clump: 164\n",
      "Number of variants after clump: 156\n",
      "RELT\n",
      "Number of variants before clump: 27\n",
      "Number of variants after clump: 10\n",
      "MMP8\n",
      "Number of variants before clump: 62\n",
      "Number of variants after clump: 33\n",
      "MMP3\n",
      "Number of variants before clump: 51\n",
      "Number of variants after clump: 39\n",
      "TMPRSS5\n",
      "Number of variants before clump: 92\n",
      "Number of variants after clump: 82\n",
      "ACVRL1\n",
      "Number of variants before clump: 30\n",
      "Number of variants after clump: 19\n",
      "INHBC\n",
      "Number of variants before clump: 130\n",
      "Number of variants after clump: 117\n",
      "PXN\n",
      "Number of variants before clump: 71\n",
      "Number of variants after clump: 66\n",
      "NID2\n",
      "Number of variants before clump: 58\n",
      "Number of variants after clump: 44\n",
      "FLRT2\n",
      "Number of variants before clump: 25\n",
      "Number of variants after clump: 17\n",
      "TCL1A\n",
      "Number of variants before clump: 85\n",
      "Number of variants after clump: 41\n",
      "SPINT1\n",
      "Number of variants before clump: 173\n",
      "Number of variants after clump: 69\n",
      "PPCDC\n",
      "Number of variants before clump: 54\n",
      "Number of variants after clump: 50\n",
      "CPPED1\n",
      "Number of variants before clump: 15\n",
      "Number of variants after clump: 15\n",
      "IL34\n",
      "Number of variants before clump: 108\n",
      "Number of variants after clump: 93\n",
      "CTRB1\n",
      "Number of variants before clump: 85\n",
      "Number of variants after clump: 63\n",
      "CDH15\n",
      "Number of variants before clump: 127\n",
      "Number of variants after clump: 115\n",
      "DPEP1\n",
      "Number of variants before clump: 281\n",
      "Number of variants after clump: 244\n",
      "CLEC10A\n",
      "Number of variants before clump: 87\n",
      "Number of variants after clump: 71\n",
      "CD300LG\n",
      "Number of variants before clump: 84\n",
      "Number of variants after clump: 62\n",
      "APOH\n",
      "Number of variants before clump: 14\n",
      "Number of variants after clump: 12\n",
      "CD300C\n",
      "Number of variants before clump: 88\n",
      "Number of variants after clump: 49\n",
      "NPTX1\n",
      "Number of variants before clump: 359\n",
      "Number of variants after clump: 165\n",
      "FUT3_FUT5\n",
      "Number of variants before clump: 109\n",
      "Number of variants after clump: 36\n",
      "PSG1\n",
      "Number of variants before clump: 169\n",
      "Number of variants after clump: 133\n",
      "CD177\n",
      "Number of variants before clump: 155\n",
      "Number of variants after clump: 113\n",
      "PVR\n",
      "Number of variants before clump: 79\n",
      "Number of variants after clump: 49\n",
      "PLA2G7\n",
      "Number of variants before clump: 63\n",
      "Number of variants after clump: 48\n",
      "TBC1D17\n",
      "Number of variants before clump: 44\n",
      "Number of variants after clump: 44\n",
      "SIGLEC5\n",
      "Number of variants before clump: 82\n",
      "Number of variants after clump: 55\n",
      "CASP1\n",
      "Number of variants before clump: 10\n",
      "Number of variants after clump: 7\n",
      "VSTM1\n",
      "Number of variants before clump: 57\n",
      "Number of variants after clump: 46\n",
      "LAIR2\n",
      "Number of variants before clump: 98\n",
      "Number of variants after clump: 43\n",
      "LILRA2\n",
      "Number of variants before clump: 89\n",
      "Number of variants after clump: 45\n",
      "AFP\n",
      "Number of variants before clump: 219\n",
      "Number of variants after clump: 182\n",
      "MATN3\n",
      "Number of variants before clump: 23\n",
      "Number of variants after clump: 22\n",
      "RBKS\n",
      "Number of variants before clump: 36\n",
      "Number of variants after clump: 26\n",
      "GNLY\n",
      "Number of variants before clump: 234\n",
      "Number of variants after clump: 97\n",
      "CD8A\n",
      "Number of variants before clump: 512\n",
      "Number of variants after clump: 233\n",
      "HNMT\n",
      "Number of variants before clump: 9\n",
      "Number of variants after clump: 8\n",
      "FRZB\n",
      "Number of variants before clump: 249\n",
      "Number of variants after clump: 99\n",
      "CHGB\n",
      "Number of variants before clump: 53\n",
      "Number of variants after clump: 51\n",
      "CST5\n",
      "Number of variants before clump: 85\n",
      "Number of variants after clump: 67\n",
      "TNFRSF6B\n",
      "Number of variants before clump: 109\n",
      "Number of variants after clump: 33\n",
      "IFNGR2\n",
      "Number of variants before clump: 71\n",
      "Number of variants after clump: 49\n",
      "IL17RA\n",
      "Number of variants before clump: 39\n",
      "Number of variants after clump: 34\n",
      "SCARF2\n",
      "Number of variants before clump: 53\n",
      "Number of variants after clump: 19\n",
      "SUSD2\n",
      "Number of variants before clump: 74\n",
      "Number of variants after clump: 67\n",
      "PIK3IP1\n",
      "Number of variants before clump: 73\n",
      "Number of variants after clump: 69\n",
      "ARSA\n",
      "Number of variants before clump: 91\n",
      "Number of variants after clump: 54\n",
      "TDGF1\n",
      "Number of variants before clump: 358\n",
      "Number of variants after clump: 311\n",
      "ABHD14B\n",
      "Number of variants before clump: 40\n",
      "Number of variants after clump: 34\n",
      "IL1RAP\n",
      "Number of variants before clump: 29\n",
      "Number of variants after clump: 21\n",
      "BST1\n",
      "Number of variants before clump: 49\n",
      "Number of variants after clump: 40\n",
      "KLB\n",
      "Number of variants before clump: 97\n",
      "Number of variants after clump: 93\n",
      "NAAA\n",
      "Number of variants before clump: 93\n",
      "Number of variants after clump: 62\n",
      "IL7R\n",
      "Number of variants before clump: 223\n",
      "Number of variants after clump: 109\n",
      "VCAN\n",
      "Number of variants before clump: 65\n",
      "Number of variants after clump: 62\n",
      "SPINK1\n",
      "Number of variants before clump: 23\n",
      "Number of variants after clump: 14\n",
      "SPINK5\n",
      "Number of variants before clump: 88\n",
      "Number of variants after clump: 43\n",
      "DDR1\n",
      "Number of variants before clump: 407\n",
      "Number of variants after clump: 194\n",
      "LRPAP1\n",
      "Number of variants before clump: 642\n",
      "Number of variants after clump: 353\n",
      "TNXB\n",
      "Number of variants before clump: 603\n",
      "Number of variants after clump: 280\n",
      "MDGA1\n",
      "Number of variants before clump: 33\n",
      "Number of variants after clump: 30\n",
      "ADGRB3\n",
      "Number of variants before clump: 13\n",
      "Number of variants after clump: 13\n",
      "CD109\n",
      "Number of variants before clump: 88\n",
      "Number of variants after clump: 68\n",
      "ULBP2\n",
      "Number of variants before clump: 80\n",
      "Number of variants after clump: 57\n",
      "THBS2\n",
      "Number of variants before clump: 35\n",
      "Number of variants after clump: 23\n",
      "VWC2\n",
      "Number of variants before clump: 56\n",
      "Number of variants after clump: 26\n",
      "SUMF2\n",
      "Number of variants before clump: 106\n",
      "Number of variants after clump: 60\n",
      "ADAM22\n",
      "Number of variants before clump: 52\n",
      "Number of variants after clump: 45\n",
      "PILRA\n",
      "Number of variants before clump: 366\n",
      "Number of variants after clump: 330\n",
      "KEL\n",
      "Number of variants before clump: 122\n",
      "Number of variants after clump: 76\n",
      "MSR1\n",
      "Number of variants before clump: 183\n",
      "Number of variants after clump: 70\n",
      "TNFRSF10B\n",
      "Number of variants before clump: 20\n",
      "Number of variants after clump: 14\n",
      "TNFRSF10A\n",
      "Number of variants before clump: 51\n",
      "Number of variants after clump: 31\n",
      "PLA2G10\n",
      "Number of variants before clump: 427\n",
      "Number of variants after clump: 170\n",
      "OGN\n",
      "Number of variants before clump: 72\n",
      "Number of variants after clump: 67\n",
      "OBP2B\n",
      "Number of variants before clump: 88\n",
      "Number of variants after clump: 56\n",
      "ISLR2\n",
      "Number of variants before clump: 123\n",
      "Number of variants after clump: 56\n",
      "BCAM\n",
      "Number of variants before clump: 141\n",
      "Number of variants after clump: 53\n",
      "PECAM1\n",
      "Number of variants before clump: 60\n",
      "Number of variants after clump: 25\n",
      "KIRREL2\n",
      "Number of variants before clump: 109\n",
      "Number of variants after clump: 72\n",
      "LGALS8\n",
      "Number of variants before clump: 116\n",
      "Number of variants after clump: 54\n",
      "CSF2RA\n",
      "Number of variants before clump: 8\n",
      "Number of variants after clump: 8\n",
      "VSIG4\n",
      "Number of variants before clump: 71\n",
      "Number of variants after clump: 40\n",
      "CCL2\n",
      "Number of variants before clump: 119\n",
      "Number of variants after clump: 66\n",
      "B4GAT1\n",
      "Number of variants before clump: 196\n",
      "Number of variants after clump: 133\n",
      "NOMO1\n",
      "Number of variants before clump: 28\n",
      "Number of variants after clump: 15\n",
      "FCER2\n",
      "Number of variants before clump: 172\n",
      "Number of variants after clump: 70\n",
      "GP6\n",
      "Number of variants before clump: 39\n",
      "Number of variants after clump: 25\n",
      "TIMP4\n",
      "Number of variants before clump: 19\n",
      "Number of variants after clump: 17\n",
      "CDCP1\n",
      "Number of variants before clump: 73\n",
      "Number of variants after clump: 41\n",
      "CPA2\n",
      "Number of variants before clump: 48\n",
      "Number of variants after clump: 38\n",
      "SCARA5\n",
      "Number of variants before clump: 30\n",
      "Number of variants after clump: 19\n",
      "VWA1\n",
      "Number of variants before clump: 75\n",
      "Number of variants after clump: 57\n",
      "ANGPTL7\n",
      "Number of variants before clump: 73\n",
      "Number of variants after clump: 63\n",
      "TACSTD2\n",
      "Number of variants before clump: 74\n",
      "Number of variants after clump: 43\n",
      "ACP6\n",
      "Number of variants before clump: 56\n",
      "Number of variants after clump: 56\n",
      "CRNN\n",
      "Number of variants before clump: 139\n",
      "Number of variants after clump: 133\n",
      "HDGF\n",
      "Number of variants before clump: 96\n",
      "Number of variants after clump: 81\n",
      "SLAMF8\n",
      "Number of variants before clump: 43\n",
      "Number of variants after clump: 24\n",
      "NECTIN4\n",
      "Number of variants before clump: 47\n",
      "Number of variants after clump: 17\n",
      "FCRLB\n",
      "Number of variants before clump: 61\n",
      "Number of variants after clump: 37\n",
      "CNTN2\n",
      "Number of variants before clump: 71\n",
      "Number of variants after clump: 65\n",
      "LEFTY2\n",
      "Number of variants before clump: 69\n",
      "Number of variants after clump: 34\n",
      "LYPD8\n",
      "Number of variants before clump: 75\n",
      "Number of variants after clump: 33\n",
      "ERP44\n",
      "Number of variants before clump: 34\n",
      "Number of variants after clump: 17\n",
      "CDNF\n",
      "Number of variants before clump: 15\n",
      "Number of variants after clump: 14\n",
      "SFTPA2\n",
      "Number of variants before clump: 43\n",
      "Number of variants after clump: 18\n",
      "KAZALD1\n",
      "Number of variants before clump: 106\n",
      "Number of variants after clump: 58\n",
      "EPS8L2\n",
      "Number of variants before clump: 97\n",
      "Number of variants after clump: 29\n",
      "NUCB2\n",
      "Number of variants before clump: 29\n",
      "Number of variants after clump: 16\n",
      "NELL1\n",
      "Number of variants before clump: 66\n",
      "Number of variants after clump: 53\n",
      "LTBP3\n",
      "Number of variants before clump: 180\n",
      "Number of variants after clump: 157\n",
      "FOLR3\n",
      "Number of variants before clump: 126\n",
      "Number of variants after clump: 84\n",
      "MMP12\n",
      "Number of variants before clump: 74\n",
      "Number of variants after clump: 39\n",
      "CLMP\n",
      "Number of variants before clump: 53\n",
      "Number of variants after clump: 32\n",
      "ADAMTS8\n",
      "Number of variants before clump: 37\n",
      "Number of variants after clump: 37\n",
      "RBP5\n",
      "Number of variants before clump: 54\n",
      "Number of variants after clump: 26\n",
      "CREG1\n",
      "Number of variants before clump: 49\n",
      "Number of variants after clump: 28\n",
      "APEX1\n",
      "Number of variants before clump: 27\n",
      "Number of variants after clump: 16\n",
      "SMOC1\n",
      "Number of variants before clump: 33\n",
      "Number of variants after clump: 22\n",
      "SERPINA9\n",
      "Number of variants before clump: 50\n",
      "Number of variants after clump: 28\n",
      "HBQ1\n",
      "Number of variants before clump: 61\n",
      "Number of variants after clump: 57\n",
      "MSLN\n",
      "Number of variants before clump: 161\n",
      "Number of variants after clump: 72\n",
      "DPEP2\n",
      "Number of variants before clump: 150\n",
      "Number of variants after clump: 66\n",
      "ITGB7\n",
      "Number of variants before clump: 219\n",
      "Number of variants after clump: 87\n",
      "VMO1\n",
      "Number of variants before clump: 118\n",
      "Number of variants after clump: 105\n",
      "CCL8\n",
      "Number of variants before clump: 31\n",
      "Number of variants after clump: 17\n",
      "C4BPB\n",
      "Number of variants before clump: 115\n",
      "Number of variants after clump: 74\n",
      "CD300LF\n",
      "Number of variants before clump: 72\n",
      "Number of variants after clump: 62\n",
      "EBI3_IL27\n",
      "Number of variants before clump: 159\n",
      "Number of variants after clump: 47\n",
      "MIA\n",
      "Number of variants before clump: 98\n",
      "Number of variants after clump: 86\n",
      "CEACAM5\n",
      "Number of variants before clump: 117\n",
      "Number of variants after clump: 32\n",
      "CEACAM1\n",
      "Number of variants before clump: 138\n",
      "Number of variants after clump: 49\n",
      "LYPD3\n",
      "Number of variants before clump: 187\n",
      "Number of variants after clump: 85\n",
      "KLK1\n",
      "Number of variants before clump: 190\n",
      "Number of variants after clump: 132\n",
      "F3\n",
      "Number of variants before clump: 109\n",
      "Number of variants after clump: 33\n",
      "FAM3B\n",
      "Number of variants before clump: 136\n",
      "Number of variants after clump: 60\n",
      "DKKL1\n",
      "Number of variants before clump: 204\n",
      "Number of variants after clump: 194\n",
      "KLK4\n",
      "Number of variants before clump: 37\n",
      "Number of variants after clump: 36\n",
      "KLK8\n",
      "Number of variants before clump: 43\n",
      "Number of variants after clump: 33\n",
      "KLK11\n",
      "Number of variants before clump: 60\n",
      "Number of variants after clump: 36\n",
      "KLK10\n",
      "Number of variants before clump: 38\n",
      "Number of variants after clump: 34\n",
      "KLK12\n",
      "Number of variants before clump: 29\n",
      "Number of variants after clump: 27\n",
      "KLK13\n",
      "Number of variants before clump: 43\n",
      "Number of variants after clump: 27\n",
      "KLK14\n",
      "Number of variants before clump: 57\n",
      "Number of variants after clump: 41\n",
      "SIGLEC9\n",
      "Number of variants before clump: 163\n",
      "Number of variants after clump: 94\n",
      "CD33\n",
      "Number of variants before clump: 63\n",
      "Number of variants after clump: 54\n",
      "SIGLEC6\n",
      "Number of variants before clump: 311\n",
      "Number of variants after clump: 112\n",
      "KIR2DL3\n",
      "Number of variants before clump: 867\n",
      "Number of variants after clump: 315\n",
      "KIR3DL1\n",
      "Number of variants before clump: 241\n",
      "Number of variants after clump: 136\n",
      "CD207\n",
      "Number of variants before clump: 297\n",
      "Number of variants after clump: 129\n",
      "CAPG\n",
      "Number of variants before clump: 85\n",
      "Number of variants after clump: 76\n",
      "ALPP\n",
      "Number of variants before clump: 123\n",
      "Number of variants after clump: 49\n",
      "GPC1\n",
      "Number of variants before clump: 84\n",
      "Number of variants after clump: 72\n",
      "CPXM1\n",
      "Number of variants before clump: 138\n",
      "Number of variants after clump: 75\n",
      "WFDC12\n",
      "Number of variants before clump: 158\n",
      "Number of variants after clump: 129\n",
      "PVALB\n",
      "Number of variants before clump: 13\n",
      "Number of variants after clump: 12\n",
      "MME\n",
      "Number of variants before clump: 185\n",
      "Number of variants after clump: 77\n",
      "LRIG1\n",
      "Number of variants before clump: 42\n",
      "Number of variants after clump: 27\n",
      "DCBLD2\n",
      "Number of variants before clump: 89\n",
      "Number of variants after clump: 58\n",
      "RARRES1\n",
      "Number of variants before clump: 19\n",
      "Number of variants after clump: 19\n",
      "IDUA\n",
      "Number of variants before clump: 148\n",
      "Number of variants after clump: 103\n",
      "BTC\n",
      "Number of variants before clump: 12\n",
      "Number of variants after clump: 9\n",
      "HPGDS\n",
      "Number of variants before clump: 30\n",
      "Number of variants after clump: 30\n",
      "TXNDC15\n",
      "Number of variants before clump: 271\n",
      "Number of variants after clump: 126\n",
      "HAVCR1\n",
      "Number of variants before clump: 38\n",
      "Number of variants after clump: 22\n",
      "FLT4\n",
      "Number of variants before clump: 115\n",
      "Number of variants after clump: 76\n",
      "TPMT\n",
      "Number of variants before clump: 21\n",
      "Number of variants after clump: 21\n",
      "LRP1\n",
      "Number of variants before clump: 571\n",
      "Number of variants after clump: 321\n",
      "CPVL\n",
      "Number of variants before clump: 623\n",
      "Number of variants after clump: 175\n",
      "CRISP2\n",
      "Number of variants before clump: 101\n",
      "Number of variants after clump: 94\n",
      "GSTA3\n",
      "Number of variants before clump: 17\n",
      "Number of variants after clump: 12\n",
      "NT5E\n",
      "Number of variants before clump: 70\n",
      "Number of variants after clump: 27\n",
      "VNN2\n",
      "Number of variants before clump: 63\n",
      "Number of variants after clump: 38\n",
      "CCN4\n",
      "Number of variants before clump: 54\n",
      "Number of variants after clump: 37\n",
      "TEK\n",
      "Number of variants before clump: 253\n",
      "Number of variants after clump: 128\n",
      "GCNT1\n",
      "Number of variants before clump: 79\n",
      "Number of variants after clump: 28\n",
      "LGALS7_LGALS7B\n",
      "Number of variants before clump: 135\n",
      "Number of variants after clump: 69\n",
      "KDR\n",
      "Number of variants before clump: 111\n",
      "Number of variants after clump: 55\n",
      "ENTPD2\n",
      "Number of variants before clump: 111\n",
      "Number of variants after clump: 58\n",
      "RP2\n",
      "Number of variants before clump: 9\n",
      "Number of variants after clump: 5\n",
      "EDA2R\n",
      "Number of variants before clump: 32\n",
      "Number of variants after clump: 26\n",
      "IL13RA1\n",
      "Number of variants before clump: 39\n",
      "Number of variants after clump: 27\n",
      "FCGR2B\n",
      "Number of variants before clump: 68\n",
      "Number of variants after clump: 32\n",
      "RET\n",
      "Number of variants before clump: 33\n",
      "Number of variants after clump: 32\n",
      "CD27\n",
      "Number of variants before clump: 50\n",
      "Number of variants after clump: 27\n",
      "CD300E\n",
      "Number of variants before clump: 66\n",
      "Number of variants after clump: 33\n",
      "LRRC25\n",
      "Number of variants before clump: 142\n",
      "Number of variants after clump: 78\n",
      "PDCD1LG2\n",
      "Number of variants before clump: 118\n",
      "Number of variants after clump: 64\n",
      "ARSB\n",
      "Number of variants before clump: 56\n",
      "Number of variants after clump: 42\n",
      "IL12A_IL12B\n",
      "Number of variants before clump: 267\n",
      "Number of variants after clump: 84\n",
      "GFRA2\n",
      "Number of variants before clump: 43\n",
      "Number of variants after clump: 31\n",
      "L1CAM\n",
      "Number of variants before clump: 124\n",
      "Number of variants after clump: 66\n",
      "PODXL\n",
      "Number of variants before clump: 76\n",
      "Number of variants after clump: 30\n",
      "FGF19\n",
      "Number of variants before clump: 61\n",
      "Number of variants after clump: 13\n",
      "PTH1R\n",
      "Number of variants before clump: 47\n",
      "Number of variants after clump: 45\n",
      "AZU1\n",
      "Number of variants before clump: 35\n",
      "Number of variants after clump: 22\n",
      "IL22RA1\n",
      "Number of variants before clump: 47\n",
      "Number of variants after clump: 16\n",
      "IL5RA\n",
      "Number of variants before clump: 316\n",
      "Number of variants after clump: 110\n",
      "BLVRB\n",
      "Number of variants before clump: 11\n",
      "Number of variants after clump: 11\n",
      "CCN3\n",
      "Number of variants before clump: 18\n",
      "Number of variants after clump: 14\n",
      "RGMA\n",
      "Number of variants before clump: 97\n",
      "Number of variants after clump: 38\n",
      "THOP1\n",
      "Number of variants before clump: 21\n",
      "Number of variants after clump: 21\n",
      "DPP6\n",
      "Number of variants before clump: 36\n",
      "Number of variants after clump: 28\n",
      "MFAP5\n",
      "Number of variants before clump: 35\n",
      "Number of variants after clump: 34\n",
      "LPO\n",
      "Number of variants before clump: 194\n",
      "Number of variants after clump: 117\n",
      "SSC4D\n",
      "Number of variants before clump: 66\n",
      "Number of variants after clump: 51\n",
      "HSPB1\n",
      "Number of variants before clump: 44\n",
      "Number of variants after clump: 41\n",
      "VEGFD\n",
      "Number of variants before clump: 31\n",
      "Number of variants after clump: 21\n",
      "OPTC\n",
      "Number of variants before clump: 31\n",
      "Number of variants after clump: 27\n",
      "PADI4\n",
      "Number of variants before clump: 24\n",
      "Number of variants after clump: 24\n",
      "TNFRSF21\n",
      "Number of variants before clump: 116\n",
      "Number of variants after clump: 50\n",
      "FUT8\n",
      "Number of variants before clump: 61\n",
      "Number of variants after clump: 43\n",
      "NUDT2\n",
      "Number of variants before clump: 36\n",
      "Number of variants after clump: 34\n",
      "ADGRE2\n",
      "Number of variants before clump: 71\n",
      "Number of variants after clump: 28\n",
      "IL6\n",
      "Number of variants before clump: 21\n",
      "Number of variants after clump: 11\n",
      "LGALS1\n",
      "Number of variants before clump: 41\n",
      "Number of variants after clump: 41\n",
      "CD55\n",
      "Number of variants before clump: 50\n",
      "Number of variants after clump: 31\n",
      "PLTP\n",
      "Number of variants before clump: 112\n",
      "Number of variants after clump: 51\n",
      "NME3\n",
      "Number of variants before clump: 133\n",
      "Number of variants after clump: 35\n",
      "ESAM\n",
      "Number of variants before clump: 40\n",
      "Number of variants after clump: 18\n",
      "ACY1\n",
      "Number of variants before clump: 57\n",
      "Number of variants after clump: 36\n",
      "B4GALT1\n",
      "Number of variants before clump: 46\n",
      "Number of variants after clump: 13\n",
      "DBI\n",
      "Number of variants before clump: 18\n",
      "Number of variants after clump: 11\n",
      "MEGF9\n",
      "Number of variants before clump: 73\n",
      "Number of variants after clump: 61\n",
      "PAPPA\n",
      "Number of variants before clump: 60\n",
      "Number of variants after clump: 57\n",
      "PDCD5\n",
      "Number of variants before clump: 52\n",
      "Number of variants after clump: 51\n",
      "COL4A1\n",
      "Number of variants before clump: 73\n",
      "Number of variants after clump: 31\n",
      "SETMAR\n",
      "Number of variants before clump: 43\n",
      "Number of variants after clump: 30\n",
      "IL1B\n",
      "Number of variants before clump: 4\n",
      "Number of variants after clump: 2\n",
      "ENTPD6\n",
      "Number of variants before clump: 133\n",
      "Number of variants after clump: 109\n",
      "HAVCR2\n",
      "Number of variants before clump: 36\n",
      "Number of variants after clump: 19\n",
      "CLPS\n",
      "Number of variants before clump: 113\n",
      "Number of variants after clump: 103\n",
      "CD58\n",
      "Number of variants before clump: 80\n",
      "Number of variants after clump: 34\n",
      "DRAXIN\n",
      "Number of variants before clump: 47\n",
      "Number of variants after clump: 24\n",
      "TIMP3\n",
      "Number of variants before clump: 28\n",
      "Number of variants after clump: 14\n",
      "PRTN3\n",
      "Number of variants before clump: 143\n",
      "Number of variants after clump: 78\n",
      "LGMN\n",
      "Number of variants before clump: 80\n",
      "Number of variants after clump: 32\n",
      "GHRL\n",
      "Number of variants before clump: 40\n",
      "Number of variants after clump: 22\n",
      "GPA33\n",
      "Number of variants before clump: 100\n",
      "Number of variants after clump: 51\n",
      "NINJ1\n",
      "Number of variants before clump: 21\n",
      "Number of variants after clump: 15\n",
      "DLK1\n",
      "Number of variants before clump: 24\n",
      "Number of variants after clump: 24\n",
      "ROR1\n",
      "Number of variants before clump: 30\n",
      "Number of variants after clump: 17\n",
      "NRP1\n",
      "Number of variants before clump: 60\n",
      "Number of variants after clump: 24\n",
      "RSPO1\n",
      "Number of variants before clump: 46\n",
      "Number of variants after clump: 25\n",
      "TFF3\n",
      "Number of variants before clump: 339\n",
      "Number of variants after clump: 144\n",
      "IL4R\n",
      "Number of variants before clump: 43\n",
      "Number of variants after clump: 21\n",
      "TNFSF13B\n",
      "Number of variants before clump: 181\n",
      "Number of variants after clump: 84\n",
      "TFF1\n",
      "Number of variants before clump: 38\n",
      "Number of variants after clump: 12\n",
      "NCAM2\n",
      "Number of variants before clump: 40\n",
      "Number of variants after clump: 32\n",
      "NTproBNP\n",
      "Number of variants before clump: 74\n",
      "Number of variants after clump: 70\n",
      "ITGB6\n",
      "Number of variants before clump: 77\n",
      "Number of variants after clump: 53\n",
      "CLEC11A\n",
      "Number of variants before clump: 39\n",
      "Number of variants after clump: 31\n",
      "LAMP3\n",
      "Number of variants before clump: 70\n",
      "Number of variants after clump: 37\n",
      "CASP8\n",
      "Number of variants before clump: 30\n",
      "Number of variants after clump: 26\n",
      "CCL13\n",
      "Number of variants before clump: 133\n",
      "Number of variants after clump: 76\n",
      "IL32\n",
      "Number of variants before clump: 69\n",
      "Number of variants after clump: 50\n",
      "CRTAM\n",
      "Number of variants before clump: 177\n",
      "Number of variants after clump: 66\n",
      "ANGPTL4\n",
      "Number of variants before clump: 28\n",
      "Number of variants after clump: 17\n",
      "GAL\n",
      "Number of variants before clump: 73\n",
      "Number of variants after clump: 35\n",
      "SFTPA1\n",
      "Number of variants before clump: 8\n",
      "Number of variants after clump: 4\n",
      "PLXDC1\n",
      "Number of variants before clump: 75\n",
      "Number of variants after clump: 40\n",
      "GUCA2A\n",
      "Number of variants before clump: 38\n",
      "Number of variants after clump: 34\n",
      "MPO\n",
      "Number of variants before clump: 78\n",
      "Number of variants after clump: 63\n",
      "GOLM2\n",
      "Number of variants before clump: 181\n",
      "Number of variants after clump: 73\n",
      "ITM2A\n",
      "Number of variants before clump: 49\n",
      "Number of variants after clump: 21\n",
      "PLA2G15\n",
      "Number of variants before clump: 339\n",
      "Number of variants after clump: 114\n",
      "IGF1R\n",
      "Number of variants before clump: 134\n",
      "Number of variants after clump: 62\n",
      "PON3\n",
      "Number of variants before clump: 57\n",
      "Number of variants after clump: 49\n",
      "FABP1\n",
      "Number of variants before clump: 10\n",
      "Number of variants after clump: 6\n",
      "PEAR1\n",
      "Number of variants before clump: 74\n",
      "Number of variants after clump: 42\n",
      "CXCL11\n",
      "Number of variants before clump: 68\n",
      "Number of variants after clump: 33\n",
      "DSC2\n",
      "Number of variants before clump: 58\n",
      "Number of variants after clump: 22\n",
      "IL15RA\n",
      "Number of variants before clump: 149\n",
      "Number of variants after clump: 59\n",
      "CDHR1\n",
      "Number of variants before clump: 36\n",
      "Number of variants after clump: 35\n",
      "WARS\n",
      "Number of variants before clump: 26\n",
      "Number of variants after clump: 26\n",
      "CD34\n",
      "Number of variants before clump: 47\n",
      "Number of variants after clump: 20\n",
      "LEPR\n",
      "Number of variants before clump: 44\n",
      "Number of variants after clump: 17\n",
      "NFKBIE\n",
      "Number of variants before clump: 10\n",
      "Number of variants after clump: 5\n",
      "TNFRSF11B\n",
      "Number of variants before clump: 34\n",
      "Number of variants after clump: 20\n",
      "FABP6\n",
      "Number of variants before clump: 98\n",
      "Number of variants after clump: 42\n",
      "GGT5\n",
      "Number of variants before clump: 44\n",
      "Number of variants after clump: 41\n",
      "NMNAT1\n",
      "Number of variants before clump: 41\n",
      "Number of variants after clump: 19\n",
      "TACC3\n",
      "Number of variants before clump: 42\n",
      "Number of variants after clump: 31\n",
      "CHI3L1\n",
      "Number of variants before clump: 37\n",
      "Number of variants after clump: 13\n",
      "CRH\n",
      "Number of variants before clump: 93\n",
      "Number of variants after clump: 41\n",
      "CHL1\n",
      "Number of variants before clump: 30\n",
      "Number of variants after clump: 23\n",
      "SPINK6\n",
      "Number of variants before clump: 20\n",
      "Number of variants after clump: 15\n",
      "MUC13\n",
      "Number of variants before clump: 69\n",
      "Number of variants after clump: 22\n",
      "DSG2\n",
      "Number of variants before clump: 62\n",
      "Number of variants after clump: 31\n",
      "FAP\n",
      "Number of variants before clump: 281\n",
      "Number of variants after clump: 105\n",
      "CCL11\n",
      "Number of variants before clump: 58\n",
      "Number of variants after clump: 46\n",
      "CCL17\n",
      "Number of variants before clump: 65\n",
      "Number of variants after clump: 36\n",
      "MPI\n",
      "Number of variants before clump: 44\n",
      "Number of variants after clump: 24\n",
      "CCL27\n",
      "Number of variants before clump: 129\n",
      "Number of variants after clump: 83\n",
      "GALNT7\n",
      "Number of variants before clump: 77\n",
      "Number of variants after clump: 60\n",
      "LY9\n",
      "Number of variants before clump: 109\n",
      "Number of variants after clump: 41\n",
      "SPON2\n",
      "Number of variants before clump: 48\n",
      "Number of variants after clump: 13\n",
      "LAMP2\n",
      "Number of variants before clump: 43\n",
      "Number of variants after clump: 29\n",
      "ITGA11\n",
      "Number of variants before clump: 53\n",
      "Number of variants after clump: 32\n",
      "SIAE\n",
      "Number of variants before clump: 32\n",
      "Number of variants after clump: 21\n",
      "CES3\n",
      "Number of variants before clump: 71\n",
      "Number of variants after clump: 45\n",
      "LAYN\n",
      "Number of variants before clump: 93\n",
      "Number of variants after clump: 59\n",
      "QDPR\n",
      "Number of variants before clump: 15\n",
      "Number of variants after clump: 15\n",
      "CXADR\n",
      "Number of variants before clump: 76\n",
      "Number of variants after clump: 18\n",
      "KLRD1\n",
      "Number of variants before clump: 408\n",
      "Number of variants after clump: 134\n",
      "FKBP1B\n",
      "Number of variants before clump: 24\n",
      "Number of variants after clump: 21\n",
      "REG1A\n",
      "Number of variants before clump: 94\n",
      "Number of variants after clump: 39\n",
      "FABP2\n",
      "Number of variants before clump: 49\n",
      "Number of variants after clump: 32\n",
      "GSTP1\n",
      "Number of variants before clump: 46\n",
      "Number of variants after clump: 44\n",
      "XPNPEP2\n",
      "Number of variants before clump: 55\n",
      "Number of variants after clump: 44\n",
      "CCL26\n",
      "Number of variants before clump: 81\n",
      "Number of variants after clump: 40\n",
      "SCARB2\n",
      "Number of variants before clump: 39\n",
      "Number of variants after clump: 23\n",
      "NPTXR\n",
      "Number of variants before clump: 40\n",
      "Number of variants after clump: 25\n",
      "RARRES2\n",
      "Number of variants before clump: 55\n",
      "Number of variants after clump: 26\n",
      "CD4\n",
      "Number of variants before clump: 137\n",
      "Number of variants after clump: 79\n",
      "IL1RN\n",
      "Number of variants before clump: 27\n",
      "Number of variants after clump: 17\n",
      "ANGPTL1\n",
      "Number of variants before clump: 35\n",
      "Number of variants after clump: 27\n",
      "SDC4\n",
      "Number of variants before clump: 80\n",
      "Number of variants after clump: 33\n",
      "WFIKKN1\n",
      "Number of variants before clump: 276\n",
      "Number of variants after clump: 149\n",
      "TNC\n",
      "Number of variants before clump: 52\n",
      "Number of variants after clump: 41\n",
      "CA13\n",
      "Number of variants before clump: 23\n",
      "Number of variants after clump: 12\n",
      "SMOC2\n",
      "Number of variants before clump: 28\n",
      "Number of variants after clump: 19\n",
      "SAMD9L\n",
      "Number of variants before clump: 23\n",
      "Number of variants after clump: 18\n",
      "FASLG\n",
      "Number of variants before clump: 154\n",
      "Number of variants after clump: 82\n",
      "FURIN\n",
      "Number of variants before clump: 31\n",
      "Number of variants after clump: 14\n",
      "DDX58\n",
      "Number of variants before clump: 18\n",
      "Number of variants after clump: 18\n",
      "PRTFDC1\n",
      "Number of variants before clump: 15\n",
      "Number of variants after clump: 6\n",
      "IGF2R\n",
      "Number of variants before clump: 250\n",
      "Number of variants after clump: 108\n",
      "AMBP\n",
      "Number of variants before clump: 267\n",
      "Number of variants after clump: 134\n",
      "HSD11B1\n",
      "Number of variants before clump: 83\n",
      "Number of variants after clump: 54\n",
      "PIK3AP1\n",
      "Number of variants before clump: 16\n",
      "Number of variants after clump: 8\n",
      "GLRX\n",
      "Number of variants before clump: 13\n",
      "Number of variants after clump: 13\n",
      "TNFRSF19\n",
      "Number of variants before clump: 36\n",
      "Number of variants after clump: 24\n",
      "PDCD1\n",
      "Number of variants before clump: 94\n",
      "Number of variants after clump: 38\n",
      "NPY\n",
      "Number of variants before clump: 82\n",
      "Number of variants after clump: 42\n",
      "EPHB6\n",
      "Number of variants before clump: 27\n",
      "Number of variants after clump: 23\n",
      "TGFB1\n",
      "Number of variants before clump: 75\n",
      "Number of variants after clump: 25\n",
      "NCF2\n",
      "Number of variants before clump: 15\n",
      "Number of variants after clump: 8\n",
      "CD163\n",
      "Number of variants before clump: 109\n",
      "Number of variants after clump: 42\n",
      "IL10\n",
      "Number of variants before clump: 87\n",
      "Number of variants after clump: 44\n",
      "S100A11\n",
      "Number of variants before clump: 20\n",
      "Number of variants after clump: 13\n",
      "PSMD9\n",
      "Number of variants before clump: 18\n",
      "Number of variants after clump: 18\n",
      "TMPRSS15\n",
      "Number of variants before clump: 81\n",
      "Number of variants after clump: 26\n",
      "CD84\n",
      "Number of variants before clump: 21\n",
      "Number of variants after clump: 14\n",
      "GPR37\n",
      "Number of variants before clump: 38\n",
      "Number of variants after clump: 26\n",
      "NCR1\n",
      "Number of variants before clump: 83\n",
      "Number of variants after clump: 39\n",
      "NUB1\n",
      "Number of variants before clump: 10\n",
      "Number of variants after clump: 7\n",
      "DSG3\n",
      "Number of variants before clump: 63\n",
      "Number of variants after clump: 48\n",
      "MSMB\n",
      "Number of variants before clump: 96\n",
      "Number of variants after clump: 64\n",
      "CTSF\n",
      "Number of variants before clump: 173\n",
      "Number of variants after clump: 126\n",
      "TIMD4\n",
      "Number of variants before clump: 330\n",
      "Number of variants after clump: 161\n",
      "GDF15\n",
      "Number of variants before clump: 26\n",
      "Number of variants after clump: 13\n",
      "LIFR\n",
      "Number of variants before clump: 73\n",
      "Number of variants after clump: 30\n",
      "CST6\n",
      "Number of variants before clump: 81\n",
      "Number of variants after clump: 61\n",
      "KLRB1\n",
      "Number of variants before clump: 110\n",
      "Number of variants after clump: 52\n",
      "CTSV\n",
      "Number of variants before clump: 78\n",
      "Number of variants after clump: 57\n",
      "PTS\n",
      "Number of variants before clump: 56\n",
      "Number of variants after clump: 15\n",
      "LAMA4\n",
      "Number of variants before clump: 68\n",
      "Number of variants after clump: 47\n",
      "CA14\n",
      "Number of variants before clump: 58\n",
      "Number of variants after clump: 53\n",
      "PTGDS\n",
      "Number of variants before clump: 47\n",
      "Number of variants after clump: 18\n",
      "MFGE8\n",
      "Number of variants before clump: 97\n",
      "Number of variants after clump: 52\n",
      "ENPP2\n",
      "Number of variants before clump: 53\n",
      "Number of variants after clump: 23\n",
      "CASP10\n",
      "Number of variants before clump: 28\n",
      "Number of variants after clump: 26\n",
      "SIRPA\n",
      "Number of variants before clump: 28\n",
      "Number of variants after clump: 11\n",
      "CPB1\n",
      "Number of variants before clump: 135\n",
      "Number of variants after clump: 91\n",
      "CR2\n",
      "Number of variants before clump: 174\n",
      "Number of variants after clump: 60\n",
      "ST3GAL1\n",
      "Number of variants before clump: 45\n",
      "Number of variants after clump: 25\n",
      "PAEP\n",
      "Number of variants before clump: 35\n",
      "Number of variants after clump: 35\n",
      "CCL28\n",
      "Number of variants before clump: 81\n",
      "Number of variants after clump: 46\n",
      "DKK3\n",
      "Number of variants before clump: 39\n",
      "Number of variants after clump: 35\n",
      "CCL5\n",
      "Number of variants before clump: 61\n",
      "Number of variants after clump: 28\n",
      "GALNT2\n",
      "Number of variants before clump: 103\n",
      "Number of variants after clump: 62\n",
      "CD2AP\n",
      "Number of variants before clump: 10\n",
      "Number of variants after clump: 7\n",
      "FOLR2\n",
      "Number of variants before clump: 235\n",
      "Number of variants after clump: 91\n",
      "IFNGR1\n",
      "Number of variants before clump: 79\n",
      "Number of variants after clump: 43\n",
      "HYOU1\n",
      "Number of variants before clump: 77\n",
      "Number of variants after clump: 29\n",
      "C1QTNF1\n",
      "Number of variants before clump: 32\n",
      "Number of variants after clump: 17\n",
      "FGFBP1\n",
      "Number of variants before clump: 70\n",
      "Number of variants after clump: 38\n",
      "CXCL16\n",
      "Number of variants before clump: 102\n",
      "Number of variants after clump: 79\n",
      "IL6ST\n",
      "Number of variants before clump: 64\n",
      "Number of variants after clump: 26\n",
      "ECE1\n",
      "Number of variants before clump: 68\n",
      "Number of variants after clump: 26\n",
      "CSF1\n",
      "Number of variants before clump: 92\n",
      "Number of variants after clump: 48\n",
      "ART3\n",
      "Number of variants before clump: 55\n",
      "Number of variants after clump: 37\n",
      "THY1\n",
      "Number of variants before clump: 19\n",
      "Number of variants after clump: 16\n",
      "TDRKH\n",
      "Number of variants before clump: 75\n",
      "Number of variants after clump: 69\n",
      "CXCL6\n",
      "Number of variants before clump: 44\n",
      "Number of variants after clump: 30\n",
      "PGF\n",
      "Number of variants before clump: 22\n",
      "Number of variants after clump: 8\n",
      "DKK4\n",
      "Number of variants before clump: 63\n",
      "Number of variants after clump: 24\n",
      "NECTIN2\n",
      "Number of variants before clump: 106\n",
      "Number of variants after clump: 46\n",
      "ITGA6\n",
      "Number of variants before clump: 50\n",
      "Number of variants after clump: 31\n",
      "SMAD5\n",
      "Number of variants before clump: 51\n",
      "Number of variants after clump: 27\n",
      "LACTB2\n",
      "Number of variants before clump: 19\n",
      "Number of variants after clump: 8\n",
      "CCL14\n",
      "Number of variants before clump: 32\n",
      "Number of variants after clump: 14\n",
      "CX3CL1\n",
      "Number of variants before clump: 252\n",
      "Number of variants after clump: 97\n",
      "CDH6\n",
      "Number of variants before clump: 12\n",
      "Number of variants after clump: 9\n",
      "MET\n",
      "Number of variants before clump: 122\n",
      "Number of variants after clump: 65\n",
      "EPHB4\n",
      "Number of variants before clump: 93\n",
      "Number of variants after clump: 48\n",
      "SDC1\n",
      "Number of variants before clump: 66\n",
      "Number of variants after clump: 27\n",
      "GLT8D2\n",
      "Number of variants before clump: 2\n",
      "Number of variants after clump: 2\n",
      "CD1C\n",
      "Number of variants before clump: 461\n",
      "Number of variants after clump: 144\n",
      "SCLY\n",
      "Number of variants before clump: 40\n",
      "Number of variants after clump: 30\n",
      "S100A4\n",
      "Number of variants before clump: 43\n",
      "Number of variants after clump: 23\n",
      "LXN\n",
      "Number of variants before clump: 16\n",
      "Number of variants after clump: 14\n",
      "CNDP1\n",
      "Number of variants before clump: 55\n",
      "Number of variants after clump: 34\n",
      "PSRC1\n",
      "Number of variants before clump: 27\n",
      "Number of variants after clump: 22\n",
      "IGFBP7\n",
      "Number of variants before clump: 63\n",
      "Number of variants after clump: 36\n",
      "NRCAM\n",
      "Number of variants before clump: 44\n",
      "Number of variants after clump: 20\n",
      "NELL2\n",
      "Number of variants before clump: 163\n",
      "Number of variants after clump: 68\n",
      "SOD2\n",
      "Number of variants before clump: 82\n",
      "Number of variants after clump: 45\n",
      "CD99L2\n",
      "Number of variants before clump: 27\n",
      "Number of variants after clump: 19\n",
      "CD79B\n",
      "Number of variants before clump: 118\n",
      "Number of variants after clump: 31\n",
      "TNFRSF8\n",
      "Number of variants before clump: 244\n",
      "Number of variants after clump: 78\n",
      "THBD\n",
      "Number of variants before clump: 92\n",
      "Number of variants after clump: 44\n",
      "CLEC14A\n",
      "Number of variants before clump: 51\n",
      "Number of variants after clump: 20\n",
      "LRRN1\n",
      "Number of variants before clump: 27\n",
      "Number of variants after clump: 21\n",
      "NOTCH3\n",
      "Number of variants before clump: 53\n",
      "Number of variants after clump: 25\n",
      "GALNT10\n",
      "Number of variants before clump: 58\n",
      "Number of variants after clump: 28\n",
      "CCL22\n",
      "Number of variants before clump: 31\n",
      "Number of variants after clump: 15\n",
      "CD83\n",
      "Number of variants before clump: 111\n",
      "Number of variants after clump: 45\n",
      "CD302\n",
      "Number of variants before clump: 294\n",
      "Number of variants after clump: 114\n",
      "IL1R1\n",
      "Number of variants before clump: 79\n",
      "Number of variants after clump: 37\n",
      "OXT\n",
      "Number of variants before clump: 57\n",
      "Number of variants after clump: 42\n",
      "CD99\n",
      "Number of variants before clump: 17\n",
      "Number of variants after clump: 10\n",
      "NRP2\n",
      "Number of variants before clump: 81\n",
      "Number of variants after clump: 41\n",
      "WWP2\n",
      "Number of variants before clump: 7\n",
      "Number of variants after clump: 5\n",
      "KITLG\n",
      "Number of variants before clump: 123\n",
      "Number of variants after clump: 42\n",
      "GDF2\n",
      "Number of variants before clump: 84\n",
      "Number of variants after clump: 59\n",
      "CPE\n",
      "Number of variants before clump: 31\n",
      "Number of variants after clump: 14\n",
      "FIS1\n",
      "Number of variants before clump: 16\n",
      "Number of variants after clump: 10\n",
      "IMPA1\n",
      "Number of variants before clump: 16\n",
      "Number of variants after clump: 11\n",
      "MARCO\n",
      "Number of variants before clump: 54\n",
      "Number of variants after clump: 27\n",
      "PIGR\n",
      "Number of variants before clump: 46\n",
      "Number of variants after clump: 14\n",
      "SEZ6L\n",
      "Number of variants before clump: 23\n",
      "Number of variants after clump: 17\n",
      "MUC16\n",
      "Number of variants before clump: 37\n",
      "Number of variants after clump: 25\n",
      "MMP7\n",
      "Number of variants before clump: 21\n",
      "Number of variants after clump: 9\n",
      "IL1R2\n",
      "Number of variants before clump: 76\n",
      "Number of variants after clump: 44\n",
      "HBEGF\n",
      "Number of variants before clump: 195\n",
      "Number of variants after clump: 97\n",
      "EPHA1\n",
      "Number of variants before clump: 50\n",
      "Number of variants after clump: 39\n",
      "COL18A1\n",
      "Number of variants before clump: 42\n",
      "Number of variants after clump: 24\n",
      "PODXL2\n",
      "Number of variants before clump: 66\n",
      "Number of variants after clump: 56\n",
      "BOC\n",
      "Number of variants before clump: 68\n",
      "Number of variants after clump: 33\n",
      "GUSB\n",
      "Number of variants before clump: 102\n",
      "Number of variants after clump: 64\n",
      "DPP4\n",
      "Number of variants before clump: 42\n",
      "Number of variants after clump: 29\n",
      "TNFRSF1B\n",
      "Number of variants before clump: 108\n",
      "Number of variants after clump: 57\n",
      "ENO1\n",
      "Number of variants before clump: 9\n",
      "Number of variants after clump: 5\n",
      "CD38\n",
      "Number of variants before clump: 27\n",
      "Number of variants after clump: 22\n",
      "CTSO\n",
      "Number of variants before clump: 118\n",
      "Number of variants after clump: 64\n",
      "GYS1\n",
      "Number of variants before clump: 90\n",
      "Number of variants after clump: 76\n",
      "SEZ6L2\n",
      "Number of variants before clump: 180\n",
      "Number of variants after clump: 78\n",
      "PRDX5\n",
      "Number of variants before clump: 24\n",
      "Number of variants after clump: 22\n",
      "ITGB5\n",
      "Number of variants before clump: 31\n",
      "Number of variants after clump: 19\n",
      "CD22\n",
      "Number of variants before clump: 231\n",
      "Number of variants after clump: 81\n",
      "HS6ST1\n",
      "Number of variants before clump: 80\n",
      "Number of variants after clump: 46\n",
      "SPINT2\n",
      "Number of variants before clump: 99\n",
      "Number of variants after clump: 68\n",
      "CPA1\n",
      "Number of variants before clump: 93\n",
      "Number of variants after clump: 69\n",
      "CBLIF\n",
      "Number of variants before clump: 184\n",
      "Number of variants after clump: 90\n",
      "CCL18\n",
      "Number of variants before clump: 31\n",
      "Number of variants after clump: 20\n",
      "MAD1L1\n",
      "Number of variants before clump: 11\n",
      "Number of variants after clump: 10\n",
      "NCAN\n",
      "Number of variants before clump: 59\n",
      "Number of variants after clump: 53\n",
      "GLB1\n",
      "Number of variants before clump: 56\n",
      "Number of variants after clump: 35\n",
      "ASGR1\n",
      "Number of variants before clump: 44\n",
      "Number of variants after clump: 16\n",
      "IL17F\n",
      "Number of variants before clump: 3\n",
      "Number of variants after clump: 3\n",
      "XG\n",
      "Number of variants before clump: 10\n",
      "Number of variants after clump: 8\n",
      "RSPO3\n",
      "Number of variants before clump: 19\n",
      "Number of variants after clump: 11\n",
      "DNER\n",
      "Number of variants before clump: 33\n",
      "Number of variants after clump: 23\n",
      "RNASE3\n",
      "Number of variants before clump: 15\n",
      "Number of variants after clump: 10\n",
      "CD5\n",
      "Number of variants before clump: 116\n",
      "Number of variants after clump: 55\n",
      "CKMT1A_CKMT1B\n",
      "Number of variants before clump: 90\n",
      "Number of variants after clump: 69\n",
      "CNTN4\n",
      "Number of variants before clump: 30\n",
      "Number of variants after clump: 20\n",
      "JAM2\n",
      "Number of variants before clump: 39\n",
      "Number of variants after clump: 21\n",
      "EGFL7\n",
      "Number of variants before clump: 102\n",
      "Number of variants after clump: 43\n",
      "GGT1\n",
      "Number of variants before clump: 59\n",
      "Number of variants after clump: 18\n",
      "VSIR\n",
      "Number of variants before clump: 40\n",
      "Number of variants after clump: 15\n",
      "HS3ST3B1\n",
      "Number of variants before clump: 18\n",
      "Number of variants after clump: 15\n",
      "PRCP\n",
      "Number of variants before clump: 48\n",
      "Number of variants after clump: 33\n",
      "DDAH1\n",
      "Number of variants before clump: 19\n",
      "Number of variants after clump: 14\n",
      "PRTG\n",
      "Number of variants before clump: 102\n",
      "Number of variants after clump: 52\n",
      "THBS4\n",
      "Number of variants before clump: 53\n",
      "Number of variants after clump: 35\n",
      "SLITRK2\n",
      "Number of variants before clump: 18\n",
      "Number of variants after clump: 14\n",
      "MCAM\n",
      "Number of variants before clump: 64\n",
      "Number of variants after clump: 35\n",
      "BMP6\n",
      "Number of variants before clump: 65\n",
      "Number of variants after clump: 29\n",
      "NPPB\n",
      "Number of variants before clump: 42\n",
      "Number of variants after clump: 38\n",
      "PLAU\n",
      "Number of variants before clump: 113\n",
      "Number of variants after clump: 46\n",
      "CNPY2\n",
      "Number of variants before clump: 3\n",
      "Number of variants after clump: 1\n",
      "KRT19\n",
      "Number of variants before clump: 21\n",
      "Number of variants after clump: 7\n",
      "ISM1\n",
      "Number of variants before clump: 31\n",
      "Number of variants after clump: 22\n",
      "HMOX1\n",
      "Number of variants before clump: 111\n",
      "Number of variants after clump: 65\n",
      "AGER\n",
      "Number of variants before clump: 368\n",
      "Number of variants after clump: 154\n",
      "ACE2\n",
      "Number of variants before clump: 54\n",
      "Number of variants after clump: 28\n",
      "AKR1B1\n",
      "Number of variants before clump: 3\n",
      "Number of variants after clump: 3\n",
      "CA9\n",
      "Number of variants before clump: 52\n",
      "Number of variants after clump: 31\n",
      "IL15\n",
      "Number of variants before clump: 282\n",
      "Number of variants after clump: 85\n",
      "PSMG3\n",
      "Number of variants before clump: 13\n",
      "Number of variants after clump: 12\n",
      "MOG\n",
      "Number of variants before clump: 166\n",
      "Number of variants after clump: 101\n",
      "ACAN\n",
      "Number of variants before clump: 38\n",
      "Number of variants after clump: 33\n",
      "ERBB2\n",
      "Number of variants before clump: 77\n",
      "Number of variants after clump: 42\n",
      "VCAM1\n",
      "Number of variants before clump: 111\n",
      "Number of variants after clump: 47\n",
      "IGFBPL1\n",
      "Number of variants before clump: 27\n",
      "Number of variants after clump: 14\n",
      "CEACAM8\n",
      "Number of variants before clump: 102\n",
      "Number of variants after clump: 69\n",
      "SUSD1\n",
      "Number of variants before clump: 24\n",
      "Number of variants after clump: 14\n",
      "COLEC12\n",
      "Number of variants before clump: 14\n",
      "Number of variants after clump: 10\n",
      "PTPRN2\n",
      "Number of variants before clump: 13\n",
      "Number of variants after clump: 11\n",
      "ANGPT2\n",
      "Number of variants before clump: 38\n",
      "Number of variants after clump: 16\n",
      "PFKFB2\n",
      "Number of variants before clump: 13\n",
      "Number of variants after clump: 8\n",
      "TINAGL1\n",
      "Number of variants before clump: 27\n",
      "Number of variants after clump: 27\n",
      "EFNA4\n",
      "Number of variants before clump: 23\n",
      "Number of variants after clump: 13\n",
      "CLEC1B\n",
      "Number of variants before clump: 46\n",
      "Number of variants after clump: 30\n",
      "ERBB3\n",
      "Number of variants before clump: 73\n",
      "Number of variants after clump: 36\n",
      "SCGN\n",
      "Number of variants before clump: 35\n",
      "Number of variants after clump: 12\n",
      "CCL19\n",
      "Number of variants before clump: 255\n",
      "Number of variants after clump: 100\n",
      "CNPY4\n",
      "Number of variants before clump: 77\n",
      "Number of variants after clump: 65\n",
      "MMP9\n",
      "Number of variants before clump: 126\n",
      "Number of variants after clump: 68\n",
      "IL10RB\n",
      "Number of variants before clump: 44\n",
      "Number of variants after clump: 21\n",
      "CXCL9\n",
      "Number of variants before clump: 143\n",
      "Number of variants after clump: 64\n",
      "CXCL8\n",
      "Number of variants before clump: 107\n",
      "Number of variants after clump: 49\n",
      "BRK1\n",
      "Number of variants before clump: 23\n",
      "Number of variants after clump: 11\n",
      "CD200\n",
      "Number of variants before clump: 68\n",
      "Number of variants after clump: 22\n",
      "ITGA5\n",
      "Number of variants before clump: 81\n",
      "Number of variants after clump: 29\n",
      "CLEC6A\n",
      "Number of variants before clump: 35\n",
      "Number of variants after clump: 19\n",
      "ENO2\n",
      "Number of variants before clump: 39\n",
      "Number of variants after clump: 34\n",
      "S100A12\n",
      "Number of variants before clump: 52\n",
      "Number of variants after clump: 14\n",
      "DPP10\n",
      "Number of variants before clump: 12\n",
      "Number of variants after clump: 10\n",
      "VAT1\n",
      "Number of variants before clump: 103\n",
      "Number of variants after clump: 40\n",
      "TGFBR2\n",
      "Number of variants before clump: 160\n",
      "Number of variants after clump: 65\n",
      "IL10RA\n",
      "Number of variants before clump: 4\n",
      "Number of variants after clump: 2\n",
      "PLAUR\n",
      "Number of variants before clump: 107\n",
      "Number of variants after clump: 41\n",
      "ALCAM\n",
      "Number of variants before clump: 53\n",
      "Number of variants after clump: 29\n",
      "CD74\n",
      "Number of variants before clump: 387\n",
      "Number of variants after clump: 121\n",
      "IQGAP2\n",
      "Number of variants before clump: 3\n",
      "Number of variants after clump: 3\n",
      "SIGLEC7\n",
      "Number of variants before clump: 28\n",
      "Number of variants after clump: 14\n",
      "ANGPT1\n",
      "Number of variants before clump: 81\n",
      "Number of variants after clump: 48\n",
      "CD40LG\n",
      "Number of variants before clump: 48\n",
      "Number of variants after clump: 30\n",
      "GFRA1\n",
      "Number of variants before clump: 18\n",
      "Number of variants after clump: 16\n",
      "CXCL12\n",
      "Number of variants before clump: 34\n",
      "Number of variants after clump: 14\n",
      "CDKN1A\n",
      "Number of variants before clump: 25\n",
      "Number of variants after clump: 15\n",
      "OGFR\n",
      "Number of variants before clump: 23\n",
      "Number of variants after clump: 18\n",
      "CA12\n",
      "Number of variants before clump: 79\n",
      "Number of variants after clump: 55\n",
      "SIRT2\n",
      "Number of variants before clump: 26\n",
      "Number of variants after clump: 15\n",
      "SORT1\n",
      "Number of variants before clump: 106\n",
      "Number of variants after clump: 37\n",
      "BMP4\n",
      "Number of variants before clump: 27\n",
      "Number of variants after clump: 12\n",
      "RTBDN\n",
      "Number of variants before clump: 38\n",
      "Number of variants after clump: 10\n",
      "FES\n",
      "Number of variants before clump: 6\n",
      "Number of variants after clump: 6\n",
      "INPP1\n",
      "Number of variants before clump: 11\n",
      "Number of variants after clump: 7\n",
      "PARP1\n",
      "Number of variants before clump: 14\n",
      "Number of variants after clump: 5\n",
      "CTSL\n",
      "Number of variants before clump: 102\n",
      "Number of variants after clump: 43\n",
      "IGSF8\n",
      "Number of variants before clump: 24\n",
      "Number of variants after clump: 16\n",
      "ITGAV\n",
      "Number of variants before clump: 37\n",
      "Number of variants after clump: 31\n",
      "TRIAP1\n",
      "Number of variants before clump: 56\n",
      "Number of variants after clump: 20\n",
      "GP1BA\n",
      "Number of variants before clump: 61\n",
      "Number of variants after clump: 48\n",
      "FGF21\n",
      "Number of variants before clump: 37\n",
      "Number of variants after clump: 16\n",
      "CES2\n",
      "Number of variants before clump: 16\n",
      "Number of variants after clump: 15\n",
      "TGFBR3\n",
      "Number of variants before clump: 24\n",
      "Number of variants after clump: 20\n",
      "TNFRSF4\n",
      "Number of variants before clump: 315\n",
      "Number of variants after clump: 83\n",
      "KYAT1\n",
      "Number of variants before clump: 32\n",
      "Number of variants after clump: 32\n",
      "DKK1\n",
      "Number of variants before clump: 60\n",
      "Number of variants after clump: 32\n",
      "SCGB3A2\n",
      "Number of variants before clump: 30\n",
      "Number of variants after clump: 25\n",
      "ADA2\n",
      "Number of variants before clump: 262\n",
      "Number of variants after clump: 88\n",
      "CCL21\n",
      "Number of variants before clump: 84\n",
      "Number of variants after clump: 34\n",
      "CD28\n",
      "Number of variants before clump: 62\n",
      "Number of variants after clump: 24\n",
      "UXS1\n",
      "Number of variants before clump: 19\n",
      "Number of variants after clump: 9\n",
      "EGFR\n",
      "Number of variants before clump: 48\n",
      "Number of variants after clump: 31\n",
      "TYMP\n",
      "Number of variants before clump: 34\n",
      "Number of variants after clump: 18\n",
      "BST2\n",
      "Number of variants before clump: 83\n",
      "Number of variants after clump: 41\n",
      "PTK7\n",
      "Number of variants before clump: 19\n",
      "Number of variants after clump: 19\n",
      "ARG1\n",
      "Number of variants before clump: 24\n",
      "Number of variants after clump: 10\n",
      "CRIP2\n",
      "Number of variants before clump: 15\n",
      "Number of variants after clump: 12\n",
      "ANGPTL2\n",
      "Number of variants before clump: 106\n",
      "Number of variants after clump: 55\n",
      "REG4\n",
      "Number of variants before clump: 21\n",
      "Number of variants after clump: 14\n",
      "PRSS2\n",
      "Number of variants before clump: 63\n",
      "Number of variants after clump: 41\n",
      "KRT5\n",
      "Number of variants before clump: 48\n",
      "Number of variants after clump: 48\n",
      "NCS1\n",
      "Number of variants before clump: 46\n",
      "Number of variants after clump: 37\n",
      "LHPP\n",
      "Number of variants before clump: 4\n",
      "Number of variants after clump: 4\n",
      "AXL\n",
      "Number of variants before clump: 58\n",
      "Number of variants after clump: 28\n",
      "ITGB2\n",
      "Number of variants before clump: 80\n",
      "Number of variants after clump: 30\n",
      "SLC39A5\n",
      "Number of variants before clump: 99\n",
      "Number of variants after clump: 57\n",
      "OMD\n",
      "Number of variants before clump: 166\n",
      "Number of variants after clump: 85\n",
      "ITGAM\n",
      "Number of variants before clump: 56\n",
      "Number of variants after clump: 26\n",
      "IL2RA\n",
      "Number of variants before clump: 253\n",
      "Number of variants after clump: 80\n",
      "DCTPP1\n",
      "Number of variants before clump: 26\n",
      "Number of variants after clump: 14\n",
      "ADGRE5\n",
      "Number of variants before clump: 88\n",
      "Number of variants after clump: 29\n",
      "PDGFA\n",
      "Number of variants before clump: 63\n",
      "Number of variants after clump: 30\n",
      "CSF3\n",
      "Number of variants before clump: 62\n",
      "Number of variants after clump: 31\n",
      "ODAM\n",
      "Number of variants before clump: 13\n",
      "Number of variants after clump: 13\n",
      "TFPI2\n",
      "Number of variants before clump: 68\n",
      "Number of variants after clump: 24\n",
      "LAG3\n",
      "Number of variants before clump: 104\n",
      "Number of variants after clump: 46\n",
      "TRIM21\n",
      "Number of variants before clump: 82\n",
      "Number of variants after clump: 37\n",
      "IGFBP3\n",
      "Number of variants before clump: 81\n",
      "Number of variants after clump: 43\n",
      "EDAR\n",
      "Number of variants before clump: 50\n",
      "Number of variants after clump: 42\n",
      "IL18BP\n",
      "Number of variants before clump: 125\n",
      "Number of variants after clump: 56\n",
      "TREML2\n",
      "Number of variants before clump: 42\n",
      "Number of variants after clump: 22\n",
      "PDGFC\n",
      "Number of variants before clump: 33\n",
      "Number of variants after clump: 13\n",
      "LILRA5\n",
      "Number of variants before clump: 71\n",
      "Number of variants after clump: 48\n",
      "FAS\n",
      "Number of variants before clump: 19\n",
      "Number of variants after clump: 16\n",
      "HSPG2\n",
      "Number of variants before clump: 56\n",
      "Number of variants after clump: 52\n",
      "HAGH\n",
      "Number of variants before clump: 20\n",
      "Number of variants after clump: 12\n",
      "VSTM2L\n",
      "Number of variants before clump: 20\n",
      "Number of variants after clump: 6\n",
      "DECR1\n",
      "Number of variants before clump: 21\n",
      "Number of variants after clump: 11\n",
      "CLTA\n",
      "Number of variants before clump: 5\n",
      "Number of variants after clump: 2\n",
      "CD164\n",
      "Number of variants before clump: 41\n",
      "Number of variants after clump: 22\n",
      "CA11\n",
      "Number of variants before clump: 28\n",
      "Number of variants after clump: 4\n",
      "LAIR1\n",
      "Number of variants before clump: 43\n",
      "Number of variants after clump: 24\n",
      "CLC\n",
      "Number of variants before clump: 105\n",
      "Number of variants after clump: 45\n",
      "APP\n",
      "Number of variants before clump: 55\n",
      "Number of variants after clump: 32\n",
      "VEGFC\n",
      "Number of variants before clump: 59\n",
      "Number of variants after clump: 31\n",
      "PMVK\n",
      "Number of variants before clump: 27\n",
      "Number of variants after clump: 16\n",
      "DTX3\n",
      "Number of variants before clump: 49\n",
      "Number of variants after clump: 32\n",
      "BCAN\n",
      "Number of variants before clump: 83\n",
      "Number of variants after clump: 46\n",
      "LCN2\n",
      "Number of variants before clump: 54\n",
      "Number of variants after clump: 34\n",
      "PCDH1\n",
      "Number of variants before clump: 35\n",
      "Number of variants after clump: 14\n",
      "CHAC2\n",
      "Number of variants before clump: 27\n",
      "Number of variants after clump: 20\n",
      "CALCA\n",
      "Number of variants before clump: 17\n",
      "Number of variants after clump: 10\n",
      "CDHR2\n",
      "Number of variants before clump: 45\n",
      "Number of variants after clump: 18\n",
      "ELOA\n",
      "Number of variants before clump: 20\n",
      "Number of variants after clump: 6\n",
      "SPARC\n",
      "Number of variants before clump: 56\n",
      "Number of variants after clump: 29\n",
      "TNFRSF14\n",
      "Number of variants before clump: 50\n",
      "Number of variants after clump: 24\n",
      "SEMA7A\n",
      "Number of variants before clump: 69\n",
      "Number of variants after clump: 32\n",
      "IDS\n",
      "Number of variants before clump: 23\n",
      "Number of variants after clump: 13\n",
      "MAP4K5\n",
      "Number of variants before clump: 29\n",
      "Number of variants after clump: 19\n",
      "LSP1\n",
      "Number of variants before clump: 44\n",
      "Number of variants after clump: 7\n",
      "GZMH\n",
      "Number of variants before clump: 222\n",
      "Number of variants after clump: 90\n",
      "THPO\n",
      "Number of variants before clump: 74\n",
      "Number of variants after clump: 38\n",
      "TNFSF14\n",
      "Number of variants before clump: 34\n",
      "Number of variants after clump: 14\n",
      "OSM\n",
      "Number of variants before clump: 96\n",
      "Number of variants after clump: 57\n",
      "CCN5\n",
      "Number of variants before clump: 10\n",
      "Number of variants after clump: 10\n",
      "PARK7\n",
      "Number of variants before clump: 19\n",
      "Number of variants after clump: 10\n",
      "EGF\n",
      "Number of variants before clump: 34\n",
      "Number of variants after clump: 17\n",
      "CPM\n",
      "Number of variants before clump: 60\n",
      "Number of variants after clump: 43\n",
      "TIMP1\n",
      "Number of variants before clump: 46\n",
      "Number of variants after clump: 20\n",
      "ADAMTS15\n",
      "Number of variants before clump: 26\n",
      "Number of variants after clump: 17\n",
      "MAPK9\n",
      "Number of variants before clump: 3\n",
      "Number of variants after clump: 3\n",
      "ICOSLG\n",
      "Number of variants before clump: 52\n",
      "Number of variants after clump: 23\n",
      "SERPINE1\n",
      "Number of variants before clump: 72\n",
      "Number of variants after clump: 43\n",
      "FLT3LG\n",
      "Number of variants before clump: 59\n",
      "Number of variants after clump: 28\n",
      "FOLR1\n",
      "Number of variants before clump: 45\n",
      "Number of variants after clump: 16\n",
      "EDIL3\n",
      "Number of variants before clump: 20\n",
      "Number of variants after clump: 16\n",
      "HEBP1\n",
      "Number of variants before clump: 2\n",
      "Number of variants after clump: 2\n",
      "PDGFB\n",
      "Number of variants before clump: 57\n",
      "Number of variants after clump: 27\n",
      "KLK6\n",
      "Number of variants before clump: 15\n",
      "Number of variants after clump: 9\n",
      "PLA2G1B\n",
      "Number of variants before clump: 77\n",
      "Number of variants after clump: 45\n",
      "SEMA4C\n",
      "Number of variants before clump: 39\n",
      "Number of variants after clump: 20\n",
      "STC1\n",
      "Number of variants before clump: 16\n",
      "Number of variants after clump: 4\n",
      "FST\n",
      "Number of variants before clump: 17\n",
      "Number of variants after clump: 3\n",
      "SERPINB6\n",
      "Number of variants before clump: 11\n",
      "Number of variants after clump: 6\n",
      "PHOSPHO1\n",
      "Number of variants before clump: 67\n",
      "Number of variants after clump: 37\n",
      "BIN2\n",
      "Number of variants before clump: 27\n",
      "Number of variants after clump: 17\n",
      "WAS\n",
      "Number of variants before clump: 15\n",
      "Number of variants after clump: 6\n",
      "NOS1\n",
      "Number of variants before clump: 10\n",
      "Number of variants after clump: 5\n",
      "HSPA1A\n",
      "Number of variants before clump: 109\n",
      "Number of variants after clump: 63\n",
      "PKLR\n",
      "Number of variants before clump: 50\n",
      "Number of variants after clump: 34\n",
      "VEGFA\n",
      "Number of variants before clump: 43\n",
      "Number of variants after clump: 31\n",
      "AHSP\n",
      "Number of variants before clump: 64\n",
      "Number of variants after clump: 62\n",
      "DEFA1_DEFA1B\n",
      "Number of variants before clump: 62\n",
      "Number of variants after clump: 30\n",
      "PLXNB3\n",
      "Number of variants before clump: 47\n",
      "Number of variants after clump: 36\n",
      "PTPRF\n",
      "Number of variants before clump: 103\n",
      "Number of variants after clump: 59\n",
      "ITGB1\n",
      "Number of variants before clump: 38\n",
      "Number of variants after clump: 16\n",
      "EPHX2\n",
      "Number of variants before clump: 8\n",
      "Number of variants after clump: 8\n",
      "TNFRSF12A\n",
      "Number of variants before clump: 20\n",
      "Number of variants after clump: 15\n",
      "COX5B\n",
      "Number of variants before clump: 9\n",
      "Number of variants after clump: 9\n",
      "CCDC80\n",
      "Number of variants before clump: 22\n",
      "Number of variants after clump: 16\n",
      "S100P\n",
      "Number of variants before clump: 46\n",
      "Number of variants after clump: 9\n",
      "KRT18\n",
      "Number of variants before clump: 26\n",
      "Number of variants after clump: 9\n",
      "CA3\n",
      "Number of variants before clump: 44\n",
      "Number of variants after clump: 29\n",
      "MANSC1\n",
      "Number of variants before clump: 169\n",
      "Number of variants after clump: 76\n",
      "PRKRA\n",
      "Number of variants before clump: 26\n",
      "Number of variants after clump: 12\n",
      "MCFD2\n",
      "Number of variants before clump: 18\n",
      "Number of variants after clump: 10\n",
      "TFF2\n",
      "Number of variants before clump: 59\n",
      "Number of variants after clump: 20\n",
      "IFNLR1\n",
      "Number of variants before clump: 15\n",
      "Number of variants after clump: 8\n",
      "TNF\n",
      "Number of variants before clump: 88\n",
      "Number of variants after clump: 50\n",
      "MDK\n",
      "Number of variants before clump: 49\n",
      "Number of variants after clump: 22\n",
      "CCN2\n",
      "Number of variants before clump: 51\n",
      "Number of variants after clump: 26\n",
      "TNFSF11\n",
      "Number of variants before clump: 33\n",
      "Number of variants after clump: 24\n",
      "TNFRSF13B\n",
      "Number of variants before clump: 58\n",
      "Number of variants after clump: 32\n",
      "MEPE\n",
      "Number of variants before clump: 32\n",
      "Number of variants after clump: 23\n",
      "TMSB10\n",
      "Number of variants before clump: 11\n",
      "Number of variants after clump: 7\n",
      "NID1\n",
      "Number of variants before clump: 59\n",
      "Number of variants after clump: 34\n",
      "RRM2B\n",
      "Number of variants before clump: 15\n",
      "Number of variants after clump: 10\n",
      "ATP5IF1\n",
      "Number of variants before clump: 43\n",
      "Number of variants after clump: 16\n",
      "DLL1\n",
      "Number of variants before clump: 83\n",
      "Number of variants after clump: 36\n",
      "CAMKK1\n",
      "Number of variants before clump: 5\n",
      "Number of variants after clump: 5\n",
      "COL1A1\n",
      "Number of variants before clump: 53\n",
      "Number of variants after clump: 37\n",
      "PRSS8\n",
      "Number of variants before clump: 89\n",
      "Number of variants after clump: 35\n",
      "MZB1\n",
      "Number of variants before clump: 60\n",
      "Number of variants after clump: 19\n",
      "F2R\n",
      "Number of variants before clump: 21\n",
      "Number of variants after clump: 12\n",
      "OLR1\n",
      "Number of variants before clump: 53\n",
      "Number of variants after clump: 29\n",
      "NPDC1\n",
      "Number of variants before clump: 50\n",
      "Number of variants after clump: 16\n",
      "RETN\n",
      "Number of variants before clump: 69\n",
      "Number of variants after clump: 46\n",
      "HGF\n",
      "Number of variants before clump: 27\n",
      "Number of variants after clump: 12\n",
      "TP53INP1\n",
      "Number of variants before clump: 8\n",
      "Number of variants after clump: 3\n",
      "MIF\n",
      "Number of variants before clump: 23\n",
      "Number of variants after clump: 13\n",
      "ICA1\n",
      "Number of variants before clump: 31\n",
      "Number of variants after clump: 27\n",
      "HMBS\n",
      "Number of variants before clump: 16\n",
      "Number of variants after clump: 16\n",
      "SORD\n",
      "Number of variants before clump: 29\n",
      "Number of variants after clump: 25\n",
      "RABEPK\n",
      "Number of variants before clump: 8\n",
      "Number of variants after clump: 8\n",
      "PLAT\n",
      "Number of variants before clump: 24\n",
      "Number of variants after clump: 8\n",
      "SCRN1\n",
      "Number of variants before clump: 8\n",
      "Number of variants after clump: 7\n",
      "CEP20\n",
      "Number of variants before clump: 11\n",
      "Number of variants after clump: 8\n",
      "GBP2\n",
      "Number of variants before clump: 6\n",
      "Number of variants after clump: 6\n",
      "WIF1\n",
      "Number of variants before clump: 68\n",
      "Number of variants after clump: 36\n",
      "TNFRSF9\n",
      "Number of variants before clump: 211\n",
      "Number of variants after clump: 76\n",
      "CASP3\n",
      "Number of variants before clump: 23\n",
      "Number of variants after clump: 15\n",
      "CD63\n",
      "Number of variants before clump: 59\n",
      "Number of variants after clump: 29\n",
      "HMOX2\n",
      "Number of variants before clump: 29\n",
      "Number of variants after clump: 23\n",
      "EPHA2\n",
      "Number of variants before clump: 163\n",
      "Number of variants after clump: 68\n",
      "FGFR2\n",
      "Number of variants before clump: 36\n",
      "Number of variants after clump: 14\n",
      "ERBB4\n",
      "Number of variants before clump: 88\n",
      "Number of variants after clump: 45\n",
      "ST6GAL1\n",
      "Number of variants before clump: 17\n",
      "Number of variants after clump: 13\n",
      "ADM\n",
      "Number of variants before clump: 77\n",
      "Number of variants after clump: 38\n",
      "MVK\n",
      "Number of variants before clump: 17\n",
      "Number of variants after clump: 13\n",
      "SIT1\n",
      "Number of variants before clump: 15\n",
      "Number of variants after clump: 10\n",
      "DFFA\n",
      "Number of variants before clump: 14\n",
      "Number of variants after clump: 10\n",
      "XCL1\n",
      "Number of variants before clump: 61\n",
      "Number of variants after clump: 21\n",
      "ANXA3\n",
      "Number of variants before clump: 32\n",
      "Number of variants after clump: 22\n",
      "SEMA3F\n",
      "Number of variants before clump: 146\n",
      "Number of variants after clump: 113\n",
      "SLC16A1\n",
      "Number of variants before clump: 10\n",
      "Number of variants after clump: 7\n",
      "COMP\n",
      "Number of variants before clump: 52\n",
      "Number of variants after clump: 27\n",
      "CALCOCO1\n",
      "Number of variants before clump: 8\n",
      "Number of variants after clump: 4\n",
      "TNR\n",
      "Number of variants before clump: 115\n",
      "Number of variants after clump: 45\n",
      "GFAP\n",
      "Number of variants before clump: 27\n",
      "Number of variants after clump: 13\n",
      "IFNL1\n",
      "Number of variants before clump: 26\n",
      "Number of variants after clump: 23\n",
      "SEMA4D\n",
      "Number of variants before clump: 18\n",
      "Number of variants after clump: 11\n",
      "SULT1A1\n",
      "Number of variants before clump: 39\n",
      "Number of variants after clump: 12\n",
      "KIT\n",
      "Number of variants before clump: 118\n",
      "Number of variants after clump: 71\n",
      "AMIGO2\n",
      "Number of variants before clump: 67\n",
      "Number of variants after clump: 33\n",
      "HCLS1\n",
      "Number of variants before clump: 34\n",
      "Number of variants after clump: 29\n",
      "SLC39A14\n",
      "Number of variants before clump: 23\n",
      "Number of variants after clump: 12\n",
      "CFC1\n",
      "Number of variants before clump: 71\n",
      "Number of variants after clump: 37\n",
      "REG3A\n",
      "Number of variants before clump: 62\n",
      "Number of variants after clump: 38\n",
      "ADH4\n",
      "Number of variants before clump: 27\n",
      "Number of variants after clump: 13\n",
      "CDON\n",
      "Number of variants before clump: 24\n",
      "Number of variants after clump: 16\n",
      "LTBP2\n",
      "Number of variants before clump: 69\n",
      "Number of variants after clump: 24\n",
      "GKN1\n",
      "Number of variants before clump: 21\n",
      "Number of variants after clump: 5\n",
      "PSME2\n",
      "Number of variants before clump: 10\n",
      "Number of variants after clump: 10\n",
      "LPCAT2\n",
      "Number of variants before clump: 5\n",
      "Number of variants after clump: 5\n",
      "PTPRS\n",
      "Number of variants before clump: 38\n",
      "Number of variants after clump: 18\n",
      "DARS1\n",
      "Number of variants before clump: 25\n",
      "Number of variants after clump: 14\n",
      "PEBP1\n",
      "Number of variants before clump: 5\n",
      "Number of variants after clump: 5\n",
      "SFRP1\n",
      "Number of variants before clump: 20\n",
      "Number of variants after clump: 15\n",
      "APBB1IP\n",
      "Number of variants before clump: 51\n",
      "Number of variants after clump: 15\n",
      "LBR\n",
      "Number of variants before clump: 25\n",
      "Number of variants after clump: 16\n",
      "FGF23\n",
      "Number of variants before clump: 12\n",
      "Number of variants after clump: 11\n",
      "CDH2\n",
      "Number of variants before clump: 41\n",
      "Number of variants after clump: 26\n",
      "IGFBP6\n",
      "Number of variants before clump: 27\n",
      "Number of variants after clump: 14\n",
      "COL9A1\n",
      "Number of variants before clump: 11\n",
      "Number of variants after clump: 11\n",
      "OMG\n",
      "Number of variants before clump: 67\n",
      "Number of variants after clump: 34\n",
      "S100A16\n",
      "Number of variants before clump: 20\n",
      "Number of variants after clump: 13\n",
      "DCXR\n",
      "Number of variants before clump: 42\n",
      "Number of variants after clump: 30\n",
      "PPP3R1\n",
      "Number of variants before clump: 27\n",
      "Number of variants after clump: 16\n",
      "CDH3\n",
      "Number of variants before clump: 38\n",
      "Number of variants after clump: 18\n",
      "PRDX6\n",
      "Number of variants before clump: 7\n",
      "Number of variants after clump: 7\n",
      "CBLN4\n",
      "Number of variants before clump: 12\n",
      "Number of variants after clump: 7\n",
      "FKBP5\n",
      "Number of variants before clump: 23\n",
      "Number of variants after clump: 18\n",
      "CNTN5\n",
      "Number of variants before clump: 28\n",
      "Number of variants after clump: 14\n",
      "AIF1\n",
      "Number of variants before clump: 113\n",
      "Number of variants after clump: 50\n",
      "RTN4R\n",
      "Number of variants before clump: 40\n",
      "Number of variants after clump: 32\n",
      "NTRK3\n",
      "Number of variants before clump: 22\n",
      "Number of variants after clump: 16\n",
      "WNT9A\n",
      "Number of variants before clump: 61\n",
      "Number of variants after clump: 47\n",
      "EREG\n",
      "Number of variants before clump: 26\n",
      "Number of variants after clump: 16\n",
      "CNTN3\n",
      "Number of variants before clump: 77\n",
      "Number of variants after clump: 47\n",
      "PADI2\n",
      "Number of variants before clump: 17\n",
      "Number of variants after clump: 2\n",
      "MAVS\n",
      "Number of variants before clump: 10\n",
      "Number of variants after clump: 4\n",
      "SRP14\n",
      "Number of variants before clump: 12\n",
      "Number of variants after clump: 6\n",
      "SPP1\n",
      "Number of variants before clump: 33\n",
      "Number of variants after clump: 22\n",
      "EZR\n",
      "Number of variants before clump: 19\n",
      "Number of variants after clump: 8\n",
      "FLT1\n",
      "Number of variants before clump: 28\n",
      "Number of variants after clump: 17\n",
      "FUS\n",
      "Number of variants before clump: 3\n",
      "Number of variants after clump: 2\n",
      "AARSD1\n",
      "Number of variants before clump: 25\n",
      "Number of variants after clump: 25\n",
      "UBAC1\n",
      "Number of variants before clump: 33\n",
      "Number of variants after clump: 32\n",
      "ACAA1\n",
      "Number of variants before clump: 14\n",
      "Number of variants after clump: 14\n",
      "ADGRG1\n",
      "Number of variants before clump: 40\n",
      "Number of variants after clump: 23\n",
      "SPON1\n",
      "Number of variants before clump: 111\n",
      "Number of variants after clump: 54\n",
      "CRIM1\n",
      "Number of variants before clump: 27\n",
      "Number of variants after clump: 15\n",
      "EFEMP1\n",
      "Number of variants before clump: 18\n",
      "Number of variants after clump: 11\n",
      "BAG3\n",
      "Number of variants before clump: 22\n",
      "Number of variants after clump: 15\n",
      "ADAMTS16\n",
      "Number of variants before clump: 8\n",
      "Number of variants after clump: 8\n",
      "SOST\n",
      "Number of variants before clump: 24\n",
      "Number of variants after clump: 22\n",
      "CD274\n",
      "Number of variants before clump: 12\n",
      "Number of variants after clump: 11\n",
      "SH2D1A\n",
      "Number of variants before clump: 10\n",
      "Number of variants after clump: 7\n",
      "BSG\n",
      "Number of variants before clump: 63\n",
      "Number of variants after clump: 34\n",
      "SPRY2\n",
      "Number of variants before clump: 16\n",
      "Number of variants after clump: 12\n",
      "ATOX1\n",
      "Number of variants before clump: 15\n",
      "Number of variants after clump: 7\n",
      "FLT3\n",
      "Number of variants before clump: 15\n",
      "Number of variants after clump: 10\n",
      "AGRP\n",
      "Number of variants before clump: 74\n",
      "Number of variants after clump: 47\n",
      "BAIAP2\n",
      "Number of variants before clump: 23\n",
      "Number of variants after clump: 11\n",
      "BGN\n",
      "Number of variants before clump: 124\n",
      "Number of variants after clump: 73\n",
      "CHMP1A\n",
      "Number of variants before clump: 34\n",
      "Number of variants after clump: 22\n",
      "IGSF3\n",
      "Number of variants before clump: 30\n",
      "Number of variants after clump: 19\n",
      "TXNDC5\n",
      "Number of variants before clump: 11\n",
      "Number of variants after clump: 10\n",
      "MB\n",
      "Number of variants before clump: 63\n",
      "Number of variants after clump: 53\n",
      "PAG1\n",
      "Number of variants before clump: 11\n",
      "Number of variants after clump: 9\n",
      "CD69\n",
      "Number of variants before clump: 17\n",
      "Number of variants after clump: 6\n",
      "STAMBP\n",
      "Number of variants before clump: 19\n",
      "Number of variants after clump: 10\n",
      "ADGRG2\n",
      "Number of variants before clump: 14\n",
      "Number of variants after clump: 12\n",
      "TGFA\n",
      "Number of variants before clump: 29\n",
      "Number of variants after clump: 9\n",
      "SCG2\n",
      "Number of variants before clump: 26\n",
      "Number of variants after clump: 13\n",
      "CXCL13\n",
      "Number of variants before clump: 19\n",
      "Number of variants after clump: 13\n",
      "EPO\n",
      "Number of variants before clump: 105\n",
      "Number of variants after clump: 58\n",
      "IL20RA\n",
      "Number of variants before clump: 2\n",
      "Number of variants after clump: 2\n",
      "RILP\n",
      "Number of variants before clump: 20\n",
      "Number of variants after clump: 14\n",
      "FKBP7\n",
      "Number of variants before clump: 17\n",
      "Number of variants after clump: 17\n",
      "SORCS2\n",
      "Number of variants before clump: 16\n",
      "Number of variants after clump: 12\n",
      "PLIN1\n",
      "Number of variants before clump: 1\n",
      "Number of variants after clump: 1\n",
      "APRT\n",
      "Number of variants before clump: 24\n",
      "Number of variants after clump: 22\n",
      "LY96\n",
      "Number of variants before clump: 77\n",
      "Number of variants after clump: 43\n",
      "APLP1\n",
      "Number of variants before clump: 13\n",
      "Number of variants after clump: 4\n",
      "BID\n",
      "Number of variants before clump: 15\n",
      "Number of variants after clump: 10\n",
      "RHOC\n",
      "Number of variants before clump: 7\n",
      "Number of variants after clump: 7\n",
      "TGM2\n",
      "Number of variants before clump: 150\n",
      "Number of variants after clump: 57\n",
      "DAPP1\n",
      "Number of variants before clump: 5\n",
      "Number of variants after clump: 4\n",
      "TNFRSF1A\n",
      "Number of variants before clump: 105\n",
      "Number of variants after clump: 53\n",
      "KYNU\n",
      "Number of variants before clump: 63\n",
      "Number of variants after clump: 33\n",
      "PPY\n",
      "Number of variants before clump: 14\n",
      "Number of variants after clump: 9\n",
      "CCN1\n",
      "Number of variants before clump: 44\n",
      "Number of variants after clump: 29\n",
      "ANXA5\n",
      "Number of variants before clump: 2\n",
      "Number of variants after clump: 2\n",
      "DSG4\n",
      "Number of variants before clump: 68\n",
      "Number of variants after clump: 42\n",
      "CCL20\n",
      "Number of variants before clump: 34\n",
      "Number of variants after clump: 14\n",
      "CXCL17\n",
      "Number of variants before clump: 34\n",
      "Number of variants after clump: 14\n",
      "SKAP1\n",
      "Number of variants before clump: 33\n",
      "Number of variants after clump: 23\n",
      "IGFBP1\n",
      "Number of variants before clump: 18\n",
      "Number of variants after clump: 3\n",
      "PSME1\n",
      "Number of variants before clump: 10\n",
      "Number of variants after clump: 10\n",
      "RBP2\n",
      "Number of variants before clump: 60\n",
      "Number of variants after clump: 30\n",
      "PTPN1\n",
      "Number of variants before clump: 9\n",
      "Number of variants after clump: 4\n",
      "VTCN1\n",
      "Number of variants before clump: 17\n",
      "Number of variants after clump: 5\n",
      "MSTN\n",
      "Number of variants before clump: 52\n",
      "Number of variants after clump: 38\n",
      "SLAMF1\n",
      "Number of variants before clump: 33\n",
      "Number of variants after clump: 16\n",
      "TBC1D23\n",
      "Number of variants before clump: 8\n",
      "Number of variants after clump: 3\n",
      "CA1\n",
      "Number of variants before clump: 14\n",
      "Number of variants after clump: 11\n",
      "JCHAIN\n",
      "Number of variants before clump: 95\n",
      "Number of variants after clump: 40\n",
      "ROBO1\n",
      "Number of variants before clump: 106\n",
      "Number of variants after clump: 61\n",
      "GFRA3\n",
      "Number of variants before clump: 49\n",
      "Number of variants after clump: 40\n",
      "RWDD1\n",
      "Number of variants before clump: 21\n",
      "Number of variants after clump: 9\n",
      "NT5C3A\n",
      "Number of variants before clump: 17\n",
      "Number of variants after clump: 6\n",
      "WFDC2\n",
      "Number of variants before clump: 28\n",
      "Number of variants after clump: 7\n",
      "CRLF1\n",
      "Number of variants before clump: 49\n",
      "Number of variants after clump: 14\n",
      "NBN\n",
      "Number of variants before clump: 4\n",
      "Number of variants after clump: 4\n",
      "PRL\n",
      "Number of variants before clump: 4\n",
      "Number of variants after clump: 4\n",
      "TST\n",
      "Number of variants before clump: 13\n",
      "Number of variants after clump: 9\n",
      "PTN\n",
      "Number of variants before clump: 17\n",
      "Number of variants after clump: 9\n",
      "NFATC3\n",
      "Number of variants before clump: 38\n",
      "Number of variants after clump: 21\n",
      "C19orf12\n",
      "Number of variants before clump: 13\n",
      "Number of variants after clump: 6\n",
      "IL7\n",
      "Number of variants before clump: 7\n",
      "Number of variants after clump: 6\n",
      "SCAMP3\n",
      "Number of variants before clump: 31\n",
      "Number of variants after clump: 12\n",
      "CARHSP1\n",
      "Number of variants before clump: 2\n",
      "Number of variants after clump: 2\n",
      "PLIN3\n",
      "Number of variants before clump: 10\n",
      "Number of variants after clump: 7\n",
      "PCOLCE\n",
      "Number of variants before clump: 22\n",
      "Number of variants after clump: 3\n",
      "CEP43\n",
      "Number of variants before clump: 17\n",
      "Number of variants after clump: 16\n",
      "CC2D1A\n",
      "Number of variants before clump: 12\n",
      "Number of variants after clump: 8\n",
      "RASSF2\n",
      "Number of variants before clump: 3\n",
      "Number of variants after clump: 2\n",
      "HEXIM1\n",
      "Number of variants before clump: 27\n",
      "Number of variants after clump: 14\n",
      "SULT2A1\n",
      "Number of variants before clump: 13\n",
      "Number of variants after clump: 5\n",
      "WASF1\n",
      "Number of variants before clump: 9\n",
      "Number of variants after clump: 9\n",
      "SLC27A4\n",
      "Number of variants before clump: 9\n",
      "Number of variants after clump: 6\n",
      "IL1A\n",
      "Number of variants before clump: 13\n",
      "Number of variants after clump: 10\n",
      "ACOX1\n",
      "Number of variants before clump: 16\n",
      "Number of variants after clump: 16\n",
      "ESM1\n",
      "Number of variants before clump: 40\n",
      "Number of variants after clump: 19\n",
      "REN\n",
      "Number of variants before clump: 10\n",
      "Number of variants after clump: 7\n",
      "RGMB\n",
      "Number of variants before clump: 16\n",
      "Number of variants after clump: 11\n",
      "TXNRD1\n",
      "Number of variants before clump: 14\n",
      "Number of variants after clump: 6\n",
      "SLAMF6\n",
      "Number of variants before clump: 5\n",
      "Number of variants after clump: 4\n",
      "EGLN1\n",
      "Number of variants before clump: 22\n",
      "Number of variants after clump: 16\n",
      "IRAK4\n",
      "Number of variants before clump: 18\n",
      "Number of variants after clump: 7\n",
      "ZBTB16\n",
      "Number of variants before clump: 36\n",
      "Number of variants after clump: 18\n",
      "ARHGAP1\n",
      "Number of variants before clump: 35\n",
      "Number of variants after clump: 22\n",
      "PNPT1\n",
      "Number of variants before clump: 5\n",
      "Number of variants after clump: 5\n",
      "SH2B3\n",
      "Number of variants before clump: 18\n",
      "Number of variants after clump: 6\n",
      "NSFL1C\n",
      "Number of variants before clump: 14\n",
      "Number of variants after clump: 8\n",
      "CRADD\n",
      "Number of variants before clump: 21\n",
      "Number of variants after clump: 10\n",
      "STC2\n",
      "Number of variants before clump: 52\n",
      "Number of variants after clump: 17\n",
      "USP8\n",
      "Number of variants before clump: 9\n",
      "Number of variants after clump: 9\n",
      "MMP13\n",
      "Number of variants before clump: 9\n",
      "Number of variants after clump: 5\n",
      "CD93\n",
      "Number of variants before clump: 25\n",
      "Number of variants after clump: 10\n",
      "BANK1\n",
      "Number of variants before clump: 18\n",
      "Number of variants after clump: 8\n",
      "MPIG6B\n",
      "Number of variants before clump: 13\n",
      "Number of variants after clump: 3\n",
      "ANXA11\n",
      "Number of variants before clump: 75\n",
      "Number of variants after clump: 31\n",
      "FGR\n",
      "Number of variants before clump: 10\n",
      "Number of variants after clump: 8\n",
      "METAP1D\n",
      "Number of variants before clump: 14\n",
      "Number of variants after clump: 7\n",
      "NTF3\n",
      "Number of variants before clump: 3\n",
      "Number of variants after clump: 3\n",
      "HSPB6\n",
      "Number of variants before clump: 24\n",
      "Number of variants after clump: 10\n",
      "CANT1\n",
      "Number of variants before clump: 45\n",
      "Number of variants after clump: 21\n",
      "GFER\n",
      "Number of variants before clump: 33\n",
      "Number of variants after clump: 17\n",
      "DNMBP\n",
      "Number of variants before clump: 8\n",
      "Number of variants after clump: 5\n",
      "ENAH\n",
      "Number of variants before clump: 10\n",
      "Number of variants after clump: 4\n",
      "COL6A3\n",
      "Number of variants before clump: 10\n",
      "Number of variants after clump: 8\n",
      "DNAJB1\n",
      "Number of variants before clump: 36\n",
      "Number of variants after clump: 24\n",
      "NUDT5\n",
      "Number of variants before clump: 14\n",
      "Number of variants after clump: 10\n",
      "IGFBP2\n",
      "Number of variants before clump: 18\n",
      "Number of variants after clump: 13\n",
      "P4HB\n",
      "Number of variants before clump: 16\n",
      "Number of variants after clump: 16\n",
      "VAMP5\n",
      "Number of variants before clump: 14\n",
      "Number of variants after clump: 5\n",
      "CDC27\n",
      "Number of variants before clump: 2\n",
      "Number of variants after clump: 2\n",
      "CD59\n",
      "Number of variants before clump: 15\n",
      "Number of variants after clump: 10\n",
      "CXCL14\n",
      "Number of variants before clump: 8\n",
      "Number of variants after clump: 6\n",
      "PSIP1\n",
      "Number of variants before clump: 20\n",
      "Number of variants after clump: 11\n",
      "CRACR2A\n",
      "Number of variants before clump: 12\n",
      "Number of variants after clump: 9\n",
      "LAT\n",
      "Number of variants before clump: 22\n",
      "Number of variants after clump: 8\n",
      "AGR3\n",
      "Number of variants before clump: 33\n",
      "Number of variants after clump: 24\n",
      "CD46\n",
      "Number of variants before clump: 23\n",
      "Number of variants after clump: 12\n",
      "LYN\n",
      "Number of variants before clump: 21\n",
      "Number of variants after clump: 8\n",
      "TNFAIP8\n",
      "Number of variants before clump: 2\n",
      "Number of variants after clump: 2\n",
      "IL17C\n",
      "Number of variants before clump: 9\n",
      "Number of variants after clump: 7\n",
      "GPC5\n",
      "Number of variants before clump: 26\n",
      "Number of variants after clump: 9\n",
      "BIRC2\n",
      "Number of variants before clump: 7\n",
      "Number of variants after clump: 4\n",
      "CXCL3\n",
      "Number of variants before clump: 40\n",
      "Number of variants after clump: 25\n",
      "STIP1\n",
      "Number of variants before clump: 9\n",
      "Number of variants after clump: 5\n",
      "TPPP3\n",
      "Number of variants before clump: 14\n",
      "Number of variants after clump: 9\n",
      "HPCAL1\n",
      "Number of variants before clump: 14\n",
      "Number of variants after clump: 5\n",
      "DEFB4A_DEFB4B\n",
      "Number of variants before clump: 112\n",
      "Number of variants after clump: 60\n",
      "DRG2\n",
      "Number of variants before clump: 16\n",
      "Number of variants after clump: 6\n",
      "DCTN1\n",
      "Number of variants before clump: 18\n",
      "Number of variants after clump: 11\n",
      "ANXA4\n",
      "Number of variants before clump: 17\n",
      "Number of variants after clump: 6\n",
      "MITD1\n",
      "Number of variants before clump: 16\n",
      "Number of variants after clump: 8\n",
      "TAFA5\n",
      "Number of variants before clump: 8\n",
      "Number of variants after clump: 3\n",
      "GDNF\n",
      "Number of variants before clump: 7\n",
      "Number of variants after clump: 5\n",
      "FKBP4\n",
      "Number of variants before clump: 18\n",
      "Number of variants after clump: 8\n",
      "ROBO2\n",
      "Number of variants before clump: 22\n",
      "Number of variants after clump: 17\n",
      "DAG1\n",
      "Number of variants before clump: 40\n",
      "Number of variants after clump: 31\n",
      "GCG\n",
      "Number of variants before clump: 18\n",
      "Number of variants after clump: 12\n",
      "CETN2\n",
      "Number of variants before clump: 15\n",
      "Number of variants after clump: 15\n",
      "TSHB\n",
      "Number of variants before clump: 40\n",
      "Number of variants after clump: 29\n",
      "XRCC4\n",
      "Number of variants before clump: 3\n",
      "Number of variants after clump: 3\n",
      "FYB1\n",
      "Number of variants before clump: 21\n",
      "Number of variants after clump: 9\n",
      "AGR2\n",
      "Number of variants before clump: 41\n",
      "Number of variants after clump: 21\n",
      "TSPAN1\n",
      "Number of variants before clump: 26\n",
      "Number of variants after clump: 8\n",
      "PSPN\n",
      "Number of variants before clump: 25\n",
      "Number of variants after clump: 20\n",
      "CLSPN\n",
      "Number of variants before clump: 2\n",
      "Number of variants after clump: 2\n",
      "AREG\n",
      "Number of variants before clump: 42\n",
      "Number of variants after clump: 19\n",
      "IKZF2\n",
      "Number of variants before clump: 57\n",
      "Number of variants after clump: 23\n",
      "CALB2\n",
      "Number of variants before clump: 1\n",
      "Number of variants after clump: 1\n",
      "IL18RAP\n",
      "Number of variants before clump: 28\n",
      "Number of variants after clump: 15\n",
      "SRPK2\n",
      "Number of variants before clump: 12\n",
      "Number of variants after clump: 6\n",
      "PRDX1\n",
      "Number of variants before clump: 12\n",
      "Number of variants after clump: 12\n",
      "ZBTB17\n",
      "Number of variants before clump: 7\n",
      "Number of variants after clump: 3\n",
      "IRAG2\n",
      "Number of variants before clump: 3\n",
      "Number of variants after clump: 2\n",
      "TBL1X\n",
      "Number of variants before clump: 13\n",
      "Number of variants after clump: 4\n",
      "SERPINB5\n",
      "Number of variants before clump: 5\n",
      "Number of variants after clump: 5\n",
      "SESTD1\n",
      "Number of variants before clump: 4\n",
      "Number of variants after clump: 3\n",
      "PLXNA4\n",
      "Number of variants before clump: 22\n",
      "Number of variants after clump: 9\n",
      "PRKAR1A\n",
      "Number of variants before clump: 4\n",
      "Number of variants after clump: 3\n",
      "NOS3\n",
      "Number of variants before clump: 11\n",
      "Number of variants after clump: 10\n",
      "NTRK2\n",
      "Number of variants before clump: 24\n",
      "Number of variants after clump: 12\n",
      "HTRA2\n",
      "Number of variants before clump: 26\n",
      "Number of variants after clump: 6\n",
      "RABGAP1L\n",
      "Number of variants before clump: 5\n",
      "Number of variants after clump: 5\n",
      "SPOCK1\n",
      "Number of variants before clump: 57\n",
      "Number of variants after clump: 32\n",
      "F11R\n",
      "Number of variants before clump: 29\n",
      "Number of variants after clump: 13\n",
      "STX4\n",
      "Number of variants before clump: 7\n",
      "Number of variants after clump: 3\n",
      "PRKAB1\n",
      "Number of variants before clump: 14\n",
      "Number of variants after clump: 11\n",
      "POLR2F\n",
      "Number of variants before clump: 6\n",
      "Number of variants after clump: 4\n",
      "TIGAR\n",
      "Number of variants before clump: 3\n",
      "Number of variants after clump: 3\n",
      "PLA2G4A\n",
      "Number of variants before clump: 12\n",
      "Number of variants after clump: 6\n",
      "YES1\n",
      "Number of variants before clump: 13\n",
      "Number of variants after clump: 8\n",
      "DOK2\n",
      "Number of variants before clump: 19\n",
      "Number of variants after clump: 9\n",
      "CEBPB\n",
      "Number of variants before clump: 34\n",
      "Number of variants after clump: 9\n",
      "CEP85\n",
      "Number of variants before clump: 21\n",
      "Number of variants after clump: 16\n",
      "NBL1\n",
      "Number of variants before clump: 28\n",
      "Number of variants after clump: 15\n",
      "AMBN\n",
      "Number of variants before clump: 5\n",
      "Number of variants after clump: 5\n",
      "HAO1\n",
      "Number of variants before clump: 23\n",
      "Number of variants after clump: 14\n",
      "SERPINB9\n",
      "Number of variants before clump: 7\n",
      "Number of variants after clump: 4\n",
      "CEACAM3\n",
      "Number of variants before clump: 1\n",
      "Number of variants after clump: 1\n",
      "GSAP\n",
      "Number of variants before clump: 5\n",
      "Number of variants after clump: 5\n",
      "TP53\n",
      "Number of variants before clump: 2\n",
      "Number of variants after clump: 1\n",
      "MANF\n",
      "Number of variants before clump: 19\n",
      "Number of variants after clump: 7\n",
      "TBCB\n",
      "Number of variants before clump: 19\n",
      "Number of variants after clump: 8\n",
      "CGA\n",
      "Number of variants before clump: 6\n",
      "Number of variants after clump: 6\n",
      "NFATC1\n",
      "Number of variants before clump: 5\n",
      "Number of variants after clump: 4\n",
      "DCN\n",
      "Number of variants before clump: 15\n",
      "Number of variants after clump: 4\n",
      "FSTL3\n",
      "Number of variants before clump: 13\n",
      "Number of variants after clump: 8\n",
      "MPHOSPH8\n",
      "Number of variants before clump: 17\n",
      "Number of variants after clump: 6\n",
      "MSRA\n",
      "Number of variants before clump: 19\n",
      "Number of variants after clump: 7\n",
      "SNAP29\n",
      "Number of variants before clump: 21\n",
      "Number of variants after clump: 9\n",
      "KIFBP\n",
      "Number of variants before clump: 20\n",
      "Number of variants after clump: 8\n",
      "HGS\n",
      "Number of variants before clump: 4\n",
      "Number of variants after clump: 3\n",
      "SNAP23\n",
      "Number of variants before clump: 16\n",
      "Number of variants after clump: 4\n",
      "MAP2K6\n",
      "Number of variants before clump: 8\n",
      "Number of variants after clump: 3\n",
      "RRM2\n",
      "Number of variants before clump: 20\n",
      "Number of variants after clump: 13\n",
      "ATG4A\n",
      "Number of variants before clump: 8\n",
      "Number of variants after clump: 8\n",
      "AKR1C4\n",
      "Number of variants before clump: 3\n",
      "Number of variants after clump: 1\n",
      "DAB2\n",
      "Number of variants before clump: 9\n",
      "Number of variants after clump: 4\n",
      "CORO1A\n",
      "Number of variants before clump: 8\n",
      "Number of variants after clump: 2\n",
      "C2CD2L\n",
      "Number of variants before clump: 3\n",
      "Number of variants after clump: 2\n",
      "ERBIN\n",
      "Number of variants before clump: 11\n",
      "Number of variants after clump: 5\n",
      "MAX\n",
      "Number of variants before clump: 11\n",
      "Number of variants after clump: 5\n",
      "AKT1S1\n",
      "Number of variants before clump: 5\n",
      "Number of variants after clump: 4\n",
      "ALDH1A1\n",
      "Number of variants before clump: 5\n",
      "Number of variants after clump: 4\n",
      "SMAD1\n",
      "Number of variants before clump: 20\n",
      "Number of variants after clump: 10\n",
      "BCR\n",
      "Number of variants before clump: 5\n",
      "Number of variants after clump: 5\n",
      "WASF3\n",
      "Number of variants before clump: 3\n",
      "Number of variants after clump: 3\n",
      "BAMBI\n",
      "Number of variants before clump: 5\n",
      "Number of variants after clump: 4\n",
      "SF3B4\n",
      "Number of variants before clump: 31\n",
      "Number of variants after clump: 19\n",
      "ANXA10\n",
      "Number of variants before clump: 4\n",
      "Number of variants after clump: 2\n",
      "CDKN2D\n",
      "Number of variants before clump: 17\n",
      "Number of variants after clump: 4\n",
      "DUSP3\n",
      "Number of variants before clump: 2\n",
      "Number of variants after clump: 1\n",
      "FEN1\n",
      "Number of variants before clump: 2\n",
      "Number of variants after clump: 2\n",
      "IGFBP4\n",
      "Number of variants before clump: 4\n",
      "Number of variants after clump: 2\n",
      "PPIB\n",
      "Number of variants before clump: 15\n",
      "Number of variants after clump: 5\n",
      "TBCC\n",
      "Number of variants before clump: 17\n",
      "Number of variants after clump: 6\n",
      "EBAG9\n",
      "Number of variants before clump: 8\n",
      "Number of variants after clump: 4\n",
      "FAM3C\n",
      "Number of variants before clump: 7\n",
      "Number of variants after clump: 4\n",
      "CIAPIN1\n",
      "Number of variants before clump: 9\n",
      "Number of variants after clump: 7\n",
      "TRAF2\n",
      "Number of variants before clump: 30\n",
      "Number of variants after clump: 18\n",
      "PLPBP\n",
      "Number of variants before clump: 18\n",
      "Number of variants after clump: 8\n",
      "GNE\n",
      "Number of variants before clump: 6\n",
      "Number of variants after clump: 3\n",
      "FLI1\n",
      "Number of variants before clump: 14\n",
      "Number of variants after clump: 6\n",
      "TJAP1\n",
      "Number of variants before clump: 8\n",
      "Number of variants after clump: 4\n",
      "FHIT\n",
      "Number of variants before clump: 8\n",
      "Number of variants after clump: 4\n",
      "RAD23B\n",
      "Number of variants before clump: 3\n",
      "Number of variants after clump: 3\n",
      "TBC1D5\n",
      "Number of variants before clump: 6\n",
      "Number of variants after clump: 2\n",
      "ARHGAP25\n",
      "Number of variants before clump: 5\n",
      "Number of variants after clump: 2\n",
      "ITGB1BP2\n",
      "Number of variants before clump: 8\n",
      "Number of variants after clump: 4\n",
      "CNTNAP2\n",
      "Number of variants before clump: 9\n",
      "Number of variants after clump: 9\n",
      "ATXN10\n",
      "Number of variants before clump: 4\n",
      "Number of variants after clump: 3\n",
      "MED18\n",
      "Number of variants before clump: 15\n",
      "Number of variants after clump: 6\n",
      "VTA1\n",
      "Number of variants before clump: 16\n",
      "Number of variants after clump: 6\n",
      "EIF4EBP1\n",
      "Number of variants before clump: 2\n",
      "Number of variants after clump: 2\n",
      "GRPEL1\n",
      "Number of variants before clump: 10\n",
      "Number of variants after clump: 5\n",
      "CLIP2\n",
      "Number of variants before clump: 10\n",
      "Number of variants after clump: 5\n",
      "EIF4B\n",
      "Number of variants before clump: 15\n",
      "Number of variants after clump: 7\n",
      "SKAP2\n",
      "Number of variants before clump: 17\n",
      "Number of variants after clump: 5\n",
      "SOD1\n",
      "Number of variants before clump: 2\n",
      "Number of variants after clump: 2\n",
      "DIABLO\n",
      "Number of variants before clump: 40\n",
      "Number of variants after clump: 7\n",
      "CNST\n",
      "Number of variants before clump: 12\n",
      "Number of variants after clump: 3\n",
      "DBNL\n",
      "Number of variants before clump: 12\n",
      "Number of variants after clump: 8\n",
      "NPPC\n",
      "Number of variants before clump: 8\n",
      "Number of variants after clump: 5\n",
      "SERPINB1\n",
      "Number of variants before clump: 11\n",
      "Number of variants after clump: 3\n",
      "STAT5B\n",
      "Number of variants before clump: 14\n",
      "Number of variants after clump: 4\n",
      "AKT3\n",
      "Number of variants before clump: 3\n",
      "Number of variants after clump: 2\n",
      "LHB\n",
      "Number of variants before clump: 5\n",
      "Number of variants after clump: 5\n",
      "CTF1\n",
      "Number of variants before clump: 13\n",
      "Number of variants after clump: 3\n",
      "IL2RB\n",
      "Number of variants before clump: 1\n",
      "Number of variants after clump: 1\n",
      "DPY30\n",
      "Number of variants before clump: 2\n",
      "Number of variants after clump: 2\n",
      "DUOX2\n",
      "Number of variants before clump: 6\n",
      "Number of variants after clump: 6\n",
      "LAP3\n",
      "Number of variants before clump: 4\n",
      "Number of variants after clump: 4\n",
      "VPS53\n",
      "Number of variants before clump: 7\n",
      "Number of variants after clump: 3\n",
      "PPP1R2\n",
      "Number of variants before clump: 19\n",
      "Number of variants after clump: 6\n",
      "SCP2\n",
      "Number of variants before clump: 3\n",
      "Number of variants after clump: 3\n",
      "ADCYAP1R1\n",
      "Number of variants before clump: 1\n",
      "Number of variants after clump: 1\n",
      "STK4\n",
      "Number of variants before clump: 8\n",
      "Number of variants after clump: 3\n",
      "FOXO3\n",
      "Number of variants before clump: 16\n",
      "Number of variants after clump: 9\n",
      "IFNG\n",
      "Number of variants before clump: 8\n",
      "Number of variants after clump: 5\n",
      "FXN\n",
      "Number of variants before clump: 18\n",
      "Number of variants after clump: 7\n",
      "BAX\n",
      "Number of variants before clump: 4\n",
      "Number of variants after clump: 4\n",
      "TIA1\n",
      "Number of variants before clump: 14\n",
      "Number of variants after clump: 4\n",
      "MAGED1\n",
      "Number of variants before clump: 4\n",
      "Number of variants after clump: 4\n",
      "SNX9\n",
      "Number of variants before clump: 6\n",
      "Number of variants after clump: 3\n",
      "RCOR1\n",
      "Number of variants before clump: 1\n",
      "Number of variants after clump: 1\n",
      "FOXO1\n",
      "Number of variants before clump: 7\n",
      "Number of variants after clump: 4\n",
      "VPS37A\n",
      "Number of variants before clump: 9\n",
      "Number of variants after clump: 5\n",
      "FADD\n",
      "Number of variants before clump: 16\n",
      "Number of variants after clump: 8\n",
      "CERT\n",
      "Number of variants before clump: 2\n",
      "Number of variants after clump: 2\n",
      "SRC\n",
      "Number of variants before clump: 6\n",
      "Number of variants after clump: 2\n",
      "MNDA\n",
      "Number of variants before clump: 10\n",
      "Number of variants after clump: 2\n",
      "MZT1\n",
      "Number of variants before clump: 11\n",
      "Number of variants after clump: 3\n",
      "TARBP2\n",
      "Number of variants before clump: 8\n",
      "Number of variants after clump: 3\n",
      "GLOD4\n",
      "Number of variants before clump: 7\n",
      "Number of variants after clump: 4\n",
      "NEFL\n",
      "Number of variants before clump: 11\n",
      "Number of variants after clump: 9\n",
      "HNRNPK\n",
      "Number of variants before clump: 9\n",
      "Number of variants after clump: 4\n",
      "STK11\n",
      "Number of variants before clump: 14\n",
      "Number of variants after clump: 7\n",
      "ATP6V1F\n",
      "Number of variants before clump: 2\n",
      "Number of variants after clump: 2\n",
      "ARNT\n",
      "Number of variants before clump: 4\n",
      "Number of variants after clump: 3\n",
      "SUGT1\n",
      "Number of variants before clump: 10\n",
      "Number of variants after clump: 6\n",
      "PPP1R12A\n",
      "Number of variants before clump: 10\n",
      "Number of variants after clump: 6\n",
      "HARS1\n",
      "Number of variants before clump: 16\n",
      "Number of variants after clump: 6\n",
      "YTHDF3\n",
      "Number of variants before clump: 14\n",
      "Number of variants after clump: 5\n",
      "MESD\n",
      "Number of variants before clump: 18\n",
      "Number of variants after clump: 7\n",
      "GOPC\n",
      "Number of variants before clump: 17\n",
      "Number of variants after clump: 4\n",
      "CRKL\n",
      "Number of variants before clump: 15\n",
      "Number of variants after clump: 5\n",
      "DCTN6\n",
      "Number of variants before clump: 3\n",
      "Number of variants after clump: 1\n",
      "CDC37\n",
      "Number of variants before clump: 9\n",
      "Number of variants after clump: 6\n",
      "PRDX3\n",
      "Number of variants before clump: 7\n",
      "Number of variants after clump: 3\n",
      "EIF4G1\n",
      "Number of variants before clump: 9\n",
      "Number of variants after clump: 4\n",
      "MYO9B\n",
      "Number of variants before clump: 6\n",
      "Number of variants after clump: 6\n",
      "ARHGEF12\n",
      "Number of variants before clump: 5\n",
      "Number of variants after clump: 1\n",
      "LAT2\n",
      "Number of variants before clump: 2\n",
      "Number of variants after clump: 2\n",
      "ATP6V1D\n",
      "Number of variants before clump: 6\n",
      "Number of variants after clump: 5\n",
      "AK1\n",
      "Number of variants before clump: 2\n",
      "Number of variants after clump: 2\n",
      "STX16\n",
      "Number of variants before clump: 1\n",
      "Number of variants after clump: 1\n",
      "NCK2\n",
      "Number of variants before clump: 13\n",
      "Number of variants after clump: 3\n",
      "AXIN1\n",
      "Number of variants before clump: 5\n",
      "Number of variants after clump: 1\n",
      "GPKOW\n",
      "Number of variants before clump: 1\n",
      "Number of variants after clump: 1\n",
      "STK24\n",
      "Number of variants before clump: 5\n",
      "Number of variants after clump: 1\n",
      "IL33\n",
      "Number of variants before clump: 1\n",
      "Number of variants after clump: 1\n",
      "GRAP2\n",
      "Number of variants before clump: 13\n",
      "Number of variants after clump: 4\n",
      "CA2\n",
      "Number of variants before clump: 2\n",
      "Number of variants after clump: 2\n",
      "NGF\n",
      "Number of variants before clump: 1\n",
      "Number of variants after clump: 1\n",
      "VASH1\n",
      "Number of variants before clump: 15\n",
      "Number of variants after clump: 5\n",
      "INPPL1\n",
      "Number of variants before clump: 6\n",
      "Number of variants after clump: 2\n",
      "METAP2\n",
      "Number of variants before clump: 25\n",
      "Number of variants after clump: 12\n",
      "FABP4\n",
      "Number of variants before clump: 3\n",
      "Number of variants after clump: 3\n",
      "FABP5\n",
      "Number of variants before clump: 4\n",
      "Number of variants after clump: 4\n",
      "TXLNA\n",
      "Number of variants before clump: 12\n",
      "Number of variants after clump: 6\n",
      "NCLN\n",
      "Number of variants before clump: 3\n",
      "Number of variants after clump: 2\n",
      "DNAJA2\n",
      "Number of variants before clump: 7\n",
      "Number of variants after clump: 3\n",
      "MGLL\n",
      "Number of variants before clump: 15\n",
      "Number of variants after clump: 6\n",
      "LEP\n",
      "Number of variants before clump: 2\n",
      "Number of variants after clump: 2\n",
      "PPME1\n",
      "Number of variants before clump: 1\n",
      "Number of variants after clump: 1\n",
      "DNAJB8\n",
      "Number of variants before clump: 2\n",
      "Number of variants after clump: 2\n",
      "ABL1\n",
      "Number of variants before clump: 8\n",
      "Number of variants after clump: 4\n",
      "CLSTN1\n",
      "Number of variants before clump: 3\n",
      "Number of variants after clump: 3\n",
      "NTF4\n",
      "Number of variants before clump: 3\n",
      "Number of variants after clump: 3\n",
      "MAP3K5\n",
      "Number of variants before clump: 5\n",
      "Number of variants after clump: 2\n",
      "IPCEF1\n",
      "Number of variants before clump: 5\n",
      "Number of variants after clump: 4\n",
      "IKBKG\n",
      "Number of variants before clump: 5\n",
      "Number of variants after clump: 1\n",
      "STX6\n",
      "Number of variants before clump: 7\n",
      "Number of variants after clump: 5\n",
      "CALB1\n",
      "Number of variants before clump: 2\n",
      "Number of variants after clump: 2\n",
      "USO1\n",
      "Number of variants before clump: 12\n",
      "Number of variants after clump: 4\n",
      "ILKAP\n",
      "Number of variants before clump: 6\n",
      "Number of variants after clump: 2\n",
      "SLIT2\n",
      "Number of variants before clump: 15\n",
      "Number of variants after clump: 7\n",
      "NDRG1\n",
      "Number of variants before clump: 16\n",
      "Number of variants after clump: 10\n",
      "MTPN\n",
      "Number of variants before clump: 1\n",
      "Number of variants after clump: 1\n",
      "LYAR\n",
      "Number of variants before clump: 4\n",
      "Number of variants after clump: 3\n",
      "IRAK1\n",
      "Number of variants before clump: 6\n",
      "Number of variants after clump: 2\n",
      "CCT5\n",
      "Number of variants before clump: 1\n",
      "Number of variants after clump: 1\n",
      "LTA4H\n",
      "Number of variants before clump: 24\n",
      "Number of variants after clump: 3\n",
      "RNF41\n",
      "Number of variants before clump: 5\n",
      "Number of variants after clump: 1\n",
      "AIFM1\n",
      "Number of variants before clump: 29\n",
      "Number of variants after clump: 4\n",
      "CASP2\n",
      "Number of variants before clump: 5\n",
      "Number of variants after clump: 2\n",
      "CLPP\n",
      "Number of variants before clump: 10\n",
      "Number of variants after clump: 8\n",
      "DCTN2\n",
      "Number of variants before clump: 1\n",
      "Number of variants after clump: 1\n",
      "BACH1\n",
      "Number of variants before clump: 4\n",
      "Number of variants after clump: 1\n",
      "NUDC\n",
      "Number of variants before clump: 8\n",
      "Number of variants after clump: 3\n",
      "ATP6AP2\n",
      "Number of variants before clump: 1\n",
      "Number of variants after clump: 1\n",
      "NDUFS6\n",
      "Number of variants before clump: 1\n",
      "Number of variants after clump: 1\n",
      "STXBP3\n",
      "Number of variants before clump: 3\n",
      "Number of variants after clump: 2\n",
      "BAG6\n",
      "Number of variants before clump: 6\n",
      "Number of variants after clump: 3\n",
      "PDLIM7\n",
      "Number of variants before clump: 4\n",
      "Number of variants after clump: 2\n",
      "PTPN6\n",
      "Number of variants before clump: 3\n",
      "Number of variants after clump: 2\n",
      "PTEN\n",
      "Number of variants before clump: 1\n",
      "Number of variants after clump: 1\n",
      "PPP1R9B\n",
      "Number of variants before clump: 1\n",
      "Number of variants after clump: 1\n",
      "GHRHR\n",
      "Number of variants before clump: 1\n",
      "Number of variants after clump: 1\n",
      "ANKRD54\n",
      "Number of variants before clump: 1\n",
      "Number of variants after clump: 1\n",
      "PAK4\n",
      "Number of variants before clump: 1\n",
      "Number of variants after clump: 1\n",
      "RUVBL1\n",
      "Number of variants before clump: 3\n",
      "Number of variants after clump: 2\n",
      "STX8\n",
      "Number of variants before clump: 1\n",
      "Number of variants after clump: 1\n",
      "GH2\n",
      "Number of variants before clump: 1\n",
      "Number of variants after clump: 1\n",
      "CPU times: user 22h 36min 52s, sys: 2min 17s, total: 22h 39min 9s\n",
      "Wall time: 23h 49min 12s\n"
     ]
    }
   ],
   "source": [
    "%%time\n",
    "test_proteins = our_protein1[\"Protein\"].unique()\n",
    "for i in our_protein1.Protein.unique():  \n",
    "    print (i)\n",
    "    try:\n",
    "        temp = our_protein1[our_protein1[\"Protein\"] == i ] \n",
    "        temp_clump = normal_clump(temp) \n",
    "        score = make_score(temp_clump)\n",
    "        file_name = \"all_protein_weights/\" + i + \".clumpbyp.weights\"\n",
    "        score.to_csv(file_name, sep=\"\\t\", index=None) \n",
    "    \n",
    "    except: \n",
    "        print (\"naughty protein- find out why\")\n",
    "        next"
   ]
  },
  {
   "cell_type": "markdown",
   "id": "f18b7e52-3da9-435d-b317-a43db0f9b6c0",
   "metadata": {},
   "source": [
    "### what the ld matrix looks like -> done in plink"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "id": "6b3388e4-d5a5-4fa1-9e6f-6fa216505dea",
   "metadata": {},
   "outputs": [],
   "source": [
    "ld_matrix = pd.read_csv(\"ld_matrix.ld\", delimiter=r\"\\s+\" )\n",
    "ld_matrix = ld_matrix[[\"SNP_A\", \"SNP_B\",\"R2\"]].copy() "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "id": "1953ce24-eddc-48a2-8aa4-6a2cb95ee965",
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>SNP_A</th>\n",
       "      <th>SNP_B</th>\n",
       "      <th>R2</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>1:1197697:A:G</td>\n",
       "      <td>1:1203822:T:C</td>\n",
       "      <td>0.192521</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>1:1197697:A:G</td>\n",
       "      <td>1:1204118:C:T</td>\n",
       "      <td>0.025365</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "           SNP_A          SNP_B        R2\n",
       "0  1:1197697:A:G  1:1203822:T:C  0.192521\n",
       "1  1:1197697:A:G  1:1204118:C:T  0.025365"
      ]
     },
     "execution_count": 6,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "ld_matrix.head(2)"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": null,
   "id": "9c2a1f75-4d9f-4677-b7e5-9433921665cb",
   "metadata": {},
   "outputs": [],
   "source": []
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3 (ipykernel)",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.10.9"
  }
 },
 "nbformat": 4,
 "nbformat_minor": 5
}
